<script>
    MathJax = {
        loader: {load: ['[tex]/textmacros']},
        tex: {
            inlineMath: [['$', '$'], ['\\(', '\\)']],
            tags: 'ams',
            packages: {'[+]': ['textmacros']},
            macros: {
                oe: '\u0153',
                OE: '\u0152',
                ae: '\u00E6',
                AE: '\u00C6',
                aa: '\u00E5',
                AA: '\u00C5',
                o: '\u00F8',
                O: '\u00D8',
                l: '\u0142',
                L: '\u0141',
                ss: '\u00DF',
                SS: '\u1E9E',
                dag: '\u2020',
                ddag: '\u2021',
                P: '\u00B6',
                copyright: '\u00A9',
                pounds: '\u00A3'
            }
        },
        chtml: {
            mtextInheritFont: true
        },
        sag: {
            mtextInheritFont: true
        },
        svg: {fontCache: 'global'}
    };
</script>
<script
        async id="MathJax-script"
        src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js">
</script>

<!DOCTYPE html>
<script>
    const Cite = require('citation-js')
</script>
<html lang="EN">
    <head>
        <style>
            h1 {
                font-size: 34px;
            }
            h1.title {
                font-size: 38px;
            }
            h2 {
                font-size: 30px;
            }
            h3 {
                font-size: 24px;
            }
            h4 {
                font-size: 18px;
            }
            h5 {
                font-size: 16px;
            }
            h6 {
                font-size: 12px;
            }
            code {
                color: inherit;
                background-color: rgba(0, 0, 0, 0.04);
            }
            pre:not([class]) {
                background-color: white
            }
            code {
                white-space: pre-wrap;
            }
            
            span.smallcaps {
                font-variant: small-caps;
            }
            
            span.underline {
                text-decoration: underline;
            }
            div.column {
                display: inline-block;
                vertical-align: top;
                width: 50%;
            }
            
            div.hanging-indent {
                margin-left: 1.5em;
                text-indent: -1.5em;
            }
            
            ul.task-list {
                list-style: none;
            }
            code {
            white-space: pre;
            }
        </style>
        <script type="text/javascript">
            if (window.hljs) {
                hljs.configure({languages: []});
                hljs.initHighlightingOnLoad();
                if (document.readyState && document.readyState === "complete") {
                    window.setTimeout(function () {
                        hljs.initHighlighting();
                    }, 0);
                }
            }
        </script>
        <style>
            img {
                max-width: 100%;
            }
            .tabbed-pane {
                padding-top: 12px;
            }
            .html-widget {
                margin-bottom: 20px;
            }
            button.code-folding-btn:focus {
                outline: none;
            }
            summary {
                display: list-item;
            }
            details > summary > p:only-child {
                display: inline;
            }
            pre code {
                padding: 0;
            }
            .dropdown-submenu {
                position: relative;
            }
            .dropdown-submenu > .dropdown-menu {
                top: 0;
                left: 100%;
                margin-top: -6px;
                margin-left: -1px;
                border-radius: 0 6px 6px 6px;
            }
            .dropdown-submenu:hover > .dropdown-menu {
                display: block;
            }
            .dropdown-submenu > a:after {
                display: block;
                content: " ";
                float: right;
                width: 0;
                height: 0;
                border-color: transparent;
                border-style: solid;
                border-left-color: #cccccc;
                margin-top: 5px;
                margin-right: -10px;
            }
            .dropdown-submenu:hover > a:after {
                border-left-color: #adb5bd;
            }
            .dropdown-submenu.pull-left {
                float: none;
            }
            .dropdown-submenu.pull-left > .dropdown-menu {
                left: -100%;
                margin-left: 10px;
                border-radius: 6px 0 6px 6px;
            }
        </style>
        <script type="text/javascript">
            // manage active state of menu based on current page
            $(document).ready(function () {
                // active menu anchor
                href = window.location.pathname
                href = href.substr(href.lastIndexOf('/') + 1)
                if (href === "")
                    href = "index.html";
                var menuAnchor = $('a[href="' + href + '"]');
                
                // mark it active
                menuAnchor.tab('show');
                
                // if it's got a parent navbar menu mark it active as well
                menuAnchor.closest('li.dropdown').addClass('active');
                
                // Navbar adjustments
                var navHeight = $(".navbar").first().height() + 15;
                var style = document.createElement('style');
                var pt = "padding-top: " + navHeight + "px; ";
                var mt = "margin-top: -" + navHeight + "px; ";
                var css = "";
                // offset scroll position for anchor links (for fixed navbar)
                for (var i = 1; i <= 6; i++) {
                    css += ".section h" + i + "{ " + pt + mt + "}\n";
                }
                style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
                document.head.appendChild(style);
            });
        </script>
        <style>
            .tabset-dropdown > .nav-tabs {
                display: inline-table;
                max-height: 500px;
                min-height: 44px;
                overflow-y: auto;
                border: 1px solid #ddd;
                border-radius: 4px;
            }
            .tabset-dropdown > .nav-tabs > li.active:before {
                content: "";
                font-family: 'Glyphicons Halflings';
                display: inline-block;
                padding: 10px;
                border-right: 1px solid #ddd;
            }
            .tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
                content: "&#xe258;";
                border: none;
            }
            .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
                content: "";
                font-family: 'Glyphicons Halflings';
                display: inline-block;
                padding: 10px;
                border-right: 1px solid #ddd;
            }
            .tabset-dropdown > .nav-tabs > li.active {
                display: block;
            }
            .tabset-dropdown > .nav-tabs > li > a,
            .tabset-dropdown > .nav-tabs > li > a:focus,
            .tabset-dropdown > .nav-tabs > li > a:hover {
                border: none;
                display: inline-block;
                border-radius: 4px;
                background-color: transparent;
            }
            .tabset-dropdown > .nav-tabs.nav-tabs-open > li {
                display: block;
                float: none;
            }
            .tabset-dropdown > .nav-tabs > li {
                display: none;
            }
        </style>
    </head>
    <div class="container-fluid main-container">
        <div class="navbar navbar-default  navbar-fixed-top" role="navigation">
            <div class="container">
                <div class="navbar-header">
                    <button type="button" class="navbar-toggle
                        collapsed" data-toggle="collapse"
                            data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                    </button>
                </div>
                <div id="navbar" class="navbar-collapse collapse">
                    <ul class="nav navbar-nav">
                        <ol>
                            <li>
                                <a href="#introduction">
                                    Introduction
                                </a>
                            </li>
                            <li>
                                <a href="#sec:deterministic_base_model">
                                    Formulation of a stochastic COVID-19 SEIR model
                                </a>
                                <ul>
                                    <a href="#deterministic-seir-structure">
                                        Deterministic SEIR structure
                                    </a>
                                </ul>
                            </li>
                            <li>
                                <a href = "#stochastic-perturbation">
                                    Stochastic Perturbation
                                </a>
                            </li>
                            <li>
                                <a href="#sec:inference">
                                    Statistical inference of the transmission rates,
                                    symptomatic ratio and noise intensity
                                </a>
                            </li>
                            <li>
                                <a href="#sec:sim">
                                    Simulation study
                                    <ul>
                                        <a href = "#sec:application_real_data">
                                            Application to real data
                                        </a>
                                        <ul>
                                            <a href="#des_data">
                                                Data description and construction
                                            </a>
                                        </ul>
                                        <ul>
                                            <a href="#des_data">
                                                Estimation of parameters
                                            </a>
                                        </ul>
                                
                                    </ul>
                                </a>
                            </li>
                            <li>
                                Conclusions
                            </li>
                            <li>
                                A computable version of the estimators
                            </li>
                            <li>
                                References
                            </li>
                        </ol>
                        <ul class="nav navbar-nav navbar-right">
                        </ul>
                    </ul>
                </div><!--/.nav-collapse -->
            </div><!--/.container -->
        </div><!--/.navbar -->
        
    </div>
    <body>
        <h1 class="title toc-ignore">
                Maximum likelihood estimation for a stochastic SEIR
                system for COVID-19
            </h1>
            <p class="author">
                Fernando Baltazar-Larios
            </p>
            <p class="author">
                Francisco Delgado-Vences
            </p>
            <p class="author">
                Saul Diaz-Infante*
            </p>
            <div class="abstract">
                <div class="abstract-title">
                    Abstract
            </div>
                <p>
                    The parameter estimation of epidemic data-driven models is a crucial
                    task. In some cases, we can formulate a better model by describing
                    uncertainty with appropriate noise terms. However, because of the
                    limited extent and partial information, (in general) this kind of model
                    leads to intractable likelihoods. Here, we illustrate how a stochastic
                    extension of the SEIR model improves the uncertainty quantification of
                    an overestimated MCMC scheme based on its deterministic model to count
                    reported-confirmed COVID-19 cases in Mexico City. Using a particular
                    mechanism to manage missing data, we developed MLE for some parameters
                    of the stochastic model, which improves the description of variance of
                    the actual data.
                </p>
            </div>
    </header>
        <div class="frontmatter">
            UNAM, CONACYT-UNAM-IMATE-OAXACA, CONACYT-Unison
        </div>
        <h1 id="introduction">
            Introduction
        </h1>
            <p>
                
                Parameter calibration of epidemiological models with data
                is a demanding but crucial task.
                The current COVID-19 pandemic has dramatically revealed
                the complexities and subtleties of developing epidemic
                data-driven models.
                Although computational cost would grow when we develop a
                model with stochastic differential equations, in some particular
                cases this would lead to tractable estimators. By quantifying uncertainty
                by Brownian motion and applying methods based on maximum likelihood,
                we can improve the calibration of a deterministic base with conventional
                methods. When we calibrate parameters of an epidemic model,
                we argue that we can sometimes improve the quality
                and computing effort by adding noise and proper information management.
            </p>
        <p>
            Models based on stochastic differential equations (SDEs)
            have the advantage of characterising the central tendencies&mdash;usually
            a regarding deterministic model to describe the mean.
            This stochastic structure could also capture sources of variability
            <span>
                [see e.g.
                <a href="#ref-Allen2007">
                    Allen (2007),
                </a>
                        <a href="#ref-Kloeden2017">
                    Kloeden (2017),
                </a>
                        <a href="#ref-Gardiner2009">
                    Gardiner (2009)
                </a>
                ].
            </span>
            These models represent an alternative to analyse
            the skewed data that is commonly sampled in the observations.
            Stochastic models, and particularly SDEs, attempt to capture
            in their formulation the random variability of the data, and therefore
            quantify the uncertainty that is naturally related to parameters or
            phenomena
            <span>
                <a href="#ref-Allen2007"}>
                Allen (2007)
                </a>.
            </span>
        </p>
        <p>
            One of the crucial challenges in likelihood-based inference
            for scalar SDEs is to compute of the underlying transition
            density, and hence the likelihood function. Thus, maximum likelihood
            estimation on the true likelihood of the solution of SDEs is a rare case.
            Moreover, because the available sampled solution
            is a sequence of discrete time observations, it is usually incomplete and
            limited, and we must instead appeal to approximations
            <span>
        <a href="#ref-iacus">
            Iacus (2008)
        </a>
    </span>.
            Furthermore, this situation worsens for a coupled system of SDEs.
        </p>
        <p>
            Regarding stochastic models for epidemiology, and in particular
            for COVID-19, in previous studies there are two types of works related
            to the one presented here. On one hand, some manuscripts report similar
            ideas but for simpler structures and with numeric experiments of
            synthetic nature. For example,
            <span>
        <a href="#ref-PaGrGrMa">
            Jiafeng et al. (2014)
        </a>
    </span>,
            reports the estimation of a SIS
            structure that only needs a scalar version of the model to
            approximate the likelihood
            <span>
        <a href="#ref-PaGrGrMa">
            Jiafeng et al. (2014)
        </a>
    </span>.
            The authors use a pseudo-likelihood and least square method to estimate
            the parameters in the model. They also calculate  confidence intervals by
            applying least squares.
            In contrast, the results presented here deal with an epidemic model
            of five non-linear coupled SDEs and with real data for COVID-19
            from Mexico City, we also have a natural discretised version
            of the MLE to estimate the parameters in the model.
        </p>
        On the other hand, there are complex stochastic models similar to
        the one considered in this manuscript that are only studied on a
        theoretical basis and/or with numerical simulations.
        However, this does not address estimations methods.
        For example, in
        <span>
        <a href="ref-faranda2020modeling">
        </a>
    </span>, the authors propose a SEIR model, with four classes of
        individuals, given by a random differential equation.
        Indeed, they consider a model where two parameters are Brownian
        motion with drift and one parameter is the solution of a SDE with
        a log-normal perturbation.
        Nevertheless, the authors do not estimate parameters
        from actual data to fit the model, the parameters are instead taken from
        previous works. Additionally, they used date information about
        confinement in Italy and France to validate the model.

        <p>
            In
            <span>
        <a href="#ref-djordjevic2021two">
           Đorđević (2021)
        </a>
    </span>
            , the authors study a version of the stochastic
            SEIR model  for the COVID-19. Their model is a system of stochastic
            differential equations driven by two independent standard Brownian motions.
            More precisely, their model depends on several parameters,
            including the constant transmission coefficients related to symptomatic
            infected individuals ($\beta$), hospitalised individuals
            and to superspreaders ($\beta'$). In their study, the authors formulate a
            stochastic model via a stochastic perturbation of the constant
            transmission rates $\beta,\beta'$. They show the existence and uniqueness
            of the global positive solution. They also study conditions
            under which extinction and persistence in mean hold.
            Finally, they provide  numerical simulations of the stochastic model.
            Nevertheless, estimation methods to fit the model is an open problem
            for their model.
        </p>

        This manuscript focus on the calibration of a stochastic SEIR
        (Susceptible-Exposed-Infective-Recovered) model
        with data of symptomatic reported and confirmed
        cases in Mexico City. According to a specific time window&mdash;where the
        outbreak follows an increasing profile&mdash;, our approach relies on the
        maximum likelihood estimation method.
        <p>
            Here, we formulate our stochastic SEIR model
            through a stochastic perturbation of the natural death rate of the
            population by a Brownian motion.  We then deduce a nonlinear and
            coupled It\^o stochastic differential equation (SDE), driven by
            an affine Gaussian multiplicative noise.
        </p>
        Our main objective pursuit the development of consistent
        Maximum Likelihood Estimator (MLE) for a stochastic
        SEIR structure.
        <p>
            Parameter estimation for SDEs is an active
            area in the statistical literature.
            For example, in
            <span>
        <a href="#ref-Young81">
            Young (1981)
        </a>
    </span>
            P. Young gives an overview of parameter estimation
            methods with continuous-time observation.
            <span>
        <a href="#ref-Soren-04">
            Sørensen (2004)
        </a>
    </span>
            H. Sørensen treat the case where the data correspond
            to the solution of SDEs observed in a sequence of discrete time points.
            We refer the reader to
            <span>
        <a href="#ref-Soren-04">
            Sørensen (2004)
        </a>
    </span>
            for an introduction of several
            inference methods as estimating functions, analytical and numerical
            approximations of the likelihood function, Bayesian analysis Multi Chain
            Monte Carlo (MCMC) methods, indirect inference,
            and Expectation-Maximisation (EM) algorithm.
        </p>
        <p>
            Due to the COVID-19 pandemic, the estimation of parameters
            and development of epidemic models are under special attention. The most
            recent advances combine several models and data
            estimation. Among others, we see relevant methods based on Kalman filters,
            data aggregation
            <span>
        <a href="#ref-Fintzi2017">
            Fintzi (2017)
        </a>
    </span>,
            and techniques from data science such as
            Artificial Intelligence and machine learning
            <span>
        <a href="#ref-Bragazzi2018">
            Bragazzi (2018).
        </a>
    </span>
        </p>
        <p>
            Our work is related to
            <span>
        <a href="#ref-Liu">
            Liu (2018),
        </a>
        <a href="#ref-PaGrGrMa">
            Pan (2014)
        </a>
        <a href="#ref-Ndanguza">
            Ndanguza (2011),
        </a>
        <a href="#ref-Hotta">
            Hotta (2010),
        </a>
        <a href="#ref-Rios">
            Rios (2020),
        </a>
    </span>
    
            Liu face the estimation
            for a system of SDEs in
            <a href="#ref-Liu">
                Liu (2018)
            </a>.
            <a href="#ref-Ndanguza">
                Ndanguza (2011),
            </a> focus on the
            calibration parameters of alternative stochastic SEIR  versions
            by adaptive MCMC and extended Kalman filter methods.
    
            Hotta
            report other Bayesian techniques in
            <a href="#ref-Hotta">
                Hotta (2010),
            </a>
            and
            <a href="#ref-Rios">
                Rios (2020),
            </a>
            apply a maximum likelihood method,  in
    
            <a href="#ref-Otunuga2021">
                Otunuga 2021
            </a>
            reports a parameter estimation with local
            lagged adapted generalised methods of moments.
        </p>
        <p>
            Our contribution consists of the development estimators for some crucial
            parameters of the stochastic SEIR model. Here, we apply the quadratic
            variation of all of three processes to estimate the volatility parameter
            and calculate the MLE for parameters representing the symptomatic infection
            rate, asymptomatic infection rate, and proportion of symptomatic individuals.
            Then, from a theoretical result, we deduce a target parameters likelihood
            without assuming a prior data distribution.
            In short, this likelihood and a mechanism to manage incomplete data
            form the core mechanism that allows us to estimate the parameters.
        </p>
        Data-driven epidemic models usually treat observations as counters.
        Assuming that data follows a given distribution, this could be a Poisson,
        negative binomial, and so on.
        <a href="#ref-ONeill2010">
            ONeill (2010)
        </a>.
        Then, by applying some statistical inference tools to estimate some parameters,
        the model is fitted to the incidence data.
        <p>
            Here, we avoid this assumption.
            A description of the procedure follows.
            We use the Radon-Nikodym derivative of two equivalent
            measures&mdash;which, in our case, are the respective solutions
            of the stochastic SEIR system with two different sets of values of
            the parameters&mdash;to obtain the likelihood of the parameters
            of interest. This means that we use the Cameron-Martin-Girsanov
            Theorem to set up the likelihood.
            Note that the method to establish the likelihood does
            not assume a particular distribution of the solution,
            therefore,&mdash;we assume that the data have
            an abstract distribution.
        </p>
        We pursue estimators that satisfy strong consistency and
        numerical schemes of high order to its computing.
        With these ideas, we also illustrate their efficacy
        in the time of computing and performance with real-data
        by simulation.
        <p>
            Our simulations suggest that the methodology developed
            here improves a particular outcome of a fitting with MCMC.
            The scheme of prior distributions returns a bias fitting
            with this method. In contrast, the fitting of the SDE model
            with MLE improves the capture of the data variance and the
            time of computing.
        </p>
        <p>
            According to our Theorem about consistency, by computing
            residuals we confirm  that the stochastic
            model follows the profile of the regarding data and is
            coherent with our modelling assumptions.
            The results presented here point to a methodology
            that would boost the uncertainty quantification by SDE
            epidemic models with incomplete information and with
            dramatically less computing time. Here, we report the first
            pieces of evidence.
        </p>
        <p>
            After this brief introduction, in
            <a href="#sec:deterministic_base_model">
                Section 2
            </a>,
            we treat a deterministic SEIR
            model for COVID-19 and its calibration with MCMC.
            <a href="#sec:inference">
                Section 3
            </a>
            forms the main content of this article:
            the construction of estimators for important parameters and the analysis
            of its consistency. We illustrate our methodology via simulation in
            <a href="#sec:simulation">
                Section 4
            </a> and
            <a href="#sec:application_real_data">
                Section 5
            </a>
            with synthetic generated data and data of confirmed symptomatic
            cases from Mexico City.
        </p>
        <!--- Deterministic section--->
        <div style="display:none">
            \(
            \def\<#1>{\left<#1\right>}
            \newcommand{\E}{\mathbb{E}}
            \)
        </div>
        <h1 id="sec:deterministic_base_model">
            Formulation of a stochastic COVID-19 SEIR model
        </h1>
            <p>
                We consider that susceptible individuals become infected when they
                are in contact with asymptomatic individuals or individuals with
                symptoms. We propose that a proportion of asymptomatic individuals have
                a way to get relief and not die. A proportion of individuals infected
                with symptoms may die of the disease or may be relieved. We will
                introduce two types of control, vaccination of asymptomatic infected and
                susceptible individuals and treatment of individuals infected with
                symptoms.
            </p>
            <h2 id="deterministic-seir-structure">
                Deterministic SEIR Structure
            </h2>
                <p>
                    We use a common SEIR deterministic structure that has been applied
                    for COVID-19. Our formulation splits the entire population
                    <span class="math inline">
                    \(N\)
                    </span> according to the following compartments:
                </p>
                <dl>
                    <dt>
                        Suceptible
                        <span class="math inline">
                            \((S)\)
                        </span>
                    </dt>
                    <dd>
                        <p>
                            This compartment denotes the populations’ member under risk of
                            contagious.
                        </p>
                    </dd>
                    <dt>
                        Exposed
                        <span class="math inline">
                            \((E)\)
                        </span>
                    </dt>
                    <dd>
                        <p>
                            Members of this class come from the susceptible class and represent
                            the population fraction that hosts but can not transmit the SARS-CoV-2.
                            Then, after a period of
                            <span class="math inline">
                                \(\kappa^{-1}\)
                            </span>
                            days, they develop the faculty to spread this virus and became
                            infected.
                        </p>
                    </dd>
                    <dt>
                        Asymptomatic Infected
                        <span class="math inline">
                            \((I_a)\)
                        </span>
                    </dt>
                    <dd>
                        <p>
                            This class denotes the population fraction that spreads the
                            SARS-CoV-2 but does not develop symptoms or just develop mild symptoms.
                            This class also enclose all non-reported COVID-19 cases.
                        </p>
                    </dd>
                    <dt>
                        Symptomatic Infected
                        <span class="math inline">
                            \((I_s)\)
                        </span>
                    </dt>
                    <dd>
                        <p>
                            Represents the COVID-19 reported and confirmed cases that develop
                            symptoms.
                        </p>
                    </dd>
                    <dt>
                        Recovered
                        <span class="math inline">
                            \((R)\)
                        </span>
                    </dt>
                    <dd>
                        <p>
                            After a time period of length
                            <span class="math inline">
                                \(\alpha_s^{-1}\)
                            </span>, a member of the infected
                            class enters this compartment to become recovered and immune.
                        </p>
                    </dd>
                    <dt>
                        Death
                    </dt>
                    <dd>
                        <p>Here we count the cases that lamentably dies due to COVID-19.
                        </p>
                    </dd>
                </dl>
                <p>
                    In symbols, our deterministic base reads
                    <span class="math display">
                        \begin{equation}
                            \label{eqn:base_dynamics}
                            \begin{aligned}
                                f_{\beta}&amp;:= \beta_s I_S + \beta_a I_a
                                \\
                                S&#39;  &amp;=
                                    \mu +\gamma R
                                    - \left(\mu  + f_{\beta} \right)  S
                                \\
                                 E&#39; &amp; =  f_{\beta}S
                                     - (\kappa E + \mu E
                                     )
                                \\
                                {I_a}&#39; &amp;=
                                    p \kappa E
                                    - \big(\alpha_a + \mu \big)I_a
                                \\
                                 {I_s}&#39; &amp;=
                                    (1 - p) \kappa E
                                    - (\alpha_s +\mu)  I_s
                                \\
                                {R}&#39; &amp;=
                                    \alpha_a I_a + \alpha_s (1 - \theta)I_s -
                                    (\mu + \gamma) R
                                \\
                                D&#39; &amp;= \theta \alpha_s I_s.
                            \end{aligned}
                        \end{equation}
                    </span>
                    The following table
                    enclose a description of the underlying
                    parameters.
                </p>
                <div class="table*">
                    <table>
                        <caption style="caption-side:bottom">
                            Notation and meaning of parameters of model in Equation
                            \eqref{eqn:base_dynamics}.
                        </caption>
                        <thead>
                        <tr class="header">
                            <th style="text-align: right;">Parameter</th>
                            <th style="text-align: left;">Description</th>
                        </tr>
                        </thead>
                        <tbody>
                        <tr class="odd">
                            <td style="text-align: right;">
                        <span class="math inline">
                            \(\mu\)
                        </span>
                            </td>
                            <td style="text-align: left;">
                                Natural death rate
                            </td>
                        </tr>
                        <tr class="even">
                            <td style="text-align: right;">
                        <span class="math inline">
                            \(\beta_s\)
                        </span>
                            </td>
                            <td style="text-align: left;">
                                Symptomatic infection rate
                            </td>
                        </tr>
                        <tr class="odd">
                            <td style="text-align: right;">
                        <span class="math inline">
                            \(\beta_a\)
                        </span>
                            </td>
                            <td style="text-align: left;">
                                Asymptomatic infection rate
                            </td>
                        </tr>
                        <tr class="even">
                            <td style="text-align: right;">
                        <span class="math inline">
                            \(\kappa\)
                        </span>
                            </td>
                            <td style="text-align: left;">
                                Transfer rate from the exposed class to
                                the infected stage,
                            </td>
                        </tr>
                        <tr class="odd">
                            <td style="text-align: right;"></td>
                            <td style="text-align: left;">thus
                                <span class="math inline">
                            \(\kappa^{-1}\)
                        </span>
                                is the average incubation time.
                            </td>
                        </tr>
                        <tr class="even">
                            <td style="text-align: right;">
                        <span class="math inline">
                            \(p\)
                        </span>
                            </td>
                            <td style="text-align: left;">
                                Proportion of asymptomatic individuals
                            </td>
                        </tr>
                        <tr class="odd">
                            <td style="text-align: right;">
                        <span class="math inline">
                            \(\theta\)
                        </span>
                            </td>
                            <td style="text-align: left;">
                                Proportion of symptomatic individuals who
                                die due to the disease
                            </td>
                        </tr>
                        <tr class="even">
                            <td style="text-align: right;">
                        <span class="math inline">
                            \(\alpha_{s}^{-1}\)
                        </span>
                            </td>
                            <td style="text-align: left;">
                                Symptomatic recovery period
                            </td>
                        </tr>
                        <tr class="odd">
                            <td style="text-align: right;">
                        <span class="math inline">
                            \(\alpha_{a}^{-1}\)
                        </span>
                            </td>
                            <td style="text-align: left;">
                                Asymptomatic recovery average time
                            </td>
                        </tr>
                        <tr class="even">
                            <td style="text-align: right;">
                        <span class="math inline">
                            \(\gamma^{-1}\)
                        </span>
                            </td>
                            <td style="text-align: left;">
                                Disease-immunity period
                            </td>
                        </tr>
                        </tbody>
                    </table>
                </div>
                <p>
                    To calibrate parameters of this deterministic base
                    \eqref{eqn:base_dynamics},
                    we deploy a MCMC by counting the cumulative incidence of new infected and reported
                    symptomatic cases.
                    We denote by
                    <span class="math inline">
                        \(Y_t\)
                    </span>
                    this cumulative incidence at time
                    <span class="math inline">
                        \(t\)
                    </span>
                    and suppose that its profile follows a Poisson distribution with data mean
                    <span class="math inline">
                        \(\lambda_t = \mathbb{E} Y_t\)
                    </span>.
                    Thus, following ideas from
                    <cite>
                        <a href="#ref-Acuna2020">
                            Acuña (2020)
                        </a> we postulate priors for
                    </cite>
                    <span class="math inline">
                        \(p, \beta_s, \beta_a\)
                    </span>
                    and count the cumulative reported-confirmed cases in the CDMX-Valle de Mexico
                    <a href="#ref-DataMX">
                        database
                    </a>.
                </p>
            <p>
                Thus, our Bayesian estimation scheme results
                <span class="math display">
                \begin{equation}
                    \begin{aligned}
                        Y_t &amp; \sim
                            \mathrm{Poisson}(\lambda_t)
                        \\
                            \lambda_t =&amp; \int_0 ^ t (1-p) \delta_E E
                        \\
                            p &amp; \sim
                            \mathrm{Uniform}(0.3, 08)
                        \\
                            \kappa &amp; \sim
                            \mathrm{Gamma}(10, 50)
                    \end{aligned}
                \end{equation}
                </span>
                display the output of the mentioned MCMCM
                using the dynamic Hamiltonian developed by Betancourt and Hoffman, [see e.g
                <span class="citation" data-cites="Hoffman2014 Betancourt2017">
                    <a href="#ref-Hoffman2014"> Hoffman (2014)</a>,
                    <a href="#ref-Betancourt2017"> Betancourt (2017)</a>
                </span>
                ]
                from the rstan implementation with a sample of 10000 paths, see [GITHUB] form
                more details.
            </p>
            $\newcommand \dag \dagger$
            $\DeclareMathOperator*{\Var}{Var}$
            <div class="table*">
                <table>
                    <caption style="caption-side:bottom">
                        Calibrated and fixed values of the parameters of model \eqref{eqn:base_dynamics}.
                    </caption>
                    <thead>
                    <tr class="header">
                        <th style="text-align: right;">Parameter</th>
                        <th style="text-align: left;">Value</th>
                        <th style="text-align: center;">Reference</th>
                    </tr>
                    </thead>
                    <tbody>
                    <tr class="odd">
                        <td style="text-align: right;">
                    <span class="math inline">
                        \(\mu^{-1}\)
                    </span>
                        </td>
                        <td style="text-align: left;">
                            70  365 days
                        </td>
                        <td style="text-align: center;">
                    <span class="math inline">
                        \(\dag\)
                    </span>
                        </td>
                    </tr>
                    <tr class="even">
                        <td style="text-align: right;">
                    <span class="math inline">
                        \(\beta_s\)
                    </span></td>
                        <td style="text-align: left;">
                            0.0582153226067555
                        </td>
                        <td style="text-align: center;">
                            Estimated
                        </td>
                    </tr>
                    <tr class="odd">
                        <td style="text-align: right;">
                    <span class="math inline">\(\beta_a\)
                    </span>
                        </td>
                        <td style="text-align: left;">
                            0.510968165093383
                        </td>
                        <td style="text-align: center;">
                            Estimated
                        </td>
                    </tr>
                    <tr class="even">
                        <td style="text-align: right;">
                    <span class="math inline">
                        \(\kappa\)
                    </span>
                        </td>
                        <td style="text-align: left;">
                            0.196078
                        </td>
                        <td style="text-align: center;">
                    <span class="citation" data-cites="Tian2020">
                        <a href="#ref-Tian2020">
                            Tian 2020
                        </a>
                    </span></td>
                    </tr>
                    <tr class="odd">
                        <td style="text-align: right;">
                    <span
                            class="math inline">\(p\)
                    </span>
                        </td>
                        <td style="text-align: left;">
                            1 - 0.414495099891349
                        </td>
                        <td style="text-align: center;">
                            Estimated
                        </td>
                    </tr>
                    <tr class="even">
                        <td style="text-align: right;">
                    <span class="math inline">\(\theta\)
                    </span>
                        </td>
                        <td style="text-align: left;">
                            0.11
                        </td>
                        <td style="text-align: center;">
                           $\ddag$
                        </td>
                    </tr>
                    <tr class="odd">
                        <td style="text-align: right;">
                    <span class="math inline">
                        \(\alpha_{s}^{-1}\)
                    </span></td>
                        <td style="text-align: left;">
                            0.0925069
                        </td>
                        <td style="text-align: center;">
                    <span class="math inline">
                        \(\dag\)
                    </span>
                        </td>
                    </tr>
                    <tr class="even">
                        <td style="text-align: right;">
                    <span class="math inline">
                        \(\alpha_{a}^{-1}\)
                    </span>
                        </td>
                        <td style="text-align: left;">
                            0.167504
                        </td>
                        <td style="text-align: center;">
                    <span class="math inline">
                        \(\dag\)
                    </span>
                        </td>
                    </tr>
                    <tr class="odd">
                        <td style="text-align: right;">
                    <span class="math inline">
                        \(\gamma^{-1}\)
                    </span>
                        </td>
                        <td style="text-align: left;">
                            1/365
                        </td>
                        <td style="text-align: center;">
                    <span class="math inline">
                        \(\ddag\)
                    </span>
                        </td>
                    </tr>
                    </tbody>
                </table>
            </div>
            <div class="table*">
                <table>
                    <thead>
                    <tr class="header">
                        <th style="text-align: right;">State</th>
                        <th style="text-align: left;">Prior</th>
                        <th style="text-align: left;"></th>
                    </tr>
                    </thead>
                    <tbody>
                    <tr class="odd">
                        <td style="text-align: right;">
                    <span class="math inline">
                        \(E(0)\)
                    </span>
                        </td>
                        <td style="text-align: left;">
                    <span class="math inline">
                        \(\displaystyle
                            \frac{1}{N}
                            \cdot
                            \mathrm{Uniform}(47, 2100)\)
                    </span>
                        </td>
                        <td style="text-align: left;">
                    <span class="math inline">
                        \(\displaystyle
                        \frac{
                            \num{198.504524717486}
                        }{N}\)
                    </span>
                        </td>
                    </tr>
                    <tr class="even">
                        <td style="text-align: right;">
                    <span class="math inline">
                        \(I_S(0)\)
                    </span>
                        </td>
                        <td style="text-align: left;">
                    <span class="math inline">
                        \(\displaystyle
                            \num{74} / N\)
                    </span>
                        </td>
                        <td style="text-align: left;">
                        </td>
                    </tr>
                    <tr class="odd">
                        <td style="text-align: right;"><span
                                class="math inline">\(I_a(0)\)</span></td>
                        <td style="text-align: left;">
                    <span class="math inline">
                        \(\displaystyle
                            \frac{1}{N}
                            \cdot
                            \mathrm{uniform}(47, 2100)\)
                    </span>
                        </td>
                        <td style="text-align: left;">
                    <span class="math inline">
                        \(\displaystyle
                            \frac{
                                \num{99.1740343019642}
                            }{N}\)
                    </span>
                        </td>
                    </tr>
                    <tr class="even">
                        <td style="text-align: right;">
                    <span class="math inline">\(R(0)\)
                    </span>
                        </td>
                        <td style="text-align: left;">
                            0.0
                        </td>
                        <td style="text-align: left;">
                        </td>
                    </tr>
                    <tr class="odd">
                        <td style="text-align: right;">
                    <span class="math inline">
                        \(S(0)\)
                    </span>
                        </td>
                        <td style="text-align: left;">
                    <span class="math inline">
                        \(1 - (E(0) + I_a(0) + I_s(0) + R(0))\)
                    </span>
                        </td>
                        <td style="text-align: left;">
                    <span class="math inline">
                        \(\displaystyle
                            \frac{
                                \num{26446062.9010246}
                            }{N}\)
                    </span>
                        </td>
                    </tr>
                    </tbody>
                </table>
            </div>
            <div class="figure*">
                <figure>
                    <img
                            alt="image"
                            src="../Figures/MCMC_CDMXDataFitting.jpeg"
                            style="width:80.0%"
                    />
                    <figcaption>
                        MCMC fit to CDMX of model \eqref{eqn:base_dynamics}
                    </figcaption>
                </figure>
            </div>
        <h2 id="stochastic-perturbation">
            Stochastic Perturbation
        </h2>
            <p>
                Many works about epidemic modeling with SDEs
                <span class="citation" data-cites="Yang2014 Cai2019 GrGrHuMa-11 Schurz2015">
                    [
                        <a href="#ref-Yang2014">
                            Yang 2014
                        </a>,
                        <a href="#ref-Cai2019">
                            Cai 2019
                        </a>,
                        <a href="#ref-GrGrHuma-11">
                            Gray et al. 2011
                        </a>,
                        <a href="#ref-Schurz2015">
                            Schurz 2015
                        </a>
                    ]
                </span>,
                usually perturbs the infection rate that in our case we denote with
                <span class="math inline">
                    \(\beta_a\)
                </span>,
                <span class="math inline">
                    \(\beta_s\)
                </span>
                under the fact that this parameter lacks of information or its
                experimental estimation suffers high uncertainty.
                We proceed in different direction. Because our aim is to deduce estimators
                for infection rates
                <span class="math inline">
                    \(\beta_a, \beta_s\)
                </span>,
                by maximizing the underlying likelihood, our methodology demands
                a tractable form of this function. Perturbing
                <span class="math inline">
                    \(\beta_a\)
                </span> and/or
                <span class="math inline">
                    \(\beta_s\)
                </span>,
                we arrive at an intractable likelihood,
                which makes very difficult to study statistical
                properties with our proposed methodology.
            </p>
            <p>
                On the other hand, by perturbing the natural death rate, we obtain a
                SDE with dynamic consistency and algebraic structure that permit us to
                deduce a manageable likelihood. Our main contributions rely on in the
                following idea:
            </p>
                <blockquote>
                    <p>Perturb the parameters that lead to a tractable likelihood, optimize
                        and then deduce your desirable estimators.
                    </p>
                </blockquote>
            <p>
                In the following, we present the evidence that justify this
                methodology.
            </p>
            <p>
                In this section, we consider a stochastic version of the model given
                by \eqref{eqn:base_dynamics}. We aim to quantify uncertainty.
                To this end, we conveniently perturb the natural mortality rate
                <span class="math inline">
                    \(\mu\)
                </span>
                by a standard Wiener process
                <span class="math inline">
                    \(W(t)\)
                </span>.
                That is, we add a stochastic infinitesimal fluctuation of the parameter
                <span class="math inline">
                    \(\mu\)
                </span>
                in the interval
                <span class="math inline">
                    \((t, t + dt)\)
                </span>
                by adding the corresponding Wiener increment
                <span class="math inline">
                    \(dW(t)\)
                </span>.
                In symbols, we have
                <span class="math display">
                    \begin{equation}
                        \label{eqn:mu_perturbation}
                        \mu dt\rightsquigarrow \mu dt + \sigma dW(t).
                    \end{equation}
                </span>
                This perturbation infinitesimally quantifies the environmental fluctuations
                of the population mortality, due, for example, to differences in the
                health of individual conditions and other-regarding environmental
                factors. Furthermore, this particular form allows us to deduce a closed
                form for the underlying likelihood. Because our model couples the spread
                and other processes, this perturbation also implicitly captures random
                fluctuations from all involved compartments.
            </p>
            <p>
                In other words, we describe this effect by a stochastic process with
                mean zero and dispersion proportional to
                <span class="math inline">
                    \(\sigma \sqrt{t}\)
                </span>.
                According to Wiener statistical properties, we have
                <span class="math inline">
                    \(
                        \mathbb{E}
                        \left (
                            \mu dt +
                            \sigma dW(t)
                        \right )
                        =
                        \mu dt
                    \)
                </span> and
                <span class="math inline">
                    \(
                        \Var
                        \left (
                            \mu dt +
                            \sigma dW(t) =
                            \sigma^2 dt
                        \right )
                    \)
                </span>.
                This approach has been employed in the emergent research of
                mathematical epidemiology
                <span class="citation" data-cites="Chang2019 Acunya2018 Liu2018">
                    [
                        see for example
                        <a href="#ref-Chang2019">
                            Chang 2019
                        </a>,
                        <a href="#ref-Acunya2018">
                            Acuña 2018
                        </a>,
                        <a href="#ref-Liu2018">
                            Liu 2018
                        </a>
                    ]
                </span>.
            </p>
            <p>
                Substituting perturbation
                \eqref{eqn:mu_perturbation} in equation
                \eqref{eqn:base_dynamics} results,
                <span class="math display">
                    \begin{equation}
                        \label{eqn:sde_model}
                        \begin{aligned}
                            d {S}(t)  =&amp;
                                \big[\mu - \mu S(t) - f_{\beta} S(t)
                                +\gamma R(t)  \big] dt
                                + \sigma \big(1- S(t)\big) dW(t)
                            \\
                              d {E}(t) =&amp; \big[  f_{\beta} S(t)
                                  - \kappa  E(t)
                                  - \mu E(t) \big] dt
                                  - \sigma E(t) dW(t)
                             \\
                             d {I_a}(t) =&amp; \big[
                                 p \kappa E(t)
                                 - (\alpha_a +\mu) I_a(t)  \big] dt
                                 - \sigma I_a(t) dW(t)
                             \\
                              d {I_s}(t) =&amp; \big[
                                 (1 - p) \kappa E (t)
                                 - (\alpha_s +\mu)  I_s(t) \big] dt
                                 - \sigma I_s(t) dW(t)
                             \\
                             d {R}(t) =&amp;
                                \big[
                                    \alpha_a I_a(t) +
                                    \alpha_s I_s(t) - (\mu + \gamma) R(t)
                                \big] dt
                                - \sigma R(t) dW(t),
                             \qquad
                             t \in &amp; [0, T] .
                         \end{aligned}
                    \end{equation}
                </span>
                Since
                <span class="math inline">
                    \(
                        dN(t) = dS(t) + dE(t) + dI_a(t) +dI_s(t) + dR(t) = 0
                    \)
                </span>,
                normalising with respect to the size of the population
                <span class="math inline">
                    \(N\)
                </span>,
                we see that
                <span class="math display">
                    \[
                        S(t) + E(t) + I_a(t) + I_s(t) + R(t) = 1.
                    \]
                </span>
                Existence and uniqueness of the solution in a compact time
                interval
                <span class="math inline">
                    \([0,T]\)
                </span>
                follows from the boundedness,
                local Lipschitz and linear growth conditions on the
                coefficients of each SDE component
                <span class="citation" data-cites="Oksendal2003">
                    <a href="#ref-Oksendal2003">
                        [see Thm. 5.2.1 Oaksendal (2003)]
                    </a>
                </span>.
                Furthermore, Burkholder-Davis-Gundy and Holder inequalities imply that,
                for
                <span class="math inline">
                    \(r \geq 2\)
                </span>
                <span class="math display">
                    \[
                        \mathbb{E}
                        \left[
                            |S(t)| ^ r
                            + |E(t)| ^ r
                            +  |I_a(t)| ^ r
                            + |I_s(t)| ^ r
                            + |R(t)| ^ r
                        \right]
                        &lt;
                        C (1 + T + T ^ {r/2} )
                        &lt; +\infty,
                    \]
                </span>
                with constant
                <span class="math inline">
                    \(C&gt;0\)
                </span>
                depending on the model parameters.
                We rewrite the SDEs in by applying Lamperti’s transformation to each
                component
                <span class="citation" data-cites="iacus">
                    <a href="#ref-iacus">
                        [see Iacus (2008)]
                    </a>
                </span>).
                Thus, the susceptible component
                <span class="math inline">
                    \(S\)
                </span>
                becomes
                <span class="math display">
                    \begin{equation}
                        \label{dlogS}
                        \begin{aligned}
                            -\frac{1}{\sigma}d
                            \big(
                                \log (1-S_t(t))
                            \big)
                            \Big[
                                \frac{\mu}{\sigma }
                                - \frac{ S(t) f_{\beta}}{\sigma \big(1-S(t)\big)}
                                + \frac{\gamma  R(t)}{\sigma \big(1-S(t)\big)}
                                + \tfrac{1}{2} \sigma \Big] dt + dW(t).
                        \end{aligned}
                    \end{equation}
                </span>
                Again, by Lamperti’s transformation for the
                equation of process
                <span class="math inline">
                    \(E\)
                </span>,
                we obtain
                <span class="math display">
                    \begin{equation}
                            \begin{aligned}
                            -\frac{1}{\sigma}
                            d(\log \big(E(t)\big))&amp;=
                            \Big[
                                -\frac{  S(t) f_{\beta} }{\sigma E(t)} +
                                \frac{ \kappa }{\sigma } +
                                \frac{\mu }{\sigma } +
                                \frac{1}{2} \sigma\Big] dt +
                                dW(t).\label{dlogE}
                            \end{aligned}
                    \end{equation}
                </span>
            Analogously, we obtain the Lamperti transformation, for the components processes
            <span class="math inline">
                \(I_a,I_s, R\)
            </span>:
            <span class="math display">
                \begin{align}
                -\frac{1}{\sigma}
                    d\big(\log (I_a(t))\big)
                    &amp;=
                        \Big[
                            -\frac{\kappa p E(t)}{\sigma I_a(t)} +
                            \frac{(\alpha_a + \mu )}{\sigma } +
                            \frac{1}{2} \sigma
                        \Big] dt + dW(t), \label{dlogIa}
                    \\
                    -\frac{1}{\sigma}
                        d\big(\log (I_s(t))\big)
                        &amp;=
                        \Big[
                            -\frac{\kappa (1-p) E(t)}{\sigma I_s(t)} +
                            \frac{(\alpha_s + \mu )}{\sigma } +
                            \frac{1}{2} \sigma\Big] dt + dW(t),
                        \label{dlogIs}
                    \\
                    -\frac{1}{\sigma}
                        d\big(\log (R(t))\big)
                        &amp;=
                        \Big[
                            - \frac{\alpha_a I_a(t) +
                            \alpha_s I_s(t)}{\sigma R(t)} +
                            \frac{\mu + \gamma}{\sigma } +
                            \frac{1}{2} \sigma
                        \Big] dt
                        + dW(t). \label{dlogR}
                \end{align}
            </span>
        </p>
        <p>
            Then, we can write these SDEs as a five dimensions system of SDEs:
            <span class="math display">
                \begin{equation}
                    \label{dlog-SE}
                    -\frac{1}{\sigma}
                        d \mathbf{X}_{\beta,p}(t) =
                        F\big(\mathbf{X}_{\beta,p}(t)\big) dt +
                        d \mathbf{W}(t),
                \end{equation}
            </span>
            where,
            <span class="math display">
                \begin{equation}
                    \begin{aligned}
                        F\big(
                            \mathbf{X}_{\beta,p}(t)
                        \big)
                        & :=
                            \begin{pmatrix}
                                \dfrac{\mu}{\sigma } -
                                \dfrac{f_\beta S(t) }{\sigma \big(1-S(t)\big)}
                                +
                                \dfrac{\gamma R(t)}{\sigma \big(1-S(t)\big)}
                                + \tfrac{1}{2} \sigma
                                \\
                                    - \dfrac{f_\beta  S(t)  }{\sigma E(t)}
                                    + \dfrac{ \kappa   }{\sigma }
                                    + \dfrac{\mu }{\sigma }
                                    + \dfrac{1}{2} \sigma
                                \\
                                    - \dfrac{\kappa p E(t)}{\sigma I_a(t)}
                                    + \dfrac{(\alpha_a + \mu )}{\sigma }
                                    + \dfrac{1}{2} \sigma
                                \\
                                    - \dfrac{\kappa (1-p) E(t)}{\sigma I_s(t)}
                                    + \dfrac{(\alpha_s + \mu )}{\sigma }
                                    + \dfrac{1}{2} \sigma
                                \\
                                    - \dfrac{\alpha_a I_a(t) + \alpha_s I_s(t)}{\sigma R(t)}
                                    + \dfrac{\mu + \gamma}{\sigma }
                                    + \dfrac{1}{2} \sigma
                            \end{pmatrix},
                        \\
                        \mathbf{X}_{\beta,p}(t)
                            &:=
                                \begin{pmatrix}
                                    \log \big(1-S(t)\big)
                                    \\
                                    \log \big(E(t)\big)
                                    \\
                                    \log \big(I_a(t)\big)
                                    \\
                                    \log \big(I_s(t)\big)
                                    \\
                                    \log\big (R(t)\big)
                                \end{pmatrix}
                        \qquad
                        d\mathbf{W}(t)
                            \begin{pmatrix}
                                    dW(t)
                                    \\
                                    dW(t)
                                    \\
                                    dW(t)
                                    \\
                                    dW(t)
                                    \\
                                    dW(t)
                            \end{pmatrix}.
                    \end{aligned}
                \end{equation}
            </span>
        </p>
        <h1 id="sec:inference">
            Statistical inference of the transmission rates,
            symptomatic ratio and noise intensity
        </h1>
        <p>
            We apply the version of the Girsanov formula presented in
            <span class="citation" data-cites="saso19">
        <a href="#ref-saso19">
            Saso (2019)
        </a>
    </span> for the system in.
            We assume that the true value of the parameters
            <span class="math inline">
        \(\beta_{s,0},\beta_{a,0}, p_0\)
    </span> are unknown.
            We denote by
            <span class="math inline">
        \(\mathbb P_{\beta,p}\)
    </span>
            the law for the solution of equation \eqref{dlog-SE}.
            It is known that the measures
            <span class="math inline">
        \(\mathbb P_{\beta,p}\)
    </span> are equivalent for different values of
            <span class="math inline">
        \({\beta,p}\)
    </span>
    
            <span class="citation" data-cites="iacus">
    [
        see for instance
        <a href="#ref-iacus">
            Iacus (2008)
        </a>
    </span>,
            <span class="citation" data-cites="Bishwal2008">
        <a href="#Bishwal2008">
            Bishwal (2008)
        </a>
    </span>,
            <span class="citation" data-cites="saso19">
        <a href="#ref-saso19">
             Särkkä and Solin (2019)
        </a>
    ]
    </span>.
            Then, by Theorem 7.4 in
            <span class="citation" data-cites="saso19">
        <a href="#ref-saso19">
            Särkkä and Solin (2019)
        </a>
    </span>,
            we have that the likelihood ratio,
            or Radon-Nikodyn derivative has the form
            <span class="math display">
        \begin{equation}
            \label{Likelihood}
            \begin{aligned}
                \frac{d\mathbb P_\beta}{d\mathbb P_{\beta_0}}
                &=
                    \exp
                    \Bigg[
                         \int_0^T [ F(\mathbf{X}_{\beta,p}(t))  -
                            F(\mathbf{X}_{\beta_0,p_0}(t)) ]^{T} Q^{-1} d\mathbf{W}(t)
                            \\
                            -\frac{1}{2}
                            &
                                \int_0^T  [ F(\mathbf{X}_{\beta,p}(t))  -
                                F(\mathbf{X}_{\beta_0,p_0}(t)) ]^{T}Q^{-1}
                                [ F(\mathbf{X}_{\beta,p}(t))  -
                                F(\mathbf{X}_{\beta_0,p_0}(t)) ] d(t)
                    \Bigg],
            \end{aligned}
        \end{equation}
    </span> where <span class="math inline">\(Q\)</span>,
            in our case, is the five-identity matrix and
            <span class="math display">
        \begin{equation}
            \begin{aligned}
                 d\mathbf{W}
                &=
                \begin{pmatrix}
                    -(f_\beta-f_{\beta_0}) \dfrac{S(t) }{\sigma (1-S(t))}
                    \\
                    -(f_\beta-f_{\beta_0}) \dfrac{S(t) }{\sigma E(t)}
                    \\
                    -(p - p_0) \dfrac{\kappa E(t)}{\sigma I_a(t)}
                    \\
                    (p-p_0) \dfrac{\kappa E(t)}{\sigma I_s(t)}
                    \\
                    0
                \end{pmatrix},
            \end{aligned}
        \end{equation}
    </span>
            with
            <span class="math display">
        \begin{equation*}
            \begin{split}
                f_\beta-f_{\beta_0}
                &=
                    (
                        \beta_s I_s(t) + \beta_a
                        I_a(t)
                    )
                    -
                    (
                        \beta_{s,0} I_s(t) + \beta_{a,0} I_a(t)
                    )
                \\
                & =
                (
                    \beta_s  -  \beta_{s,0}
                ) I_s(t)
                +
                (
                    \beta_a - \beta_{a,0}
                ) I_a(t).
            \end{split}
        \end{equation*}
    </span>
        </p>
        <p>
            Note that the MLE for
            <span class="math inline">
        \(\beta_a\)
    </span>
            and
            <span class="math inline">
        \(\beta_s\)
    </span> is coupled, but independent of
            <span class="math inline">
        \(p\)
    </span>.
            Therefore, we calculate the MLE for
            <span class="math inline">
        \(\beta_a\)
    </span> and
            <span class="math inline">
        \(\beta_s\)
    </span>
            together and for
            <span class="math inline">
        \(p\)
    </span>
            separately.
        </p>
        <p>
            Thus,
        </p>
        <p>
        <span class="math display">
            \begin{equation}
                \begin{aligned}
                    d\mathbf{W}(t) ^{T}
                        Q^{-1}
                        d\mathbf{W}(t)
                    =&  -(f_\beta-f_{\beta_0})
                        [
                            \dfrac{S(t)}{\sigma (1 - S(t)) }
                            dW(t)
                            +
                            \dfrac{S(t)}{\sigma E(t)}
                            dW(t)
                        ]
                    \\
                    &
                        - (p-p_0)
                        [
                            \dfrac{\kappa E(t)}{\sigma I_a(t)} dW(t)
                            +
                            \dfrac{\kappa E(t)}{\sigma I_s(t)} dW(t)
                        ] ,
                \end{aligned}
            \end{equation}
        </span>
        </p>
        <p>
            and
            <span class="math display">
            \begin{equation}
                \begin{aligned}
                    d\mathbf{W}^{T} Q^{-1}
                        [
                            F(
                                \mathbf{X}_{\beta,p}(t)
                            )  -
                            F(
                                \mathbf{X}_{\beta_0, p_0}(t)
                            )
                        ]
                        =&
                        (
                            f_\beta - f_{\beta_0})^2
                            \Big[
                                \Big(
                                    \frac{S(t)}{\sigma (1-S(t))}
                                \Big) ^ 2
                                +
                                \Big(
                                    \frac{S(t)}{ \sigma E(t)}
                                \Big)^2
                            \Big]
                        \\
                        & +
                            (p-p_0)^2
                            \Big[
                                \Big(
                                    \frac{\kappa E(t)}{\sigma I_a(t)}
                                \Big)^2
                                +
                                \Big(
                                    \frac{\kappa E(t)}{\sigma I_s(t)}
                                \Big)^2\Big].
                \end{aligned}
             \end{equation}
        </span>
        </p>
        <p>
            We denote by
            <span class="math inline">
            \(\star=a\)
        </span>
            or
            <span class="math inline">
            \(s\)
        </span>,
            thus
            <span class="math inline">
            \(\beta_\star\)
        </span>
            will mean the variable
            <span class="math inline">
            \(\beta_s\)
        </span> or
            <span class="math inline">
            \(\beta_a\)
        </span> and similarly
            <span class="math inline">
            \(I_\star\)
        </span> means
            <span class="math inline">
            \(I_a\)
        </span> or
            <span class="math inline">
            \(I_s\)
        </span>.
            In addition, when, for instance,
            <span class="math inline">
            \(\star=a\)
        </span>
            then
            <span class="math inline">
            \(\star^c = s\)
        </span>
            and vice versa.
        </p>
        <p>
            Then, by maximising the log likelihood with respect to
            <span class="math inline">
            \(\beta_\star\)
        </span> we have that
        </p>
        <p>
        <span class="math display">
            \begin{equation}
                \begin{aligned}
                    &
                    0 =
                    \int_0^T
                        \Big[
                            - \frac{S(t) I_\star(t)}{\sigma (1-S(t))}
                            - \frac{S(t) I_\star(t)}{\sigma E(t)} \Big]
                            dW(t)
                    \\
                 - &
                    \int_0^T
                    \Big[
                        (\beta\star-\beta_{\star,0} )I_\star(t)
                        + (\beta{\star^c} - \beta_{\star^c,0} ) I_{\star^c}(t)
                    \Big]I_\star(t)
                    \left(
                            \Big[
                                \frac{S(t) }{\sigma (1-S(t))}
                            \Big] ^ 2
                        +
                        \Big[
                            \frac{S(t)}{\sigma E(t)}
                        \Big]^2
                    \right) dt.
                \end{aligned}
            \end{equation}
        </span>
            then,
            <span class="math display">
            \begin{equation}
                \begin{aligned}
                    &amp;
                        (\beta\star-\beta_{\star,0} )
                            \int_0^T
                            \left(
                                \Big[
                                    \frac{S(t) I_\star(t)}{ (1-S(t))}
                                \Big]^2
                            +
                            \Big[
                                \frac{S(t) I_\star(t)}{ E(t)}
                            \Big]^2
                        \right) dt
                \\
                + &
                    (
                        \beta_{\star^c} - \beta_{\star^c,0}
                    )
                    \int_0 ^ T
                        I_{\star^c}(t) I_\star(t)
                        \left(
                            \Big[
                                \frac{S(t) }{ (1-S(t))}
                            \Big]^2
                            +
                            \Big[
                                \frac{S(t) }{ E(t)}
                            \Big]^2
                        \right) dt
                \\
                = &
                    - \sigma
                    \int_0^T
                        \Big[
                             \frac{S(t) I_\star(t)}{ (1-S(t))}
                             +  \frac{S(t) I_\star(t)}{ E(t)}
                        \Big]  dW(t).
                \end{aligned}
            \end{equation}
        </span>
            From this expression
            we obtain that
            <span class="math display">
            \begin{equation}
                \label{MLE-beta-beta0_1}
                \begin{aligned}
                    \begin{pmatrix}
                        J_s(T) &amp; J_{sa}(T)\\
                        J_{sa}(T &amp; J_a(T)
                    \end{pmatrix}
                    %
                    \begin{pmatrix}
                        \hat{\beta_s}  - \beta_{s,0}
                        \\
                        \hat{\beta_a} - \beta_{a,0}
                    \end{pmatrix}
                    =
                        - \sigma
                    \begin{pmatrix}
                        \int_0^T
                            \Big[
                                \dfrac{S(t) I_s(t)}{ (1-S(t))}
                                + \dfrac{S(t) I_s(t)}{ E(t)}
                            \Big] dW(t)
                        \\
                        \int_0^T
                            \Big[
                                \dfrac{S(t) I_a(t)}{(1 - S(t))}
                                + \dfrac{S(t) I_a(t)}{E(t)}
                            \Big]
                            dW(t)
                    \end{pmatrix},
                \end{aligned}
            \end{equation}
        </span>
            where
            <span class="math display">
        \begin{equation}
            \begin{aligned}
                J_\star(T)
                    &amp;:=
                        \int_0^T
                            \left(
                                \Big[
                                    \dfrac{S(t) I_\star(t)}{ (1-S(t))}
                                \Big]^2
                                +
                                \Big[
                                    \dfrac{S(t) I_\star(t)}{ E(t)}
                                \Big]^2
                            \right) dt,
                    \\
                J_{sa}(T)
                    &amp;:=
                        \int_0^T
                            I_a(t) I_s(t)
                            \left(
                                \Big[
                                    \dfrac{S(t)}{(1-S(t))}
                                \Big] ^ 2
                                +
                                \Big[
                                    \dfrac{S(t)}{E(t)}
                                \Big] ^ 2
                            \right) dt.
            \end{aligned}
        \end{equation}
    </span>
        </p>
        <p>
            Therefore,
            <span class="math display">
        \begin{equation}
            \label{MLE-beta-beta0}
            \begin{pmatrix}
                \hat{\beta_s}  - \beta_{s,0}
                \\
                \hat{\beta_a} - \beta_{a,0}
            \end{pmatrix}
            =
            - \sigma
            \begin{pmatrix}
                J_s(T) &amp; J_{sa}(T)
                \\
                J_{sa}(T &amp; J_a(T)
            \end{pmatrix}^{-1} \,
        %
            \begin{pmatrix}
                \int_0^T
                    \Big[
                        \dfrac{S(t) I_s(t)}{ (1-S(t))}
                        + \dfrac{S(t) I_s(t)}{ E(t)}
                    \Big]
                    dW(t)
                \\
                \int_0^T
                    \Big[
                        \dfrac{S(t) I_a(t)}{ (1-S(t))}
                        + \dfrac{S(t) I_a(t)}{ E(t)}
                    \Big]  dW(t)
            \end{pmatrix}.
        \end{equation}
    </span>
        </p>
        <p>Meanwhile, deriving the log likelihood with respect to <span
                class="math inline">\(p\)</span> yields <span
                class="math display">\[\begin{aligned}
    (p-p_0)
    \left(
        \int_0^T
            \Big[
                \dfrac{\kappa^2 E^2(t)}{ I_s^2(t)}
                + \dfrac{\kappa^2 E^2(t)}{ I_a^2(t)}
            \Big] dt
    \right)
    &amp;=
        \sigma
        \int_0^T
            \Big[
                -\dfrac{\kappa E(t)}{ I_a(t)} +
                \dfrac{\kappa E(t)}{ I_s(t)} \Big] dW(t).
  \end{aligned}\]</span> This implies that <span
                class="math display">\[\begin{aligned}
    \label{MLE_p}
    \hat{p}_{ML}-p_0 &amp;=
        \frac{\sigma }{J_2(T)}
            \int_0^T
                \Big[
                    - \dfrac{\kappa E(t)}{ I_a(t)}
                    + \dfrac{\kappa E(t)}{ I_s(t)}
                \Big] dW(t),\end{aligned}\]</span> with <span
                class="math display">\[J_2(T):=
        \int_0^T
            \Big[
                \dfrac{\kappa ^ 2 E ^ 2(t)}{I_s^2(t)}
                + \dfrac{\kappa ^ 2 E ^ 2(t)}{ I_a^2(t)}
            \Big] dt.\]</span></p>
        <p>Let <span class="math inline">\(X_0 ^ + :=
        \{
            (
                S(t_0),
                E(t_0),
                I_a(t_0),
                I_s(t_0)
            )
        \}\)</span> initial state where all populations classes are
            strictly positive. Denote by <span class="math inline">\(\varphi:=
        \left\{
            \mu,
            \beta_s,
            \beta_a,
            \kappa,
            p ,
            \theta,
            \alpha_s ,
            \alpha_a,
            \gamma
        \right\},\)</span> a model parameter configuration. Accordingly,
            to the van den Driessche definition, the reproductive number for the
            deterministic version of our dynamics results <span
                    class="math display">\[\mathcal{R}_0^{D}:=
        \dfrac{p \kappa \beta_s }{(\mu + \kappa) (\mu + \alpha_s)}
        +
        \dfrac{(1 - p) \kappa \beta_a}{(\mu + \kappa)(\mu +
\alpha_a)}.\]</span> Thus given the initial condition <span
                    class="math inline">\(X_0 ^ +\)</span> and a parameter configuration
            <span class="math inline">\(\varphi\)</span>, we define the set <span
                    class="math inline">\(\Omega ^* = \Omega (X_0 ^ + , \varphi,
T^*)\)</span> by <span class="math display">\[\begin{aligned}
        \Omega^{*} :=
        \left \{
            (S, E, I_a, I_s, R) \times [t_0 , T]:
        \right.
                &amp;
            S(T) \leq S(t) &lt; S(t_0),
            \\
            &amp;
        \left.
                E(t) &gt; E(t_0), \
                I_a(t) &gt; I_a(t_0), \
                I_s(t) &gt; I_s(t_0)
        \right \}.
     \end{aligned}\]</span> To prove consistency and asymptotic
            normality of the parameter, we will make the following hypotheses.</p>
        <div class="hypotheses" id="set_hypotheses">
            <p><strong>Hypotheses 1</strong>. <em>There exists <span
                    class="math inline">\(T_0&gt;0\)</span> such that for all <span
                    class="math inline">\(t\in [0,T_0]\)</span></em></p>
        </div>
        <ol>
            <li><p><span class="math inline">\(\mathcal{R}_0 ^ D &gt;
1\)</span></p></li>
            <li><p>The initial condition <span class="math inline">\(X_0 ^
+\)</span> and parameter configuration <span
                    class="math inline">\(\varphi\)</span> are such that <span
                    class="math inline">\({\Omega^* \neq  \emptyset}\)</span></p></li>
        </ol>
        <p>This implies that our results are only valid in a sufficiently large
            time window.</p>
        <div class="rem">
            <p><strong>Remark 1</strong>. <em>In other words, Hypothesis <a
                    data-reference="set_hypotheses" data-reference-type="ref"
                    href="#set_hypotheses">1</a> assures conditions to estimate
                parameters in a growth phase of the outbreak. Under these conditions,
                the susceptible population decreases while the Exposed and Infected
                classes are in the growth phase. Furthermore, according to the
                definition of set <span class="math inline">\(\Omega ^ {*}\)</span>,
                this growth phase occurs in a time window of size <span
                        class="math inline">\((T^* - t_0)\)</span>.</em></p>
        </div>
        <p>Hypotheses <a data-reference="set_hypotheses" data-reference-type="ref"
                         href="#set_hypotheses">1</a> implies that <span
                class="math display">\[\begin{aligned}
    \dfrac{1}{1-S(t)} &lt; \dfrac{1}{1-S(0)},
    \qquad
    \dfrac{1}{E(t)} &lt; \frac{1}{E(0)}
    \qquad \forall t\in [0, T_0].\end{aligned}\]</span> Now we prove the
            consistency of our estimators.</p>
        <div class="theorem">
            <p><span id="Consitency-th" label="Consitency-th"></span> Assume that
                Hypotheses <a data-reference="set_hypotheses" data-reference-type="ref"
                              href="#set_hypotheses">1</a> are satisfied. The estimators
                <span
                        class="math inline">\((\hat{\beta}_{s,ML},\hat{\beta}_{a,ML},\hat{p}_{ML})\)</span>
                are strongly consistent; that is, <span
                        class="math display">\[\label{consistency}
        \lim_{T \rightarrow T_0}
        \begin{pmatrix}
            \hat \beta_{s,ML}
            \\
            \hat \beta_{a,ML}
            \\
            \hat p_{ML}
        \end{pmatrix}
        =
        \begin{pmatrix}
            \beta_{s,0}
        \\
            \beta_{a,0}
        \\
            p_0
        \end{pmatrix},
        \qquad \mbox{with probability one.}\]</span></p>
        </div>
        <p>The proof of this theorem is deferred to supplementary material.</p>
        <p>To estimate the parameter of diffusion
            <span class="math inline">
                \(\sigma\)
            </span>, we can use the quadratic
            variation over
            <span class="math inline">
                \([0,T]\)
            </span> of the
            processes of system of SDEs denoted by
            <span class="math inline">
                \(&lt;*,*&gt;_T\)
            </span>. The estimator is given by
            <span class="math display">
                \begin{equation}
                    \label{est_sigma}
                    \hat{\sigma}^2:=\sum_{i=1}^5\frac{\hat{\sigma}^2_i}{5},
                \end{equation}
            </span>
            where
            <span class="math display">
                \begin{equation}
                    \label{sigmas}
                    \begin{aligned}
                        &amp;\hat{\sigma}^2_1
                             = \frac{&lt;S,S&gt;_T}{\int_0^T(1-S(t))^2dt},
                        %
                        \qquad
                          \hat{\sigma}^2_2
                             = \frac{&lt;E,E&gt;_T}{\int_0^TE(t)^2dt},
                        % %
                         \qquad
                          \hat{\sigma}^2_3
                             = \frac{&lt;I_a,I_a&gt;_T}{\int_0^TI_a(t)^2dt},
                        % %
                         \\
                         &amp; \hat{\sigma}^2_4
                             = \frac{&lt;I_s,I_s&gt;_T}{\int_0^TI_s(t)^2dt},
                        % %
                         \qquad
                          \hat{\sigma}^2_5
                             = \frac{&lt;R,R&gt;_T}{\int_0^TR(t)^2dt}.
                    \end{aligned}
                \end{equation}
            </span>
        </p>
        <h1 id="sec:sim">
            Simulation study
        </h1>
        <p>
    <span id="sec:simulation" label="sec:simulation">

    </span>
        </p>
        <h1 id="sec:application_real_data">
            Application to real data
        </h1>
        <h2 id="des_data">
            Data description and construction
        </h2>
        <p>In this section, we use the dataset of new symptomatic and confirmed
            COVID19 reported cases daily in Mexico City considering that its
            population is
            <span class="math inline">
        \(N=\num{26446435}\)
    </span>. The dataset contains 47 records in the period March 10 to April 25 of 2020.
            We use these records
            <span class="math inline">
        \(I^o_s\)
    </span> to
            construct the observations of the process
            <span class="math inline">
        \(I_s=I^{mx}_s:=I^o_s/N\)
    </span> of the system
            \eqref{eqn:sde_model}. Combining
            <span class="math inline">
        \(I_s\)
    </span> and Milstein scheme, we generate the
            others four processes of system
            \eqref{eqn:sde_model} using Algorithm
            <a data-reference="Alg-ds"
               data-reference-type="ref"
               href="#Alg-ds">[Alg-ds]
            </a>. With the parameters of ,
            <span class="math inline">
        \(\sigma=1/100\)
    </span>
            and
            <span class="math inline">
        \(\Delta=1/1000\)
    </span>
            Algorithm
            <a
                    data-reference="Alg-ds"
                    data-reference-type="ref"
                    href="#Alg-ds"
            >[Alg-ds]
            </a>
            works as follows.
        </p>
        <div class="algorithm">
            <div class="algorithmic">
                <p>
                    <em>
                        1 Choose
                    </em>
                    the initial values from Table
                    <a data-reference="#table:initial"
                       data-reference-type="ref"
                       href="#table:initial">
                        [table:initial]
                    </a>
                    for
                    <span class="math display">
                \[
                    E(0),
                    I_a(0)
                    \text{ and }
                     (R(0).
                \]
            </span>
                </p>
                <p>
                    <em>
                        2 Compute
                    </em>
                    <span class="math inline">
                \(S(0)=1-E(0)-I_a(0) -R(0)-I^{mx}_s\)
            </span>,
                    and make
                    <span class="math inline">
                \(n=0\)
            </span>.
                </p>
                <p>
                    <em>
                        3 Generate a Browninan Increment
                    </em>
                    <span class="math inline">
                    \(\Delta W\sim N(0,\Delta)\)
                </span>.
                </p>
                <p>
                    <em>
                        4 Compute Milstein's iteration
                    </em>
                    <span class="math display">
                \begin{equation*}
                    \begin{aligned}
                    S(t_{n+1})
                        = &
                            S(t_n) - ( \mu +\ beta_a I_a (t_{n})
                            + \beta_s I_s^{mx} (t_n)) \Delta S(t_{n})
                            + (\mu+\gamma R(t_n))*\Delta
                        \\
                        &
                        - \sigma(1-S(t_n)) \Delta W
                    \\
                    E(t_{n+1})
                        =&
                            E(t_{n}) - (\kappa+\mu)*\Delta E(t_{n})
                            + \Delta * (\beta_sI_s^{mx}(t_n)+ \beta_aI_a(t_{n})S(t_n)
                        \\
                            & -\sigma(E(t_n))\Delta W
                        \\
                    I_a(t_{n+1})
                        =&
                            I_a(t_{n}) - p \kappa E(t_{n}) \Delta
                            I_a(t_{n}) - (\alpha_a+\mu) I_a (t_{n}) \Delta
                        \\
                         &
                            -\sigma I_a(t_{n})\Delta W
                        \\
                    R(t_{n+1})
                        =&
                        1 - S(t_{n+1} - E(t_{n+1}) - I_a(t_{n+1}) - I^{mx}_s(t_{n+1})
                \end{aligned}
            \end{equation*}
        </span>
                <p>
                    <em>
                        5  If
                    </em>
                    <span class="math inline">
                \(n&lt;47\)
            </span> then
                    <em>
                        make
                    </em>
                    <span class="math inline">
                \(n=n+1\)
            </span>
                    and go to 2.
                </p>
                <p>
                    <em>
                        6 Else stop.
                    </em>
                </p>
            </div>
        </div>
        <p>
            With this procedure, we have 47 records in the same observation
            period for each of the five processes that should fit to the stochastic
            model given by  \eqrefeqn:sde_model}.
            In other words, we have observations of the processes
            of system \eqref{eqn:sde_model} at the times
            <span class="math inline">
        \(t_0 = 0\)
    </span>,
            <span class="math inline">
        \(t_1= \num{0.001}\)
    </span>,
            <span class="math inline">
        \(t_2 =\num{0.002}\)
    </span>,
            <span class="math inline">
        \(\ldots\)
    </span>,
            <span class="math inline">
        \(t_{46} = \num{0.046}\)
    </span>.
        </p>
        <h2 id="es_real">Estimation of parameters</h2>
        <p>We assume the following parameters as given: <span
                class="math inline">\(\gamma =1 /\num{365}\)</span>, <span
                class="math inline">\(\kappa = \num{0.196078}\)</span>, <span
                class="math inline">\(p = \num{0.5855049}\)</span>, <span
                class="math inline">\(\alpha_a = \num{0.167504}\)</span> <span
                class="math inline">\(\alpha_s = \num{0.0925069}\)</span>. Thus, using
            the described methodology in to generate data and applying <a
                    data-reference="est_sigma" data-reference-type="eqref"
                    href="#est_sigma">[est_sigma]</a>, we estimate <span
                    class="math inline">\(\sigma\)</span> and expressions in . Then, with
            these calculations, we deduce the MLE function of <span
                    class="math inline">\(\beta_s,\beta_a\)</span>, and <span
                    class="math inline">\(p\)</span>. Next, we generate 1000 datasets using
            the data of Mexico City. gives the average of these estimators and the
            corresponding 95% confidence interval.</p>
        <div class="table*">
            <table>
                <thead>
                <tr class="header">
                    <th style="text-align: right;">Parameter</th>
                    <th style="text-align: center;">Estimator</th>
                    <th style="text-align: center;">CIL</th>
                    <th style="text-align: center;">CIU</th>
                </tr>
                </thead>
                <tbody>
                <tr class="odd">
                    <td style="text-align: right;"><span
                            class="math inline">\(\beta_s\)</span></td>
                    <td style="text-align: center;">0.05915915</td>
                    <td style="text-align: center;">0.002545795</td>
                    <td style="text-align: center;">0.1157725</td>
                </tr>
                <tr class="even">
                    <td style="text-align: right;"><span
                            class="math inline">\(\beta_a\)</span></td>
                    <td style="text-align: center;">0.5099253</td>
                    <td style="text-align: center;">0.3782477</td>
                    <td style="text-align: center;">0.6416028</td>
                </tr>
                <tr class="odd">
                    <td style="text-align: right;"><span
                            class="math inline">\(p\)</span></td>
                    <td style="text-align: center;">0.5828079</td>
                    <td style="text-align: center;">0.5823265</td>
                    <td style="text-align: center;">0.5832893</td>
                </tr>
                <tr class="even">
                    <td style="text-align: right;"><span
                            class="math inline">\(\sigma\)</span></td>
                    <td style="text-align: center;">1.31701E-002</td>
                    <td style="text-align: center;">8.32156E-003</td>
                    <td style="text-align: center;">1.94434E-002</td>
                </tr>
                </tbody>
            </table>
        </div>

        <figure>
            <img
                    alt="
                    Comparison between the estimation of accumulated incidence of new
                    symptomatic cases according
                    to the Bayesian MCMC and our MLE approach. We refer the reader to
                    https://chart-studio.plotly.com/~sauld/47 for a plot visualisation.
            "
                    id="fig:real_data_fitting"
                    src="../Figures/Likening.jpeg"
                    style="width:80.0%"
            />
            <figcaption aria-hidden="true">
                Figure 2.
                Comparison between the estimation of
                accumulated incidence of new symptomatic cases according to the Bayesian
                MCMC and our MLE approach. We refer the reader to
                <a class="uri" href="https://chart-studio.plotly.com/~sauld/47">
                    https://chart-studio.plotly.com/~sauld/47
                </a> for a plot visualisation.
            </figcaption>
        </figure>
        <p>
            <a href="#fig:real_data_fitting">
                Figure 2
            </a> shows the average and the 95 % confidence interval of 1000 paths of
            the cumulative new symptomatic and confirmed COVID-19 reported cases
            from Mexico City. Note that our estimation follows the given data
            profile in a closer way than the corresponding Bayesian calibration.
            Further, all observations drop in the corresponding confidence interval.
            So, this suggests that our proposed methodology and the stochastic
            extension improves the Bayesian fit of this dataset.</p>
        <h2 id="validation-of-the-model">Validation of the model</h2>
        <!-- QQ Figure-->

        <figure>
            <img alt="Quantile-Quantile plot of the numerical residuals.
            Here we plot the quantile for the error computed by Equation \eqref{Wt_increments}
            and a theoretical Gaussian distribution distribution with zero mean and variance 1. "
                 id="fig:fig_qq"
                 src="../Figures/qq_plot.svg"/>
            <figcaption aria-hidden="true"> Quantile-Quantile plot of the numerical
                residuals. Here we plot the quantile for the error computed by Equation
                \eqref{Wt_increments} and a theoretical Gaussian distribution distribution
                with zero mean and variance 1.
            </figcaption>
        </figure>

        <p>
            To validate our stochastic model, we compute the residuals
            corresponding to data of reported and confirmed COVID-19 cases of Mexico
            City. Our target is to show that the residuals are Gaussian. If this is
            the case, then the noise
            <span class="math inline">
                \(W\)
            </span> in our
            stochastic model would be consistent with Brownian motion.
            <br/>
            Using equation \eqref{dlogS} we define the increments
            <span class="math display">
                \begin{equation}
                    \label{Wt_increments}
                    \begin{aligned}
                        \Delta W(t_k)
                        &amp;:=
                            \big[W(t_k)-W(t_{k-1})\big]
                            \\
                        &amp;= -\frac{1}{\sigma}
                            \Big[
                                \log
                                \big(1-S(t_k) \big)
                                    - \log\big(1-S(t_{k-1}) \big)
                            \Big]  -
                            \Big[
                                \frac{\mu}{\sigma}+\frac{1}{2}\sigma
                            \Big]\Delta
                        \\
                        &amp;\quad +
                        \frac{\Delta }{\sigma}
                        \Bigg(
                            \Bigg[
                                \frac{S(t_{k-1}) f_\beta(t_{k-1})}{1-S(t_{k-1})}
                            \Bigg]
                            - \gamma
                            \Bigg[
                                \frac{R(t_{k-1})}{1-S(t_{k-1})}
                            \Bigg]
                        \Bigg),
                    \end{aligned}
                \end{equation}
            </span>
            where
            <span class="math inline">
                \(\Delta=t_k-t_{k-1}\)
            </span> and
            <span class="math inline">
                \(f_\beta(t_{k})\)
            </span> is the evaluation of the function
            <span class="math inline">
                \(f_\beta\)
            </span>
                in time
            <span class="math inline">
                \(t_{k}\)
            </span>;
            that is,
            <span class="math display">
                \[
                    \begin{aligned}
                    f_{\beta}(t_k) =&amp; \beta_s I_s(t_k) + \beta_a
                    I_a(t_k).
                \end{aligned}
                \]
            </span>
        </p>

        <p>
            If the right-hand side of equation
            \eqref{Wt_increments} is Gaussian
            <span class="math inline">}
                \(\mathcal{N}(0,\Delta)\)
            </span>
            for
            <span class="math inline">
                \(k=1,\ldots, 46\)
            </span>,
            then the increments are Gaussian
            and also implies that the stochastic
            process
            <span class="math inline">
                \(W\)
            </span>
            is a Brownian motion, and
            therefore the stochastic model is well-fitted.
        </p>
        <p>
            We note that in expression \eqref{Wt_increments} there are four components
            of the stochastic processes, namely
            <span class="math inline">
                \(S, R, I_a, I_s\)
            </span>,
            but we only observed the
            component
            <span class="math inline">
                \(I_s\)
            </span>
            and the other three processes had to be approximated using the methodology
            from the previous sections.
        </p>
        <p> We use the parameter values of
            <a data-reference="es_real"
               data-reference-type="ref"
               href="#es_real">
                Section 5.2
            </a>, estimators
            of and
            <span class="math inline">
                \(\Delta=\num{0.001}\)
            </span>
                into
                \eqref{Wt_increments} and we get the residual
            <span class="math inline">
                \(\Delta\widehat{W}(t_k)\)
            </span>.
            Then, we prove that
            <span class="math inline">
                \(\Delta\widehat{W}(t_k)\)
            </span>
            are Gaussian with zero mean and finite variance, which leaves us
            to conclude that the model fits properly to the actual data.
            Indeed, shows a QQ-plot of standardised residual of 1000 paths of the real
            data versus a sample of same size of standard normal random variable.
        </p>
        <p>
            Since an important part of the quantile frequency in follows the
            reference profile, we can conclude that the residuals are Gaussian. This
            suggests that our stochastic model is well-fitted to the data.
        </p>

        <h1 id="conclusions">
            Conclusions
        </h1>

        <p>
            We studied the MLE for some key parameters of a stochastic SEIR
            model. These parameters are important to study and quantify uncertainty
            related to the stochastic nature of data. By using synthetic data, we
            verified that our approximation of the MLE is a good estimator for the
            infection transmission, symptomatic ratio and noise intensity
            parameters. In the supplementary material, we provide a detailed proof
            for the Gaussian consistency of our estimators and other important
            properties.
        </p>

        <p>
            We then fitted our stochastic model to a dataset of COVID-19
            confirmed symptomatic cases from México City. Because this information
            is limited and incomplete in the context of the regarding model, the
            database lacks of reliable information of the susceptible, exposed,
            asymptomatic and recover classes, we implemented the ideas as was
            explained in
            <a data-reference="des_data"
               data-reference-type="ref"
               href="#des_data">
                Section 5.1
            </a>. Thus, via simulation, we obtain confidence intervals that capture
            more observations than the given MCMC outcome with its respective
            deterministic version, see.
        </p>

        <p>
            One of our goals was to improve the uncertainty quantification of
            COVID-19 data by a stochastic SEIR structure based on Itô SDEs. Our
            simulations suggest that our MLE-estimators improve the variance
            description of a prescribed overfitting MCMC outcome. Previous results
            confirmed this behavior in other applications, see for example
            <span class="citation" data-cites="Chatzilena2019">
        <a href="#ref-Chatzilena2019">
            Chatzilena (2019)
        </a>
    </span>.
            Although our findings do not implicate a general methodology,
            most of the presented techniques would be applicable to problems of the
            same sort.
        </p>

        <p>
            The estimators developed here capture the noisy nature of the data
            with more precision. Furthermore, our simulations run in much less time
            with respect to the MCMC run with the STAN implementation. However, our
            conclusions rely on a very particular input for the underlying outcome,
            and we depend on the algebraic form of a given model to construct an
            estimator. We have confirmed the well-fitness of our stochastic model to
            real data, via error residual computing. However, it is necessary to
            explore our ideas in other epidemic structures, and with other datasets
            and target parameters. We should also develop more and stronger theories
            to construct consistent and computable versions of our estimators.
        </p>

        <p>
            As was discussed in the introduction,
            <span class="citation" data-cites="PaGrGrMa">
        <a href="#ref-PaGrGrMa">
             Jiafeng et al. (2014)
        </a>
    </span>
            reports the estimation of a simple stochastic SIS structure that needs
            only a scalar version of the model to approximate the likelihood.
            There are more complex stochastic models similar to the one studied here,
            but they have not addressed estimation methods,
            see for instance
            <span class="citation" data-cites="faranda2020modeling">
        <a href="#ref-faranda2020modeling">
            Faranda (2020)
        </a>
    </span>
            or
            <span class="citation" data-cites="djordjevic2021two">
        <a href="#ref-djordjevic2021two">
            Djordjevic (2021)
        </a>
    </span>.
            In contrast, our results deal
            with an epidemic model of five non-linear coupled SDEs. Furthermore, we
            combine the stochastic model with real data for COVID-19 from Mexico
            City and, using a natural discrete version of the MLE, we estimate
            the parameters. Therefore, we fit the model to the actual data, and we
            obtain a good behavior of the stochastic model.
        </p>

        <p>
    <span class="citation" data-cites="Otunuga2021">
        <a href="#ref-Otunuga2021">
            [Otunuga (2021)]
        </a>
    </span>
            also derives and calibrates a similar SEIR stochastic model
            for COVID-19 in
            <span class="citation" data-cites="Otunuga2021">
        <a href="#ref-Otunuga2021">
            [Otunuga (2021)]
        </a>
    </span>.
            Their methodology relies on in the Stratonovich stochastic calculus
            and new estimation scheme, which they named ’lagged adapted generalized
            method of moments’
            (LLGMM). The LLGMM is a dynamic non-parametric method and seems to be a
            well-behaved method; however, the properties of the estimated parameters
            (e.g. consistency, bias, efficiency, etc.) are (up to our knowledge)
            still an open question. Since we base our proposal in the Itô framework,
            and the MLE method, the results of our contribution are
            complementary.
        </p>

        <p>
            One limitation of our work is that we have only one noise in all the
            system. We then we pretend to extend the model in such manner that
            each process has an independent noise. One open question is if such
            model could be susceptible to be studied with the methodology developed
            in this work, which means applying the MLE method. In addition, it will
            be necessary to study if the model is well-fitted to the actual
            data.
        </p>
        <p>
            We also want to investigate if the model studied in this manuscript
            (and a possible extension) could be applied to other type of diseases or
            phenomena. Another question is to determine, via simulation, the minimum
            number of observations that ensure a nice approximation to the true
            parameters. Furthermore, this work missed a proof of the asymptotic
            normality of the estimators, which we guess that could be achieved by
            using the so-called Malliavin-Stein method. In addition, we want to
            study rates of convergence of the discretised version of the estimator
            to the continuous one.
        </p>

        <h1 id="Ap-1">
            A computable version of the estimators
        </h1>

        <p>
            It is possible to prove (see supplementary material) that
            <span class="math display">
        \begin{equation*}
            \begin{aligned}
                \label{beta_hat_f}
                \hat{\beta}_{s,ML}
                    = &amp;
                    \dfrac{1 }{J_s J_a - J_{sa}^2}
                    \Bigg[
                        J_a(T) \int_0^T  \dfrac{S(t) I_s(t)}{ (1-S(t))}
                        \Big[
                            d
                            \big(
                                \log (1-S_t)
                            \big)
                        \Big]
                    \\
                    &amp;
                    + J_a(T)
                \int_0^T
                    \dfrac{S(t) I_s(t)}{ (1-S(t))}
                    \Big[
                        \Big(
                            \mu +
                            \frac{\gamma  R}{ (1-S)} +
                            \tfrac{1}{2}
                            \sigma^2
                        \Big) dt
                    \Big]
                \\
                &amp;
                    +
                     J_a(T)
                    \int_0^T
                        \frac{S(t) I_s(t)}{ E(t)}
                        d(\log (E_t)) + J_a(T)
                        \int_0^T
                        \frac{S(t) I_s(t)}{ E(t)}
                        \Big(
                            \mu
                            +
                            \frac{1}{2}
                            \sigma^2
                        \Big) dt
                \\
                &
                    +
                        \kappa
                        J_a(T)
                        \int_0^T
                            \frac{S(t) I_s(t)}{ E(t)}
                            dt
                    - J_{sa}(T)
                        \int_0^T
                            \frac{S(t) I_a(t)}{ (1-S(t))}
                            d\big(\log (1-S_t) \big)
                \\
                &
                    -
                        J_{sa}(T)
                        \int_0^T
                            \frac{S(t) I_a(t)}{ (1-S(t))}
                            \Big(
                                \mu + \frac{\gamma  R}{ (1-S)} + \tfrac{1}{2}
                                \sigma^2
                            \Big) dt
                \\
                &amp; \
                    -   J_{sa}(T) \int_0^T
                        \frac{S(t) I_a(t)}{ E(t)}  d(\log (E_t)) -
                        J_{sa}(T)
                            \int_0^T
                                \frac{S(t) I_a(t)}{ E(t)}
                                \Big(
                                    \mu +
                                    \frac{1}{2}
                                    \sigma^2
                                \Big) dt
                \\
                &amp; - \kappa J_{sa}(T)
                    \int_0^T
                        \frac{S(t) I_a(t)}{ E(t)} dt
                    \Bigg]
            \end{aligned}
        \end{equation*}
    </span>
            and
        </p>
        <p>
    <span class="math display">
        \begin{equation}
            \label{beta_hat_a_f}
            \begin{aligned}
                \hat{\beta}_{a,ML} &amp;=
                    \frac{1 }{J_s J_a - J_{sa}^2}
                    \Bigg[
                        - J_{sa}(T)
                        \int_0^T
                        \frac{S(t) I_s(t)}{ (1-S(t))}
                        \Big[ d\big(\log (1-S_t) \big)
                    \Big]
                \\
                &amp;
                -
                J_{sa}(T)
                \int_0^T
                    \frac{S(t) I_s(t)}{ (1-S(t))}
                    \Big[
                        \Big(
                            \mu
                            + \frac{\gamma R}{(1-S)}
                            + \tfrac{1}{2} \sigma^2
                        \Big) dt
                    \Big]
                \\
                &amp; \,
                -
                J_{sa}(T)
                    \int_0^T
                            \frac{S(t) I_s(t)}{ E(t)}
                            d(\log (E_t))
                        -
                        J_{sa}(T)
                        \int_0^T
                            \frac{S(t) I_s(t)}{ E(t)}
                            \Big(
                                \mu
                                + \frac{1}{2}
                                \sigma^2
                            \Big) dt
                \\
                &
                -
                    \kappa
                    J_{sa}(T)
                        \int_0^T
                            \frac{S(t) I_s(t)}{ E(t)} dt
                           + J_{s}(T)
                    \int_0^T
                        \frac{S(t) I_a(t)}{ (1-S(t))}
                        \Big[
                            d\big(\log (1-S_t) \big)
                        \Big]
                \\
                &amp;\,
                    + J_{s}(T)
                    \int_0^T
                    \frac{S(t) I_a(t)}{ (1-S(t))}
                    \Big[
                        \Big(
                            \mu
                            + \frac{\gamma  R}{ (1-S)}
                            + \tfrac{1}{2} \sigma^2
                        \Big) dt
                    \Big]
                \\
                    &amp;
                    +
                    J_{s}(T)
                    \int_0^T
                        \frac{S(t) I_a(t)}{ E(t)}
                        d(\log (E_t))
                    +
                    J_{s}(T)
                    \int_0^T
                        \frac{S(t) I_a(t)}{ E(t)}
                            \Big(
                                \mu +
                                    \frac{1}{2} \sigma^2
                            \Big) dt   \,
                    \\
                    &
                    + \kappa
                        J_{s}(T)
                            \int_0^T
                            \frac{S(t) I_a(t)}{E(t)}
                        dt
                \Bigg].
            \end{aligned}
        \end{equation}
    </span>
        </p>
        <p>
            Moreover,
            <span class="math display">
    \begin{equation}
        \label{p_hat}
            \begin{aligned}
                \hat{p}_{ML}
                    = &amp;
                    \frac{1}{J_2(T)}
                    \Bigg[
                        \kappa^2
                        \int_0^T
                            \frac{E^2(t) }{I_S^2(t)}
                        dt
                \\
                &
                -
                        \kappa
                        \int_0^T
                            \frac{E(t) }{
                                I_S(t)
                            } d \big(\ln (I_S(t)) \big)
                            +
                            \kappa
                            \int_0^T
                            \dfrac{E(t)}{I_a(t)} d\big(\ln (I_a(t)) \big)
                    \\
                    &amp; -
                        \kappa (\alpha_S+\mu)
                        \int_0^T
                            \dfrac{E(t) }{I_S(t)}
                        dt
                        -
                        \dfrac{1}{2}\kappa \sigma^2
                        \int_0^T
                        \dfrac{E(t) }{I_S(t)} dt
                    \\
                        &amp;
                        + \kappa (\alpha_a+\mu)
                        \int_0^T \frac{E(t) }{I_a(t)} dt
                        +
                        \dfrac{1}{2}
                        \kappa \sigma^2
                        \int_0 ^ T
                        \dfrac{E(t)}{I_a(t)} dt
                        \Bigg].
            \end{aligned}
        \end{equation}
    </span>
        </p>
        
        
        
        <!--- Reference section--->
        <h1 id="bibliography">
            References
        </h1>
            <div class="references csl-bib-body hanging-indent"
                 id="refs"
                 role="doc-bibliography">
                <div
                        class="csl-entry"
                        id="ref-Acuna-Zegarra2021"
                        role="doc-biblioentry"
                >
                    Acuña-Zegarra, Manuel Adrian, Saúl Díaz-Infante, David Baca-Carrasco,
                    and Daniel Olmos-Liceaga.
                    <span>“
                        <span class="nocase">
                            COVID-19 optimal
                            vaccination policies: A modeling study on efficacy, natural and
                            vaccine-induced immunity responses
                        </span>.”
                    </span>
                    <em>
                        Mathematical Biosciences
                    </em>
                    337 (2021): 108614. doi:
                    <a href="https://doi.org/10.1016/j.mbs.2021.108614">
                        10.1016/j.mbs.2021.108614
                    </a>.
                </div>
                <div
                        class="csl-entry"
                        id="ref-Acunya2018"
                        role="doc-biblioentry"
                >
                    Acuña-Zegarra, Manuel Adrian, and Saúl Dı́az-Infante.
                    <span>
                    “Stochastic Asymptotic Analysis of a Multi-Host Model with Vector
                    Transmission.”
                </span>
                    <em>
                        Phys. A
                    </em> 510 (2018): 243–60. doi:
                    <a href="https://doi.org/10.1016/j.physa.2018.06.105">
                        10.1016/j.physa.2018.06.105
                    </a>.
                </div>
                <div
                        class="csl-entry"
                        id="ref-Acuna2020"
                        role="doc-biblioentry"
                >
                    Acuña-Zegarra,
                    Manuel Adrian,
                    Mario Santana-Cibrian,
                    and Jorge X Velasco-Hernandez.
                    <span>
                        “<span class="nocase">
                            Modeling behavioral
                            change and COVID-19 containment in Mexico: A trade-off between lockdown
                            and compliance
                        </span>.”
                    </span>
                    <em>
                        Mathematical Biosciences
                    </em> 325 (2020): 108370. doi:
                    <a href="https://doi.org/10.1016/j.mbs.2020.108370">
                        10.1016/j.mbs.2020.108370
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Allen2007" role="doc-biblioentry">
                    Allen, E. <em>Modeling with <span>I</span>tô Stochastic Differential
                    Equations</em>. Vol. 22. Mathematical Modelling: Theory and
                    Applications. Springer, Dordrecht, 2007.
                </div>
                <div class="csl-entry" id="ref-Anderson" role="doc-biblioentry">
                    Anderson, R. M., and R. M. May. <em>Infectious Diseases of Humans</em>.
                    Oxford, UK: Oxford University Press, 1991. doi:<a
                        href="https://doi.org/10.1002/hep.1840150131">10.1002/hep.1840150131</a>.
                </div>
                <div class="csl-entry" id="ref-Betancourt2017" role="doc-biblioentry">
                    Betancourt, Michael.
                    <span>
                        “A Conceptual Introduction to Hamiltonian
                        Monte Carlo.
                        ”
                    </span>
                    <em>arXiv</em>,
                    January 10, 2017.
                    <a href="https://arxiv.org/abs/1701.02434v1">
                        https://arxiv.org/abs/1701.02434v1
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Bishwal2008" role="doc-biblioentry">
                    Bishwal, Jaya P. N. <em>Parameter Estimation in Stochastic Differential
                    Equations</em>. Vol. 1923. Lecture Notes in Mathematics. Springer,
                    Berlin, 2008. doi:
                    <a href="https://doi.org/10.1007/978-3-540-74448-1">
                        10.1007/978-3-540-74448-1
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Bragazzi2018" role="doc-biblioentry">
                    Bragazzi, N. L., V. Giangredi, M. Villarini, R. Roselli, A. Nasr, A.
                    Hussein, M. Martini, and M. Behzadifar.
                    <span>
                        “<span class="nocase">
                            vaccines Meet Big Data: State-ofthe-Art and Future
                            Prospects. From the Classical 3Is (’Isolate–Inactivate–Inject’)
                            vaccinology 1.0 to vaccinology 3.0, vaccinomics, and Beyond: A
                            Historical Overview
                        </span>.”
                    </span>
                    <em>
                        Frontiers of Public Health
                    </em> 6, no. 62 (2018): 1–9.
                    doi:
                    <a href="https://doi.org/10.3389/fpubh.2018.00062">
                        10.3389/fpubh.2018.00062
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Cai2019" role="doc-biblioentry">
                    Cai, Siyang, Yongmei Cai, and Xuerong Mao.
                    <span>
                        “A Stochastic
                        Differential Equation SIS Epidemic Model with Two Independent Brownian
                        Motions.”
                    </span>
                    <em>
                        Journal of Mathematical Analysis and
                        Applications
                    </em> 474, no. 2 (2019): 1536–50.
                    doi:
                    <a href="https://doi.org/10.1016/j.jmaa.2019.02.039">
                        https://doi.org/10.1016/j.jmaa.2019.02.039
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Chang2019" role="doc-biblioentry">
                    Chang, Zhengbo, Xinzhu Meng, and Tonghua Zhang.
                    <span>
                        “A New Way of
                        Investigating the Asymptotic Behaviour of a Stochastic SIS System with
                        Multiplicative Noise.”
                    </span>
                    <em>
                        Applied Mathematics Letters
                    </em>
                    87 (2019): 80–86.
                    doi:
                    <a href="https://doi.org/10.1016/j.aml.2018.07.014">
                        https://doi.org/10.1016/j.aml.2018.07.014
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Chatzilena2019" role="doc-biblioentry">
                    Chatzilena, Anastasia, Edwin van Leeuwen, Oliver Ratmann, Marc Baguelin,
                    and Nikolaos Demiris.
                    <span>
                    “
                        Contemporary Statistical Inference for
                        Infectious Disease Models Using Stan.
                    ”
                    </span>
                    <em>
                        Epidemics
                    </em> 29 (2019): 100367.
                    doi:
                    <a href="https://doi.org/10.1016/j.epidem.2019.100367">
                        https://doi.org/10.1016/j.epidem.2019.100367
                    </a>.
                </div>
                <div class="csl-entry" id="ref-CialencoGongHuang2016"
                     role="doc-biblioentry">
                    Cialenco, Igor, Ruoting Gong, and Yicong Huang.
                    <span>
                        “Trajectory
                        Fitting Estimators for <span>SPDE</span>s Driven by Additive
                        Noise.”
                    </span>
                    <em>
                        Stat. Inference Stoch. Process.
                    </em> 21, no. 1
                    (2018): 1–19.
                    doi:
                    <a href="https://doi.org/10.1007/s11203-016-9152-2">
                        10.1007/s11203-016-9152-2
                    </a>.
                </div>
                <div class="csl-entry" id="ref-IgorSergeyJan2008"
                     role="doc-biblioentry">
                    Cialenco, Igor, Sergey V. Lototsky, Jan Pospı́šil, and Jan Pospı́šil.
                    <span>
                        “Asymptotic Properties of the Maximum Likelihood Estimator for
                        Stochastic Parabolic Equations with Additive Fractional
                        Brownian Motion.”
                    </span>
                    <em>
                        Stoch. Dyn.
                    </em> 9, no. 2
                    (2009): 169–85.
                    doi:
                    <a href="https://doi.org/10.1142/S0219493709002610">
                        10.1142/S0219493709002610
                    </a>.
                </div>
                <div class="csl-entry" id="ref-faranda2020modeling"
                     role="doc-biblioentry">
                    Faranda, Davide, and Tommaso Alberti.
                    <span>
                        “Modeling the Second Wave of
                        COVID-19 Infections in France and Italy via a Stochastic SEIR
                        Model.”
                    </span>
                    <em>
                        Chaos: An Interdisciplinary Journal of Nonlinear
                        Science
                    </em> 30, no. 11 (2020): 111101.
                </div>
                <div class="csl-entry" id="ref-Fa-Al20" role="doc-biblioentry">
                    ———.
                    <span>
                        “Modeling the Second Wave of
                        COVID-19 Infections France and Italy via a Stochastic
                        SEIR Model.”
                    </span>
                    <em>
                        Chaos
                    </em> 30, no. 11 (2020):
                    111101, 13.
                    doi:
                    <a href="https://doi.org/10.1063/5.0015943">
                        10.1063/5.0015943
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Fintzi2017" role="doc-biblioentry">
                    Fintzi, Jonathan, Xiang Cui, Jon Wakefield, and Vladimir N. Minin.
                    <span>
                    “Efficient Data Augmentation for Fitting Stochastic Epidemic
                    Models to Prevalence Data.”
                </span>
                    <em>
                        Journal of Computational and
                        Graphical Statistics
                    </em> 26, no. 4 (2017): 918–29.
                    doi:
                    <a href="https://doi.org/10.1080/10618600.2017.1328365">
                        10.1080/10618600.2017.1328365
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Gardiner2009" role="doc-biblioentry">
                    Gardiner, Crispin.
                    <em>
                        Stochastic Methods
                    </em>. Fourth. Springer Series
                    in Synergetics. Springer-Verlag, Berlin, 2009.
                </div>
                <div class="csl-entry" id="ref-DataMX" role="doc-biblioentry">
                    Gobierno de México. <span>“Datos Abiertos.”</span> Edited by Gobierno de
                    México.
                    <a class="uri" href="https://www.gob.mx/salud/documentos/datos-abiertos-152127">
                        https://www.gob.mx/salud/documentos/datos-abiertos-152127</a>;
                    <span>
                        Gobierno de México
                    </span>,
                    n.d.
                </div>
                <div class="csl-entry" id="ref-GrGrHuMa-11" role="doc-biblioentry">
                    Gray, A., D. Greenhalgh, L. Hu, X. Mao, and J. Pan.
                    <span>
                        “A Stochastic Differential Equation SIS Epidemic Model.”
                    </span>
                    <em>
                        SIAM J. Appl. Math.
                    </em> 71, no. 3 (2011): 876–902.
                    doi:
                    <a href="https://doi.org/10.1137/10081856X">
                        10.1137/10081856X
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Groendyke2021" role="doc-biblioentry">
                    Groendyke, Chris, and Adam Combs.
                    <span>“
                        <span class="nocase">
                            Modifying the network-based stochastic SEIR model to account for quarantine: an
                            application to COVID-19
                        </span>.”
                    </span>
                        <em>
                            Epidemiologic Methods
                        </em>
                        10, no. s1 (2021): 20200030.
                        doi:
                    <a href="https://doi.org/10.1515/em-2020-0030">
                        10.1515/em-2020-0030
                    </a>.
                </div>
                <div class="csl-entry" id="ref-GroendykeCombs2021"
                     role="doc-biblioentry">
                    ———.
                    <span>
                    “Modifying the Network-Based Stochastic SEIR Model to Account
                    for Quarantine: An Application to COVID-19:”
                        </span>
                        <em>
                            Epidemiologic Methods
                        </em> 10, no. s1 (2021): 20200030.
                        doi:
                        <a href="https://doi.org/doi:10.1515/em-2020-0030">
                            doi:10.1515/em-2020-0030
                        </a>.
                </div>
                <div class="csl-entry" id="ref-Kloeden2017" role="doc-biblioentry">
                    Han, Xiaoying, and Peter E. Kloeden.
                    <em>
                        Random Ordinary Differential
                        Equations and Their Numerical Solution
                    </em>. Vol. 85.
                    Probability Theory and Stochastic Modelling. Springer, Singapore, 2017.
                    doi:
                    <a href="https://doi.org/10.1007/978-981-10-6265-0">
                        10.1007/978-981-10-6265-0
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Hoffman2014" role="doc-biblioentry">
                    Hoffman, Matthew D., and Andrew Gelman.
                    <span>
                        “TheNo-<span>U</span>-Turn Sampler: Adaptively Setting Path Lengths in
                        Hamiltonian Monte Carlo.”
                    </span>
                    <em>
                        J. Mach. Learn. Res.
                    </em> 15 (2014): 1593–623.
                </div>
                <div class="csl-entry" id="ref-Hotta" role="doc-biblioentry">
                    Hotta, Luiz K.
                    <span>
                        “Bayesian Melding Estimation of a Stochastic SEIR
                        Model.”
                    </span>
                    <em>
                        Mathematical Population Studies
                    </em>, no. 17 (2010): 101, 111.
                    doi:
                    <a href="https://doi.org/10.1080/08898481003689528">
                        10.1080/08898481003689528
                    </a>.
                </div>
                <div class="csl-entry" id="ref-iacus" role="doc-biblioentry">
                    Iacus, Stefano M.
                    <em>
                        Simulation and Inference for Stochastic
                        Differential Equations
                    </em>.
                    Springer Series in Statistics. Springer, New York, 2008.
                    doi:
                    <a href="https://doi.org/10.1007/978-0-387-75839-8">
                        10.1007/978-0-387-75839-8
                    </a>.
                </div>
                <div class="csl-entry" id="ref-JSh" role="doc-biblioentry">
                    Jacod, Jean, and Albert N. Shiryaev.
                    <em>
                        Limit Theorems for Stochastic Processes
                    </em>.
                    Second. Vol. 288. Grundlehren Der Mathematischen
                    Wissenschaften [Fundamental Principles of Mathematical Sciences].
                    Springer-Verlag, Berlin, 2003.
                    doi:
                    <a href="https://doi.org/10.1007/978-3-662-05265-5">
                        10.1007/978-3-662-05265-5
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Kloeden1992" role="doc-biblioentry">
                    Kloeden, Peter E., and Eckhard Platen.
                    <em>
                        Numerical Solution of Stochastic Differential Equations
                    </em>.
                    Vol. 23. Applications of Mathematics (New York). Springer-Verlag, Berlin, 1992.
                    doi:
                    <a href="https://doi.org/10.1007/978-3-662-12616-5">
                        10.1007/978-3-662-12616-5
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Lekone2006" role="doc-biblioentry">
                    Lekone, Phenyo E., and Bärbel F. Finkenstädt.
                    <span>“
                        <span class="nocase">
                            Statistical Inference in a Stochastic Epidemic SEIR
                            Model with Control Intervention: Ebola as a Case Study
                        </span>.”
                    </span>
                    <em>
                        Biometrics
                    </em> 62, no. 4 (2006): 1170–77.
                    doi:
                    <a href="https://doi.org/10.1111/j.1541-0420.2006.00609.x">
                        10.1111/j.1541-0420.2006.00609.x
                    </a>.
                </div>
                <div class="csl-entry" id="ref-LSh3" role="doc-biblioentry">
                    Liptser, R. Sh., and A. N. Shiryayev.
                    <em>
                        Theory of Martingales
                    </em>.
                    Vol. 49. Mathematics and Its Applications (Soviet Series).
                    Kluwer Academic Publishers Group, Dordrecht, 1989.
                    doi:
                    <a href="https://doi.org/10.1007/978-94-009-2438-3">
                        10.1007/978-94-009-2438-3
                    </a>.
                </div>
                <div class="csl-entry" id="ref-LSh1" role="doc-biblioentry">
                    Liptser, Robert S., and Albert N. Shiryaev.
                    <em>
                        Statistics of Random Processes. I. Expanded. Vol. 5. Applications of
                        Mathematics (New York). Springer-Verlag, Berlin, 2001.
                    </em>
                </div>
                <div class="csl-entry" id="ref-Liu" role="doc-biblioentry">
                    Liu, G.
                    <em>
                        Statistical Inference for Multivariate Stochastic
                        Differential Equations
                    </em>. ProQuest Dissertations Publishing, 2019.
                </div>
                <div class="csl-entry" id="ref-Liu2018" role="doc-biblioentry">
                    Liu, Qun, Daqing Jiang, Tasawar Hayat, and Ahmed Alsaedi.
                    <span>
                        “Stationary Distribution and Extinction of a Stochastic Dengue
                        Epidemic Model.”
                    </span>
                    <em>
                        J. Franklin Inst.
                    </em> 355, no. 17 (2018): 8891–8914.
                    doi:
                    <a href="https://doi.org/10.1016/j.jfranklin.2018.10.003">
                        10.1016/j.jfranklin.2018.10.003
                    </a>.
                </div>
                <div class="csl-entry" id="ref-mao2007stochastic"
                     role="doc-biblioentry">
                    Mao, Xuerong. <em>Stochastic Differential Equations and
                    Applications</em>. Elsevier, 2007.
                </div>
                <div class="csl-entry" id="ref-Wei2006" role="doc-biblioentry">
                    Miao, Wei-Cheng.
                    <span>
                        “Estimation of Diffusion Parameters in Diffusion
                        Processes and Their Asymptotic Normality.”
                    </span>
                    <em>
                        Int. J. Contemp. Math. Sci.
                    </em> 1, no. 13–16 (2006): 763–76.
                    doi:
                    <a href="https://doi.org/10.12988/ijcms.2006.06076">
                        10.12988/ijcms.2006.06076
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Ndanguza" role="doc-biblioentry">
                    Ndanguza, D., I. S. Mbalawata, and Nsabimana J. P.
                    <span>
                        “Analysis of SDEs Applied to SEIR Epidemic
                        Models by Extended Kalman Filter Method.”
                    </span>
                    <em>
                        Applied Mathematics
                    </em>, no. 7 (2011): 2195, 2211.
                    doi:
                    <a href="https://doi.org/10.4236/am.2016.717175">
                        10.4236/am.2016.717175
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Nualart2006" role="doc-biblioentry">
                    Nualart, David.
                    <em>
                        The Malliavin Calculus and Related Topics
                    </em>. Second. Probability and Its Applications (New York).
                    Springer-Verlag, Berlin, 2006.
                </div>
                <div class="csl-entry" id="ref-ONeill2010" role="doc-biblioentry">
                    O’Neill, Philip D.
                    <span>
                        “<span class="nocase">
                            Introduction and snapshot review: Relating infectious disease
                            transmission models to data
                        </span>.”
                    </span>
                    <em>
                        Statistics in Medicine
                    </em> 29, no. 20 (2010): 2069–77.
                    doi:
                    <a href="https://doi.org/10.1002/sim.3968">
                        10.1002/sim.3968
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Oksendal2003" role="doc-biblioentry">
                    Øksendal, Bernt.
                    <em>
                        Stochastic Differential Equations
                    </em>.
                    Sixth. Universitext. Springer-Verlag, Berlin, 2003.
                    doi:
                    <a href="https://doi.org/10.1007/978-3-642-14394-6">
                        10.1007/978-3-642-14394-6
                    </a>.
                </div>
                <div class="csl-entry" id="ref-djordjevic2021two"
                     role="doc-biblioentry">
                    Dorević, J, I Papić, and N Šuvak.
                    <span>
                        “ A Two Diffusion Stochastic Model
                        for the Spread of the New Corona Virus SARS-CoV-2.”
                    </span>
                    <em>
                        Chaos, Solitons &amp; Fractals
                    </em> 148 (2021): 110991.
                </div>
                <div class="csl-entry" id="ref-Otunga2021" role="doc-biblioentry">
                    Otunuga, Olusegun M.
                    <span>“
                    <span class="nocase">
                        Estimation of epidemiological parameters
                        for COVID-19 cases using a stochastic SEIRS
                        epidemic model with vital dynamics
                    </span>.
                    ”</span>
                    <em>
                        Results in Physics
                    </em>, September 2021, 104664.
                    doi:
                    <a href="https://doi.org/10.1016/j.rinp.2021.104664">
                        10.1016/j.rinp.2021.104664
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Otunuga2021" role="doc-biblioentry">
                    ———.
                    <span>
                        “<span class="nocase">
                            Estimation of epidemiological
                            parameters for COVID-19 cases using a stochastic SEIRS epidemic
                            model with vital dynamics
                        </span>.”
                    </span>
                    <em>
                        Results in Physics
                    </em>,
                    September 2021, 104664.
                    doi:
                    <a href="https://doi.org/10.1016/j.rinp.2021.104664">
                        10.1016/j.rinp.2021.104664
                    </a>.
                </div>
                <div class="csl-entry" id="ref-PaGrGrMa" role="doc-biblioentry">
                    Pan, Jiafeng, Alison Gray, David Greenhalgh, and Xuerong Mao.
                    <span>
                        “Parameter Estimation for the Stochastic SIS Epidemic
                        Model.”
                    </span>
                        <em>
                            Stat. Inference Stoch. Process.
                        </em>
                        17, no. 1 (2014): 75–98.
                        doi:
                        <a href="https://doi.org/10.1007/s11203-014-9091-8">
                            10.1007/s11203-014-9091-8
                        </a>.
                </div>
                <div class="csl-entry" id="ref-PrakasaRao2003" role="doc-biblioentry">
                    Prakasa Rao, B. L. S.
                    <span>
                        “Estimation for Some Stochastic Partial
                        Differential Equations Based on Discrete Observations.
                        II.”
                    </span>
                    <em>
                        Calcutta Statist. Assoc. Bull.
                    </em>
                    54, no. 215–216 (2003): 129–41.
                    doi:
                    <a href="https://doi.org/10.1177/0008068320030301">
                        10.1177/0008068320030301
                    </a>.
                </div>
                <div class="csl-entry" id="ref-PrakasaRao2002" role="doc-biblioentry">
                    ———.
                    <span>
                        “Nonparametric Inference for a Class of Stochastic Partial
                        Differential Equations Based on Discrete Observations.”</span>
                    <em>
                        Sankhyā Ser. A
                    </em> 64, no. 1 (2002): 1–15.
                </div>
                <div class="csl-entry" id="ref-Rios" role="doc-biblioentry">
                    Ríos-Gutiérrez, A., V. Arunachalam, and A. Mubayi.
                    <span>
                        “Stochastic Anlysis and Statistical Inference for SEIR Models of Infectious
                        Diseases.”
                    </span>
                    <em>Preprint</em>, 2020.
                </div>
                <div class="csl-entry" id="ref-saso19" role="doc-biblioentry">
                    Särkkä, Simo, and Arno Solin. <em>Applied Stochastic Differential
                    Equations</em>. Vol. 10. Institute of Mathematical Statistics Textbooks.
                    Cambridge University Press, Cambridge, 2019.
                </div>
                <div class="csl-entry" id="ref-Schurz2015" role="doc-biblioentry">
                    Schurz, Henri, and Kursad Tosun.
                    <span>
                        “<span class="nocase">
                            Stochastic Asymptotic Stability of SIR Model with Variable Diffusion
                            Rates
                        </span>.”
                    </span>
                    <em>
                        Journal of Dynamics and Differential
                        Equations
                    </em> 27, no. 1 (2015): 69–82.
                    doi:
                    <a href="https://doi.org/10.1007/s10884-014-9415-9">
                        10.1007/s10884-014-9415-9
                    </a>.
                </div>
                <div class="csl-entry" id="ref-ShiryaevBookProbability"
                     role="doc-biblioentry">
                    Shiryaev, A. N. <em>Probability</em>. Second. Vol. 95. Graduate Texts in
                    Mathematics. Springer-Verlag, New York, 1996. doi:<a
                        href="https://doi.org/10.1007/978-1-4757-2539-1">10.1007/978-1-4757-2539-1</a>.
                </div>
                <div class="csl-entry" id="ref-soize2017uncertainty"
                     role="doc-biblioentry">
                    Soize, Christian. <em>Uncertainty Quantification</em>. Springer, 2017.
                </div>
                <div class="csl-entry" id="ref-Soren-04" role="doc-biblioentry">
                    Sørensen, H.
                    <span>
                        “Parametric Inference for Diffusion Processes
                        Observed at Discrete Points in Time: A Survey.”
                    </span>
                    <em>
                        Int. Stat. Rev.
                    </em> 72, no. 3 (2004): 337, 354.
                    doi:
                    <a href="https://doi.org/10.1111/j.1751-5823.2004">
                        10.1111/j.1751-5823.2004
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Tian2020" role="doc-biblioentry">
                    Tian, Huaiyu, Yonghong Liu, Yidan Li, Chieh-Hsi Wu, Bin Chen, Moritz UG
                    Kraemer, Bingying Li, et al.
                    <span>
                        “<span class="nocase">
                            An investigation of transmission control measures during the first 50 days
                            of the COVID-19 epidemic in China
                        </span>.”
                    </span>
                    <em>
                        Science
                    </em> 368, no. 6491 (2020): 638–42.
                    doi:
                    <a href="https://doi.org/10.1126/science.abb6105">
                        10.1126/science.abb6105
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Yang2014" role="doc-biblioentry">
                    Yang, Qingshan, and Xuerong Mao.
                    <span>
                        “Stochastic Dynamics of SIRS Epidemic Models Withrandom Perturbation.”
                    </span>
                    <em>
                        Mathematical Biosciences and Engineering
                    </em> 11, no. 4 (2014): 1003–25.
                    doi:
                    <a href="https://doi.org/10.3934/mbe.2014.11.1003">
                        10.3934/mbe.2014.11.1003
                    </a>.
                </div>
                <div class="csl-entry" id="ref-Young81" role="doc-biblioentry">
                    Young, P.
                    <span>
                        “Parameter Estimation for Continuous-Time Models: A
                        Survey.”
                    </span>
                    <em>
                        Automatica
                    </em>
                    17, no. 1 (1981): 23, 29.
                    doi:
                    <a href="https://doi.org/10.1016/0005-1098(81)90082-0">
                        10.1016/0005-1098(81)90082-0
                    </a>.
                </div>
            </div>
        <script>
            // add bootstrap table styles to pandoc tables
            function bootstrapStylePandocTables() {
                $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
            }
            
            $(document).ready(function () {
                bootstrapStylePandocTables();
            });
        
        
        </script>
        
        <script>
            $(document).ready(function () {
                window.buildTabsets("TOC");
            });
            
            $(document).ready(function () {
                $('.tabset-dropdown > .nav-tabs > li').click(function () {
                    $(this).parent().toggleClass('nav-tabs-open');
                });
            });
        </script>
        <script>
            (function () {
                var script = document.createElement("script");
                script.type = "text/javascript";
                //script.src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
                script.src = "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js";
                document.getElementsByTagName("head")[0].appendChild(script);
            })();
        </script>
        <script>
            MathJax = {
                loader: {load: ['[tex]/textmacros']},
                tex: {
                    inlineMath: [['$', '$'], ['\\(', '\\)']],
                    tags: 'ams',
                    packages: {'[+]': ['textmacros']},
                    macros: {
                        oe: '\u0153',
                        OE: '\u0152',
                        ae: '\u00E6',
                        AE: '\u00C6',
                        aa: '\u00E5',
                        AA: '\u00C5',
                        o: '\u00F8',
                        O: '\u00D8',
                        l: '\u0142',
                        L: '\u0141',
                        ss: '\u00DF',
                        SS: '\u1E9E',
                        dag: '\u2020',
                        ddag: '\u2021',
                        P: '\u00B6',
                        copyright: '\u00A9',
                        pounds: '\u00A3'
                    }
                },
                chtml: {
                    mtextInheritFont: true
                },
                sag: {
                    mtextInheritFont: true
                },
                svg: {fontCache: 'global'}
            };
        </script>
    </body>
</html>
