





  
            <!DOCTYPE html>
<html lang="en">
  <!-- [INFO   ] [j1.layout.default.html                ] [ page generated: 2022-07-14 02:11:15 +0200 ] -->
  <!-- [INFO   ] [j1.layout.default.html                ] [ environment detected as: development ] -->
  <!-- [INFO   ] [j1.layout.default.html                ] [ version detected as: 2022.4.4 ] -->
  <head data-proofer-ignore>
    <!-- [INFO   ] [j1.layout.default.html                ] [ call default_writer for region: 'head' ] -->
    <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ start processing: load region head, layout: page ] -->
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ start processing: place meta tags (taken from site config|page frontmatter) ] -->
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ page|content encoding ] -->
    <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ added for MS Edge Browser ] -->
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ initial zoom level when the page is first loaded by the browser ] -->
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ added version info for the templte ] -->
    <meta name="template version" content="2022.4.4">
    <title>a study based on mathematical analysis and simulation</title>
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ detected content type: PAGE ] -->
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ SIZE of page description: 36 chars ] -->
    <!-- [SEO    ] [j1.layout.meta_data_generator.html    ] [ Consider to IMPROVE|INCREASE your page description! ] -->
    <meta name="description" content="TODO: Rewrite
">
    <link rel="icon" type="image/ico" href="/assets/images/modules/icons/j1/j1-32x32.ico">
    <meta name="author" content="J1 Team">
    <meta name="keywords" content="opensource, free, load, download, start, starter, example, high, easy, use, secure, encrypt, standard, popular, generate, create, learn, distribute, publish, deploy, beginner, advanced, expert, student, learner, writer, reader, visitor framework, toolkit, integration, extension, module, api, dynamic, static, generator, client, server, internet, local, localhost page, web, website, webdesign, material, design, responsive, javascript, nodejs, ruby, windows, linux, osx, mac, os, http, https, html, html5, css, scss, style, browser, firefox, chrome, edge, opera, safari, configuration, generator, navigation, menu, dropdown, fab, action, button application, interface, provider, api, repository, cookie, language, translation, gdpr, dsgvo, privacy, asciidoc, aciidoctor, bootstrap, jekyll, liquid, hyvor, disqus, git, github, netlify, heroku, apple, microsoft, provider, service, internet, support, google, analytics, advertising, search, console, silverlight, score, j1, template, jekyllone, comment
">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ Check robot settings from SITE configuration ] -->
    <!-- [DEBUG  ] [j1.layout.meta_data_generator.html    ] [ Found SITE configuration robots_index: true ] -->
    <!-- [DEBUG  ] [j1.layout.meta_data_generator.html    ] [ Found SITE configuration robots_follow: true ] -->
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ Check robot settings from PAGE configuration ] -->
    <!-- [DEBUG  ] [j1.layout.meta_data_generator.html    ] [ Found PAGE configuration robots_index: false ] -->
    <!-- [DEBUG  ] [j1.layout.meta_data_generator.html    ] [ Found PAGE configuration robots_follow: false ] -->
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ Place detected/overloaded meta tags for robots ] -->
    <meta name="robots" content="noindex">
    <meta name="robots" content="nofollow">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ meta tag for provider APIs: analytics ] -->
    <meta name="analytics" content="false">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ meta tag for provider APIs: comments ] -->
    <meta name="comments" content="false">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ meta tag for provider APIs: advertising ] -->
    <meta name="advertising" content="false">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ meta tag for provider APIs: advertising ] -->
    <meta name="translation" content="false">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ meta tag for provider APIs: youtube ] -->
    <meta name="youtube" content="true">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ meta tag for provider APIs: vimeo ] -->
    <meta name="vimeo" content="true">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ meta tag for privacy settings: personalization ] -->
    <meta name="personalization" content="false">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ PRELOAD of fonts: disabled ] -->
    <!-- [SEO    ] [j1.layout.meta_data_generator.html    ] [ Consider! PRELOAD primary used fonts to prioritize fetching ] -->
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ Gratulations! PRELOAD of css: enabled ] -->
    <!-- [SEO    ] [j1.layout.meta_data_generator.html    ] [ request PRELOAD of font to prioritize fetching ] -->
    <link rel="preload" as="style" href="/assets/themes/j1/core/css/themes/unolight/bootstrap.css">
    <link rel="preload" as="style" href="/assets/themes/j1/core/css/vendor.css">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ Gratulations! PRELOAD of js: enabled ] -->
    <!-- [SEO    ] [j1.layout.meta_data_generator.html    ] [ request PRELOAD of font to prioritize fetching ] -->
    <link rel="preload" as="script" href="/assets/themes/j1/modules/bmd/js/bmd.js">
    <link rel="preload" as="script" href="/assets/themes/j1/modules/backstretch/js/backstretch.js">
    <link rel="preload" as="script" href="/assets/themes/j1/core/js/template.js">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ PRELOAD js as FILE ] -->
    <link rel="preload" as="script" href="/assets/themes/j1/adapter/js/attic.js">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ PRELOAD js as FILE ] -->
    <link rel="preload" as="script" href="/assets/themes/j1/adapter/js/logger.js">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ PRELOAD js as FILE ] -->
    <link rel="preload" as="script" href="/assets/themes/j1/adapter/js/bmd.js">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ PRELOAD js as FILE ] -->
    <link rel="preload" as="script" href="/assets/themes/j1/adapter/js/navigator.js">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ PRELOAD of images: disabled ] -->
    <!-- [SEO    ] [j1.layout.meta_data_generator.html    ] [ Consider! PRELOAD primary|globally used image files to prioritize fetching ] -->
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ Gratulations! PRECONNECT of remote URLs: enabled ] -->
    <!-- [SEO    ] [j1.layout.meta_data_generator.html    ] [ request PRECONNECT of 3rd party origins to speed up remote connections ] -->
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ establish early connection: https://bootswatch.com ] -->
    <link rel="dns-prefetch" href="https://bootswatch.com">
    <link rel="preconnect" href="https://bootswatch.com">
    <!-- [INFO   ] [j1.layout.meta_data_generator.html    ] [ end processing ] -->
    <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ end processing: load region head, layout: page ] -->
    <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ start processing: load region head, layout: page ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ start processing region: head, layout: page ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: core ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, template core ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place css files for resource ID: core ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/spinner/css/spin.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, template core ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place css files for resource ID: core ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/core/css/icon-fonts/mdi.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, template core ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place css files for resource ID: core ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/core/css/icon-fonts/mdil.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, template core ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place css files for resource ID: core ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/core/css/icon-fonts/fontawesome.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, template core ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place css files for resource ID: core ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/core/css/themes/unolight/bootstrap.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, template core ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place css files for resource ID: core ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/core/css/vendor.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, template core ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: core ] -->
    <script sync src="/assets/themes/j1/adapter/js/j1.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all template core ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: core ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script sync src="/assets/themes/j1/modules/spinner/js/spin.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all template core ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: core ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script sync src="/assets/themes/j1/modules/jquery/js/jquery.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all template core ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: core ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script sync src="/assets/themes/j1/modules/popper/js/popper.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all template core ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: core ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script sync src="/assets/themes/j1/core/js/template.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: ga-opt-in ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, gaOptIn ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: ga-opt-in ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all gaOptIn ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: ga-opt-in ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script defer src="/assets/themes/j1/modules/gtag-opt-in/js/gtag-opt-in.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: jquery-ui ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: logger ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, module logger ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: logger ] -->
    <script sync src="/assets/themes/j1/adapter/js/logger.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all module logger ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: logger ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script sync src="/assets/themes/j1/modules/log4javascript/js/log4javascript.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: backstretch ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, Backstretch ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: backstretch ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all Backstretch ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: backstretch ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script sync src="/assets/themes/j1/modules/backstretch/js/backstretch.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: attic ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, module attic ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: attic ] -->
    <script sync src="/assets/themes/j1/adapter/js/attic.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: navigator ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, module navigator ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: navigator ] -->
    <script sync src="/assets/themes/j1/adapter/js/navigator.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: mmenu ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, module mmenu ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place css files for resource ID: mmenu ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/mmenuLight/css/mmenu-light.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, module mmenu ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place css files for resource ID: mmenu ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/mmenuLight/css/theme/uno/mmenu.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, module mmenu ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: mmenu ] -->
    <script sync src="/assets/themes/j1/adapter/js/mmenu.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all module mmenu ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: mmenu ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script sync src="/assets/themes/j1/modules/mmenuLight/js/mmenu.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: themer ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, module themer ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: themer ] -->
    <script sync src="/assets/themes/j1/adapter/js/themer.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: toccer ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: blog_archive, module toccer ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: blog_archive, place general file resources for resource_settings.id: toccer ] -->
    <script sync src="/assets/themes/j1/adapter/js/toccer.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: cookieConsent ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, module cookieConsent ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: cookieConsent ] -->
    <script sync src="/assets/themes/j1/adapter/js/cookieConsent.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all module cookieConsent ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: cookieConsent ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script sync src="/assets/themes/j1/modules/cookieConsent/js/cookieConsent.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: jq_mouse_wheel ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: theme_switcher ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, module theme_switcher ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: theme_switcher ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all module theme_switcher ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: theme_switcher ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script sync src="/assets/themes/j1/modules/themeSwitcher/js/switcher.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: bs5 ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, Bootstrap JS ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: bs5 ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all Bootstrap JS ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: bs5 ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script sync src="/assets/themes/j1/modules/bootstrap/js/bootstrap.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: bmd ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, Bootstrap Material Design ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: bmd ] -->
    <script sync src="/assets/themes/j1/adapter/js/bmd.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all Bootstrap Material Design ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: bmd ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script sync src="/assets/themes/j1/modules/bmd/js/bmd.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: materialze ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: rtable ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, module rtable ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place css files for resource ID: rtable ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/rtable/css/theme/uno/rtable.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, module rtable ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: rtable ] -->
    <script sync src="/assets/themes/j1/adapter/js/rtable.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all module rtable ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: rtable ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script sync src="/assets/themes/j1/modules/rtable/js/rtable.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: asciidoctor ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, HTML writer for Asciidoctor ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, place general file resources for resource_settings.id: asciidoctor ] -->
    <script defer src="/assets/themes/j1/adapter/js/asciidoctor.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: css_flags_previewer ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, module css_flags_previewer ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place css files for resource ID: css_flags_previewer ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/core/country-flags/css/theme/uno.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, module css_flags_previewer ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: css_flags_previewer ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: fabm ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, FAB Menus ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: fabm ] -->
    <script sync src="/assets/themes/j1/modules/fab/js/cash.js"></script>
    <script sync src="/assets/themes/j1/modules/fab/js/fab.js"></script>
    <script sync src="/assets/themes/j1/adapter/js/fab.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: lunr ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, J1 Lunr ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place css files for resource ID: lunr ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/lunr/css/j1.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, J1 Lunr ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: lunr ] -->
    <script defer src="/assets/themes/j1/adapter/js/lunr.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all J1 Lunr ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: lunr ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script defer src="/assets/themes/j1/modules/lunr/js/lunr.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all J1 Lunr ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: lunr ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script defer src="/assets/themes/j1/modules/lunr/js/mustache.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all J1 Lunr ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: lunr ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script defer src="/assets/themes/j1/modules/lunr/js/dateformat.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all J1 Lunr ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: lunr ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script defer src="/assets/themes/j1/modules/lunr/js/uri.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all J1 Lunr ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: lunr ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script defer src="/assets/themes/j1/modules/lunr/js/j1.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: rtextresizer ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: msDropdown ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, module msDropdown ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: msDropdown ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/msDropdown/css/themes/uno/msDropdown.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all module msDropdown ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: msDropdown ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script async src="/assets/themes/j1/modules/msDropdown/js/msDropdown.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: translator ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, module translator ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place css files for resource ID: translator ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/translator/css/translator.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, module translator ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: translator ] -->
    <script sync src="/assets/themes/j1/adapter/js/translator.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all module translator ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout all, place js files for resource ID: translator ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script sync src="/assets/themes/j1/modules/translator/js/translator.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: analytics ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, module analytics ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: analytics ] -->
    <script sync src="/assets/themes/j1/adapter/js/analytics.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: CustomModule ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: customFunctions ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: rouge ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, theme rouge ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, place general file resources for resource_settings.id: rouge ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/rouge/css/base16/theme.css">
    <script defer src="/assets/themes/j1/adapter/js/rouge.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: animate ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, Animate ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place css files for resource ID: animate ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/core/css/animate.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, Animate ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: all, place general file resources for resource_settings.id: animate ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: justifiedGalleryCustomizer ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: framer ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: lightbox ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: blog_archive, module lightbox ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: blog_archive, place css files for resource ID: lightbox ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/lightbox/css/lightbox.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: blog_archive, module lightbox ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: blog_archive, place css files for resource ID: lightbox ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/lightbox/css/theme/uno.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: blog_archive, module lightbox ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: blog_archive, place general file resources for resource_settings.id: lightbox ] -->
    <script defer src="/assets/themes/j1/adapter/js/lightbox.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: blog_archive module lightbox ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout blog_archive, place js files for resource ID: lightbox ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script defer src="/assets/themes/j1/modules/lightbox/js/lightbox.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: twemoji ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: twemoji_picker ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: mdi_previewer ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: mdil_previewer ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: iconify ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: algolia ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: clipboard ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: collection, module clipboard ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: collection, place css files for resource ID: clipboard ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/clipboard/css/theme/uno.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: collection, module clipboard ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: collection, place general file resources for resource_settings.id: clipboard ] -->
    <script defer src="/assets/themes/j1/adapter/js/clipboard.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: collection module clipboard ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout collection, place js files for resource ID: clipboard ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script defer src="/assets/themes/j1/modules/clipboard/js/clipboard.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: carousel ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: blog_archive, module carousel ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: blog_archive, place general file resources for resource_settings.id: carousel ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/carousel/css/carousel.css">
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/carousel/css/carousel_transitions.css">
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/carousel/css/theme/uno.css">
    <script defer src="/assets/themes/j1/modules/carousel/js/carousel.js"></script>
    <script defer src="/assets/themes/j1/adapter/js/carousel.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: justifiedGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, module justifiedGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, place css files for resource ID: justifiedGallery ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/justifiedGallery/css/justifiedGallery.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, module justifiedGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, place css files for resource ID: justifiedGallery ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/justifiedGallery/css/theme/uno.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, module justifiedGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, place general file resources for resource_settings.id: justifiedGallery ] -->
    <script defer src="/assets/themes/j1/adapter/js/justifiedGallery.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app module justifiedGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout app, place js files for resource ID: justifiedGallery ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script defer src="/assets/themes/j1/modules/justifiedGallery/js/justifiedGallery.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: lightGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, module lightGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, place css files for resource ID: lightGallery ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/lightGallery/css/core/lightgallery.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, module lightGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, place css files for resource ID: lightGallery ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/lightGallery/css/core/lg-transitions.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, module lightGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, place css files for resource ID: lightGallery ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/lightGallery/css/core/lg-fb-comment-box.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, module lightGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, place css files for resource ID: lightGallery ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <link rel="stylesheet" type="text/css" href="/assets/themes/j1/modules/lightGallery/css/themes/uno.css">
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, module lightGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app, place general file resources for resource_settings.id: lightGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app module lightGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout app, place js files for resource ID: lightGallery ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script defer src="/assets/themes/j1/modules/lightGallery/js/core/j1/lightgallery-x.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app module lightGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout app, place js files for resource ID: lightGallery ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script defer src="/assets/themes/j1/modules/lightGallery/js/modules/j1/j1-x-video.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app module lightGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout app, place js files for resource ID: lightGallery ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script defer src="/assets/themes/j1/modules/lightGallery/js/modules/lg-autoplay.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app module lightGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout app, place js files for resource ID: lightGallery ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script defer src="/assets/themes/j1/modules/lightGallery/js/modules/lg-fullscreen.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app module lightGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout app, place js files for resource ID: lightGallery ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script defer src="/assets/themes/j1/modules/lightGallery/js/modules/lg-hash.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app module lightGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout app, place js files for resource ID: lightGallery ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script defer src="/assets/themes/j1/modules/lightGallery/js/modules/lg-pager.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app module lightGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout app, place js files for resource ID: lightGallery ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script defer src="/assets/themes/j1/modules/lightGallery/js/modules/lg-share.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app module lightGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout app, place js files for resource ID: lightGallery ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script defer src="/assets/themes/j1/modules/lightGallery/js/modules/lg-thumbnail.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout: app module lightGallery ] -->
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ layout app, place js files for resource ID: lightGallery ] -->
    <!-- [ERROR  ] [j1.procedures.select_location.proc    ] [ no valid resource locator for LOCAL found ] -->
    <!-- [WARN   ] [j1.procedures.select_location.proc    ] [ fallback location LOCAL selected ] -->
    <script defer src="/assets/themes/j1/modules/lightGallery/js/modules/lg-zoom.js"></script>
    <!-- [INFO   ] [j1.layout.resource_writer.proc        ] [ placement for all data_keys finished ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: video_js ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: vimeo_froogaloop ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: vimeo_player ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: rangeslider ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: datatables ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: scroller ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: deeplAPI ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: crypto ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: advertising ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: comments ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: dropdowns ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: nbinteract ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: plotly ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: bokeh ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: vega ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ process resource: require ] -->
    <!-- [INFO   ] [j1.layout_resource_generator.html     ] [ end processing ] -->
    <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ end processing: load region head, layout: page ] -->
    <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ placement for all lane items finished ] -->
    <!-- [DEBUG  ] [j1.layout.default_writer.proc         ] [ reset all lane items ] -->
    <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ end processing ] -->
    <!-- [INFO   ] [j1.layout.default.html                ] [ end default_writer for region: 'head' ] -->
  </head>
  <body class="nomathjax">
    <!-- [INFO   ] [j1.layout.default.html                ] [ start page processing for region: 'body' ] -->
    <!-- [INFO   ] [j1.layout.default.html                ] [ place wrapper to prevent "DOM flicker" ] -->
    <!-- [INFO   ] [j1.layout.default.html                ] [ make page hidden (suppress flicker) ] -->
    <!-- [INFO   ] [j1.layout.default.html                ] [ suppress page overflows (overflow: hidden) ] -->
    <!-- div id="no_flicker" style="display: none" -->
    <div id="no_flicker" style="overflow: hidden; display: none">
      <!-- [INFO   ] [j1.layout.default.html                ] [ call default_writer for region: 'body-navigation' ] -->
      <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ start processing: load region body-navigation, layout: page ] -->
      <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ check for topsearch ] -->
      <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ start processing: topsearch ] -->
      <!-- [INFO   ] [j1.modules.navigator.topsearch.proc   ] [ start processing: top search ] -->
      <div id="navigator_nav_topsearch" class="top-search ml-4 mr-4" style="background-color: #eceff1 !important; display: none;">
        <div class="input-group">
          <span class="input-group-addon"><i class="mdi mdi-magnify mdi-2x mb-2" style="color:#9E9E9E"></i></span>
          <input id="search-query" type="text" name="q" placeholder="Your search expression" autocomplete="off" class="form-control">
          <span id="clear-topsearch" class="form-clear form-clear-searcher d-none" style="margin-top: 0.75rem"><i class="mdi mdi-format-clear mdi-2x" style="color:#9E9E9E"></i></span>
          <span class="input-group-addon close-search"><i class="mdi mdi-close mdi-3x" style="color:#9E9E9E"></i></span>
        </div>
        <!-- [INFO   ] [j1.modules.navigator.topsearch.proc   ] [ place: results container ] -->
        <div class="search-results hide-scrollbar">
          <section id="search-results" style="display: none"></section>
        </div>
        <!-- [INFO   ] [j1.modules.navigator.topsearch.proc   ] [ place: mustache template ] -->
        <script id="search-results-template" type="text/mustache">
          <ul style="list-style: none; margin-left: .5rem; margin-right: 4.25rem">
          {{#docs}}
            <li>
              <h4 class="result-item"> <a href="{{url}}" target="_blank">{{title}} · {{tagline}}</a> </h4>
              <p class="result-item-text small text-muted mt-2 mb-0">
                <i class="mdi mdi-calendar-blank mdi-18px mr-1"></i>{{displaydate}}
              </p>
              <p class="result-item-text">{{description}}</p>
              <p class="result-item-text small text-muted mb-3">
                <i class="mdi mdi-tag-text-outline mdi-18px mr-1"></i><span class="sr-categories">{{#categories}} {{.}} · {{/categories}}</span>
                <i class="mdi mdi-tag mdi-18px mr-1 ml-2"></i><span class="sr-tags">{{#tags}} {{.}} · {{/tags}}</span>
              </p>
            </li>
          {{/docs}}
          <ul>
        </script>
      </div>
      <!-- [INFO   ] [j1.modules.navigator.topsearch.proc   ] [ end processing: top search ] -->
      <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ end processing: topsearch ] -->
      <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ start processing: mmenu ] -->
      <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ place mmenu container navigator_nav_mmenu for AJAX load ] -->
      <nav id="navigator_nav_mmenu"></nav>
      <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ place mmenu container navigator_toc_mmenu for AJAX load ] -->
      <nav id="navigator_toc_mmenu"></nav>
      <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ place mmenu container navigator_sidebar_mmenu for AJAX load ] -->
      <nav id="navigator_sidebar_mmenu"></nav>
      <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ start processing: navigation ] -->
      <nav id="navigator_nav_navbar" class="navbar navigator navbar-expand-lg navbar-default light navbar-fixed navbar-transparent">
        <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ start processing: main menue ] -->
        <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ place brand to the: right ] -->
        <a href="#" id="navbar-brand" class="navbar-brand link-no-decoration"> <img src="/assets/images/modules/icons/j1/j1-512x512.png" height="48" alt="Jekyll One"></a>
        <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ place mobile menu button to the: left ] -->
        <button id="mmenu-button" type="button" class="navigator navbar-toggler animated fadeIn" data-bs-toggle="collapse" data-bs-target="#navigator_nav_menu" aria-controls="navigator_nav_menu" aria-expanded="false" aria-label="Toggle navigation">
          <i class="mdi mdi-menu mdi-36px"></i>
        </button>
        <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ start processing: nav bar ] -->
        <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ place nav menu container for AJAX load (desktop) ] -->
        <div id="navigator_nav_menu" class="collapse"></div>
        <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ check for quicklinks ] -->
        <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ place container for: quicklinks ] -->
        <div id="navigator_nav_quicklinks" class="quicklink-nav collapse navbar-collapse justify-content-end"></div>
      </nav>
      <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ check for: authclient ] -->
      <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ authclient enabled: false ] -->
      <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ place container for: authclient ] -->
      <div id="authclient_modals" class="authclient_modals"></div>
      <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ end processing: auth_client ] -->
      <!-- [INFO   ] [j1.modules.navigator.generator.html   ] [ end processing: navigation ] -->
      <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ end processing: load region head, layout: page ] -->
      <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ placement for all lane items finished ] -->
      <!-- [DEBUG  ] [j1.layout.default_writer.proc         ] [ reset all lane items ] -->
      <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ end processing ] -->
      <!-- [INFO   ] [j1.layout.default.html                ] [ end processing default_writer for region: 'body-navigation' ] -->
      <!-- [INFO   ] [j1.layout.default.html                ] [ place content inherited by layout: page ] -->
      <!-- [INFO   ]  [j1.layout.page.html                   ] [ start content processing ] -->
      <!-- [INFO   ] [j1.layout.page.html                    ] [ call default_writer for region: 'body-header' ] -->
      <header id="header">
        <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ start processing: load region body-header, layout: page ] -->
        <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ load page specific header data from yaml data file ] -->
        <!-- [INFO   ] [j1.attic.generator.html               ] [ attic enabled: true ] -->
        <!-- [INFO   ] [j1.attic.generator.html               ] [ attic enabled for layout: page ] -->
        <!-- [INFO   ] [j1.attic.generator.html               ] [ place attic detected: attic_page_post ] -->
        <div id="attic_page_post" class="attic notranslate">
          <div class="container">
            <div id="head-title" class="head-title animated"> 
              <h2 id="head-title-text" class="notoc">Threshold behaviour of a stochastic vector plant model for Tomato Yellow Curl Leaves disease</h2>
            </div>
            <div id="head-tagline" class="head-tagline animated"> 
              <h3 id="head-tagline-tagline" class="notoc">a study based on mathematical analysis and simulation</h3>
            </div>
            <!-- Image caption container (display: none per default) -->
            <!-- div class="attic-caption text-emphasis-"></div -->
            <div class="attic-caption"></div>
          </div>
        </div>
        <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ end processing: load region head, layout: page ] -->
        <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ placement for all lane items finished ] -->
        <!-- [DEBUG  ] [j1.layout.default_writer.proc         ] [ reset all lane items ] -->
        <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ end processing ] -->
      </header>
      <!-- [INFO   ] [j1.layout.page.html                    ] [ end default_writer processing for region: 'body-header' ] -->
      <!-- [INFO   ] [j1.layout.page.html                    ] [ call content_writer for region: 'body-main' ] -->
      <div id="content" class="container mb-5">
        <!-- [INFO   ] [j1.layout.content_writer.proc         ] [ start processing: load CONTENT portion for layout: page ] -->
        <!-- [INFO   ] [j1.layout.content_writer.proc         ] [ load region 'body-main' for layout: page ] -->
        <!-- [INFO   ] [j1.layout.content_generator_page.html ] [ start main for full page ] -->
        <main id="main-content" class="r-text-300 js-toc-content">
          <!-- [INFO   ] [j1.layout.content_generator_page.html ] [ begin place content ] -->
          <div id="toc" class="toc">
            <div id="toctitle">Table of Contents</div>
            <ul class="sectlevel1">
              <li>
                <a href="#abstract">Abstract</a>
                <ul class="sectlevel2">
                  <li><a href="#keywords">Keywords</a></li>
                </ul>
              </li>
              <li><a href="#introduction">Introduction</a></li>
              <li>
                <a href="#sec:model_formulation">Deterministic base dynamics</a>
                <ul class="sectlevel2">
                  <li><a href="#subsec:deterministic_fixed_points">Deterministic Fixed Points</a></li>
                </ul>
              </li>
              <li><a href="#sec:stochastic_extension">Stochastic Extension</a></li>
              <li><a href="#sec:solution_existence">Existence and uniqueness of positive Solution</a></li>
              <li><a href="#sec:extinction">Extinction</a></li>
              <li><a href="#sec:persistence">Persistence</a></li>
              <li><a href="#sec:numerics">Numerical Experiments</a></li>
              <li><a href="#references">References</a></li>
            </ul>
          </div>
          <script>
            MathJax = {
                loader: {load: ['[tex]/textmacros', '[tex]/newcommand', '[tex]/configmacros']},
                tex: {
                    inlineMath: [['$', '$'], ['\\(', '\\)']],
                    tags: 'ams',
                    packages: {'[+]': ['textmacros', 'newcommand', 'configmacros']},
                    macros: {
                        oe: '\u0153',
                        OE: '\u0152',
                        ae: '\u00E6',
                        AE: '\u00C6',
                        aa: '\u00E5',
                        AA: '\u00C5',
                        o: '\u00F8',
                        O: '\u00D8',
                        l: '\u0142',
                        L: '\u0141',
                        ss: '\u00DF',
                        SS: '\u1E9E',
                        dag: '\u2020',
                        ddag: '\u2021',
                        P: '\u00B6',
                        copyright: '\u00A9',
                        pounds: '\u00A3',
                        EX:["{ \\mathbb{E}{ {#1}} \\left[ {#2} \\right] }", 2],
                        E:["{ \\mathbb{E} \\left[ {#1} \\right] }", 1],
                        probX:[ "{ \\mathbb{P}{ {#1} } \\left[ {#2} \\right] }", 2 ]
                    }
                },
                chtml: {
                    mtextInheritFont: true
                },
                sag: {
                    mtextInheritFont: true
                },
                svg: {fontCache: 'global'}
            };
          </script>
          <script async id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js"></script>
          <style>
            p{
                text-indent: 30px;
                margin-top: 2em ;
                margin-bottom: 2em ;
            }
            a:link {
                color: #6296be;
            }
            /* visited link */
            a:visited {
                color: #2c78b4;
            }
            /* mouse over link */
                a:hover {
                color: #00b2c8;
            }
            /* selected link */
            a:active {
                color: #626262;
            }
            .theorem {
                display: block;
                font-style: italic;
                    margin-top: 2em ;
                    margin-bottom: 2em ;
            }
            .theorem:before {
                content: "Theorem. ";
                font-weight: bold;
                font-style: normal;
            }
            .theorem[text]:before {
                content: "Theorem (" attr(text) ") ";
            }
            .definition {
                display: block;
                font-style: italic;
                margin-top: 2em ;
                margin-bottom: 2em ;
            }
            .definition:before {
                content: "Definition. ";
                font-weight: bold;
                font-style: normal;
            }
            .definition[text]:before {
                content: "Definition (" attr(text) ") ";
            }
            .proof {
                display: block;
                font-style: italic;
            }
            .proof:before {
                content: "Proof. ";
                font-weight: bold;
                font-style: normal;
            }
            .proof[text]:before {
                content: "Proof (" attr(text) ") ";
            }
            .remark {
                display: block;
                font-style: italic;
                background-color: #3f92af;
                border-radius: 15px;
                margin-top: 6em ;
                margin-bottom: 2em ;
            }
            .remark:before {
                content: "Remark. ";
                font-weight: bold;
                font-style: normal;
            }
            .remark[text]:before {
                content: "Remark (" attr(text) ") ";
            }
            .assumption{
                display: block;
                font-style: italic;
                background-color: #3f92af;
                border-radius: 15px;
                margin-top: 6em ;
                margin-bottom: 2em ;
            }
            .assumption:before {
                content: "Assumption. ";
                font-weight: bold;
                font-style: normal;
            }
            .assumption[text]:before {
                content: "Remark (" attr(text) ") ";
            }
            span.right{
                float:right;
                margin-top: 1em ;
                margin-bottom: 1em ;
            }
            .custom {
                 list-style-type: none;
            }
            .custom li {
                 counter-increment: step-counter;
            }
            .custom li::before {
                font-weight: bold;
                font-style: normal;
                content: '(ASS-'counter(step-counter)')';
                margin-right: 5px;
            }
          </style>
          <div class="sect1">
            <h2 id="abstract">Abstract</h2>
            <div class="sectionbody">
              <div class="paragraph">
                <p>Diseases transmitted by vectors in crops represent a severe risk to
                  the farmer, causing low production and lower quality, which results
                  in a drastic reduction in crop yield. Here we study a stochastic
                  plant-vector-host epidemic model with direct transmission.
                  Perturbing with Brownian motion plant replanting and vector
                  fumigation rates, we formulate an Ito stochastic differential
                  equations that capture the uncertainty of the controls.
                  We derive conditions to assure extinction and persistence
                  of disease using two different stochastic versions of the so-called
                  basic reproductive number.
                  Finally, we verify and illustrate our theory by numerical experiments.
                  Our simulations suggest that uncertainty could drive dramatic stability changes.
                  Because in practice, to confirm an infected plant via laboratory
                  tests is not necessarily feasible, replanting and fumigation strategies
                  suffer considerable uncertainty.
                  Here, we quantify and study the impact and consequences of this uncertainty.
                  We conclude that environmental noise promotes dramatic stability
                  changes.</p>
              </div>
              <div class="sect2">
                <h3 id="keywords">Keywords</h3>
                <div class="literalblock">
                  <div class="content">
                    <pre>Tomato yellow curl virus disease, stochastic Model, threshold behaviour,
vector-plant-host, persistence, extinction, Brownian motion</pre>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div class="sect1">
            <h2 id="introduction">Introduction</h2>
            <div class="sectionbody">
              <div class="paragraph">
                <p>White-fly (Bemisia tabaci) is the responsible vector for
                  critical insect-transmitted diseases in crops like
                  tomato, soybeans, and cotton
                  <a class="citation" href="#Yan2021">[1], [2], [3], [4]</a>.
                  Begomoviruses transmitted by this insect provoke reductions in the
                  production and quality of this and other products in India, Mexico,
                  USA, Colombia, and other producer countries
                  <a class="citation" href="#Li2021">[5], [6]</a>.
                  For example, Tomato Yellow Curl Lives (TYCL) disease cause that tomato
                  plants to become futile and diminish their growth
                  <a class="citation" href="#Yan2021">[1], [7]</a>.</p>
              </div>
              <div class="paragraph">
                <p>Despite control strategies like fumigation or replanting,
                  it is complicated to eradicate this vector. This insect has
                  sophisticated survival mechanism that permits shelter itself
                  in another perennial—​and abundant on the borders of
                  the plantation—​insecticide-resistant plants.
                  Thus, fumigation promotes partial migration instead of extinction.
                  In addition, plant symptoms like curl leaves and yellow
                  discolouration also are associated with iron or phosphate soil
                  deficiencies. Consequently, control methods like replanting
                  neighbours or physical barriers suffer from high
                  uncertainty—​a farmer could replant a healthy plant instead of
                  a latent one and vice-versa.</p>
              </div>
              <div class="paragraph">
                <p>There is well-established literature that treats deterministic
                  models for vector plant structures. Our primary reference is the work of
                  <a class="citation" href="#Holt1999">[8], [7], [9]</a> and other important reference.
                  Recently, control strategies have been studied as optimal control
                  problems by Lenhart et al. in
                  <a class="citation" href="#Bokil2019">[10]</a>
                  for a tomato disease and Luo et al. <a class="citation" href="#Luo2021">[11]</a> for Huanglongbing.
                  However, the uncertainty quantification or stochastic counterparts
                  still are under development. In this direction, we mention the
                  recent works of Allen et al. <a class="citation" href="#Hebert2016">[12]</a>, Stella \&amp; Ghosh
                  <a class="citation" href="#Stella2019">[13]</a> Agarwal et al. <a class="citation" href="#Agarwal2019">[14]</a>,
                  and Chen-Charpentier <a class="citation" href="#Chen-Charpentier2021">[15]</a>
                  as significant contributions.</p>
              </div>
              <div class="paragraph">
                <p>Here, we aim to quantify the uncertainty due to possible errors
                  in identifying infected plants and random fluctuations of vector
                  migration produced in fumigation. Our study attempts to capture
                  the impact of ambient noise and try to explain it by formal
                  analysis and simulation.</p>
              </div>
              <div class="paragraph">
                <p>According to Britton et al. <a class="citation" href="#Britton2019">[16]</a>, stochastic effects
                  of an epidemic model are essential when the underlying
                  population size is relatively small (at most hundreds).
                  However, recently have been reported studies that attempt
                  the same methodology of deterministic models in
                  mathematical epidemiology—being positivity,
                  invariance, extinction, and persistence the primary
                  objects of study. In respect of this regard, we refer
                  the reader to
                  <a class="citation" href="#Gray2011">[17], [18], [19]</a>.
                  However, the application to a specific disease
                  driven with data is still under development.
                  Roughly speaking, the methodology bases
                  the analysis on study versions of a SIR structure
                  and characterizes conditions to extinction-persistence
                  for a convenient definition of a stochastic
                  reproductive number, see <a class="citation" href="#Cao2016a">[18], [20], [21], [22]</a>.</p>
              </div>
              <div class="paragraph">
                <p>Regarding plant-vector-host structures, we see relevant
                  the contributions of Allen et al. <a class="citation" href="#Gray2011">[17]</a>
                  and Agarwal et al. <a class="citation" href="#Agarwal2019">[14]</a>.
                  These works would apply the most common techniques to
                  formulate stochastic models for epidemics
                  based on Ito differential equations. Agarwal studied
                  pine wilt disease with a stochastic optimal control
                  problem, where the controls are related to prevention and
                  fumigation. Allen considers a stochastic plant-vector-host
                  structure deduced as the limit of a Continuous Markov chain
                  model. With this structure, Allen et al. study the effects
                  of vector aggregation and dispersal subject to noise.
                  On this occasion, we deploy a compartmental $SLI-SI$
                  plant vector structure with Brownian noise by formulating
                  an Itô stochastic differential equation (SDE).
                  Our main results characterize conditions to assure
                  extinction and persistence.
                  First, we identify requirements to ensure extinction
                  due only to noise intensity. Then, for sufficiently small noise, we
                  develop the threshold theory for two proposals of the
                  stochastic reproductive number. Finally, we verify and
                  illustrate our results by numerical simulation.</p>
              </div>
              <div class="paragraph">
                <p>We aim to quantify the effects of uncertainty
                  of the mentioned controls over the extinction and persistence
                  of the disease.
                  We conclude that noise would play a crucial role
                  in the trade-off between extinction and persistence.
                  In other words, the combination of errors due by plant
                  identification and random fluctuations of vector migration
                  would lead to the death of healthy and infected plants and or
                  its prevalence.
                  Consequently, the local behaviour of vectors follows similar
                  codependency.</p>
              </div>
              <div class="paragraph">
                <p>Using the package <strong>DifferentialEquation.jl</strong> from <strong>Julia</strong>
                  <a class="citation" href="#rackauckas2017differentialequations">[23]</a>, we simulate
                  our stochastic model. This package provides the most
                  recent solvers for stiff Itô problems.
                  We apply the parallel implementation of <strong>S-ROCK</strong> method
                  <a class="citation" href="#Abdulle2008">[24]</a>---getting a stable, and high convergent
                  simulation of 10 000 paths after a reasonable computational
                  time.
                  Our simulations suggest that environmental noise induces large dispersion.
                  Thus, we deduce that uncertainty would lead to unexpected results.
                  Therefore, a significant implication is related to the good identification
                  of healthy crops---our recommendation is to balance replanting
                  with efficiency.</p>
              </div>
              <div class="paragraph">
                <p>After this brief introduction, in <a href="#sec:model_formulation">Section 2</a>
                  we discuss the dynamic properties of the deterministic base. Then,
                  <a href="#sec:stochastic_extension">Section 3</a> extends this base by stochastic
                  perturbation to formulate our SDE model.  <a href="#sec:solution_existence">Section 4</a>
                  deploys the results about the existence and uniqueness of the global
                  positivity solution. After that, <a href="#sec:extinction">Section 5</a> characterizes
                  conditions on the noise intensity and other parameters to assure disease
                  extinction.  Then we complete the analysis with results to ensure disease
                  persistence in <a href="#sec:persistence">Section 6</a>. Finally in <a href="#sec:numerics">Section 7</a>,
                  we verify and illustrate our theory with numerical experiments.</p>
              </div>
            </div>
          </div>
          <div class="sect1">
            <h2 id="sec:model_formulation">Deterministic base dynamics</h2>
            <div class="sectionbody">
              <div class="paragraph">
                <p>Here, we adapt the deterministic dynamics presented by
                  Holt et al. <a class="citation" href="#Holt1999">[8]</a>. Following the authors,
                  we split the plant population according to,
                  susceptible $ (S_p) $, latent $ (L_p) $, and infected
                  $ (I_p) $. And divide vector population into susceptible
                  $S_v$ and $I_v$ infected classes.
                  We consider the following assumptions: Susceptible plants become latent
                  plants when an infected vector feeds on them at constant rate, $\beta_p$.
                  Latent plants become infected plants at a rate of $b$. We replant latent and
                  infected plants that are detected at rate $r_1$ and $r_2$, respectively.
                  Removed plants are replanting as susceptible plants in the crop.
                  For vector dynamics, we consider that susceptible vectors become acquirer
                  disease, at rate $\beta_v$, when they eat an infected plant. We assume natural
                  vector die at rate $\gamma$, and by fumigation at rate $\gamma_f$. A proportion
                  of the susceptible and infected vectors migrates from alternatives host plants
                  to the crop.</p>
              </div>
              <div class="paragraph">
                <p>With these hypotheses, the model has the following equations</p>
              </div>
              \begin{equation}
              \label{eq:DeterministicSystem}
              \begin{aligned}
              \dot{S_p} &amp;=
              -\beta_p S_p
              \frac{I_v}{N_v} + r_1 L_p + r_2 I_p
              \\
              \dot{L_p} &amp;=
              \beta_p S_p
              \frac{I_v}{N_v} - (b + r_1) L_p
              \\
              \dot{I_p} &amp;=
              b L_p - r_2 I_p  \\
              \dot{S_v} &amp;=
              -\beta_v S_v
              \frac{I_p}{N_p} - (\gamma + \gamma_f) S_v
              +(1-\theta) \mu
              \\
              \dot{I_v} &amp;=
              \beta_v S_v \frac{I_p}{N_p}
              - (\gamma + \gamma_f) I_v
              + \theta \mu
              \\
              S_p(0) &amp;=S_{p0}, \qquad
              L_p(0) = L_{p0}, \qquad
              I_p(0) = I_{p0},
              \\
              S_v(0) &amp;= S_{v0}, \qquad
              I_v(0) = I_{v0},
              \\
              N_p(t) &amp;= S_p(t) + L_p(t) + I_p(t),
              \\
              N_v(t) &amp;= S_v(t) + I_v(t).
              \end{aligned}
              \end{equation}
              <div class="paragraph">
                <p>We conduct a calibration of the model
                  <a href="#eq:DeterministicSystem">(1)</a>
                  based on the parameters and data reported in
                  <a class="citation" href="#Holt1999">[8]</a> and  via Multi Chain Montecarlo Method (MCMC).
                  Here we used the <strong>R-Stan</strong> implementation and followed the workflow from
                  <a class="citation" href="#Grinsztajn2021">[25]</a>.
                  We refer the reader to the supplementary material
                  <a class="citation" href="#Salcedo-varela2021">[26]</a>
                  for more details.</p>
              </div>
              <div class="paragraph">
                <p>Figure <a href="#fig:fitnonresistance"> 1</a> and <a href="#fig:fitresistance">2</a>
                  show the confidence band, and the median for <a href="#eq:DeterministicSystem">model (1)</a>
                  to each tomato data set reported by <a class="citation" href="#Holt1999">[8]</a>.
                  autoref{tbl:parameter_value} displays the estimates of the parameters
                  for each tomato species.</p>
              </div>
              <table id="tbl:deterministic_tbl" class="tableblock frame-all grid-all stretch">
                <caption class="title">Table 1. Parameters description of deterministic dynamics in ODE <a href="#eq:DeterministicSystem">(1)</a>
                </caption>
                <colgroup>
                  <col style="width: 33.3333%;">
                  <col style="width: 33.3333%;">
                  <col style="width: 33.3334%;">
                </colgroup>
                <thead>
                  <tr>
                    <th class="tableblock halign-left valign-top">Par.</th>
                    <th class="tableblock halign-left valign-top">Description</th>
                    <th class="tableblock halign-left valign-top">Reference sources</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock">$\beta_p$</p>
                    </td>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock">infection rate of susceptible plants by infected vectors</p>
                    </td>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock"><a class="citation" href="#Butter1978">[27]</a>, <a class="citation" href="#ramappa1994">[28]</a></p>
                    </td>
                  </tr>
                  <tr>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock">$\beta_v$</p>
                    </td>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock">infection rate of susceptible vectors by an infected plant</p>
                    </td>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock"><a class="citation" href="#Butter1978">[27], [28]</a></p>
                    </td>
                  </tr>
                  <tr>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock">$r_1$, $r_2$</p>
                    </td>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock">replanting rate of latent and infected plants</p>
                    </td>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock"><a class="citation" href="#Nateshan1996">[29]</a></p>
                    </td>
                  </tr>
                  <tr>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock">$b$</p>
                    </td>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock">latency rate</p>
                    </td>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock"><a class="citation" href="#Nateshan1996">[29]</a></p>
                    </td>
                  </tr>
                  <tr>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock">$\gamma$, $\gamma_f$</p>
                    </td>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock">natural and fumigation vector death rate</p>
                    </td>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock"><a class="citation" href="#Holt1999">[8], [7]</a></p>
                    </td>
                  </tr>
                  <tr>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock">$\mu$</p>
                    </td>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock">vector migration rate from alternative plants to crop</p>
                    </td>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock"><a class="citation" href="#Ramappa1998">[30]</a></p>
                    </td>
                  </tr>
                  <tr>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock">$\theta$</p>
                    </td>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock">proportion of infected vector from migration</p>
                    </td>
                    <td class="tableblock halign-left valign-top">
                      <p class="tableblock"><a class="citation" href="#ramappa1994">[28], [30]</a></p>
                    </td>
                  </tr>
                </tbody>
              </table>
              <div id="fig:fitnonresistance" class="paragraph">
                <div class="title">Figure 1 Callibration Confidence bands for cumulative disease incidence of <a href="#eq:DeterministicSystem">model (1)</a>
                </div>
                <p>for non-resistant tomato data. Squares denote Rashmi, and points PSCL-4 tomato species data. The blue and red line
                  represents the median incidence curve. Parameter values fixed in Table 2.
                  Data from Holt and Muniyappa. Digital edition figures feasible in <a href="https://plotly.com/AdrianSalcedo/347/" class="bare">https://plotly.com/AdrianSalcedo/347/</a>.</p>
              </div>
              <div id="lb-example-standalone" class="imageblock">
                <div class="content mb-4" style="margin-bottom: .75rem;">
                  <a class="notoc" href="/assets/images/pages/preprints/sto_tycl/figures/figure_01.jpg" data-lightbox="pages/preprints/sto_tycl/figures/figure_01.jpg" data-title="CB 95% of Calibration of parameters for model (1)">
                    <img class="img-fluid" src="/assets/images/pages/preprints/sto_tycl/figures/figure_01.jpg" alt="" width="800">
                  </a>
                </div>
              </div>
              <div id="fig:fitresistance" class="paragraph">
                <div class="title">Figure 2 Confidence bands of cumulative disease incidence to resistan tdata set.</div>
                <p>($\color{darkblue}{\blacksquare}$,$\color{orange}{\bullet}$) LA1582, and
                  Tyking tomato species data.
                  The blue and red line represents the median incidence curve.
                  Parameter values fixed in <a href="#tbl:parameter_value">Table 1</a>.
                  Data from Holt and Muniyappa <a class="citation" href="#Holt1999">[8]</a>.
                  See <a href="https://plotly.com/~AdrianSalcedo/339/" class="bare">https://plotly.com/~AdrianSalcedo/339/</a> for a interactive visualization.</p>
              </div>
              <div id="lb-example-standalone" class="imageblock">
                <div class="content mb-4" style="margin-bottom: .75rem;">
                  <a class="notoc" href="/assets/images/pages/preprints/sto_tycl/figures/figure_02.jpg" data-lightbox="pages/preprints/sto_tycl/figures/figure_02.jpg" data-title="Calibration of parameters">
                    <img class="img-fluid" src="/assets/images/pages/preprints/sto_tycl/figures/figure_02.jpg" alt="" width="800">
                  </a>
                </div>
              </div>
              <div class="paragraph">
                <p>TODO: Reproduce Table 2.</p>
              </div>
              <div class="sect2">
                <h3 id="subsec:deterministic_fixed_points">Deterministic Fixed Points</h3>
                <div class="paragraph">
                  <p>According to  ODE <a href="#eq:DeterministicSystem">(1)</a>, let
                    $N_v^{\infty}:= {\mu}/{( \gamma + \gamma_f)}$.
                    Let $\mathbf{R} ^ 5$ and consider the set</p>
                </div>
                \begin{equation*}
                %\label{eqn:invariant_set}
                \Gamma:=
                \left \{
                (S_p, L_p, I_p, S_v, I_v)^{\top} \in \mathbf{R} ^ 5: \
                0\leq S_p + L_p + I_p \leq N_p, \quad
                0 \leq S_v + I_v \leq N_v^{\infty}
                \right \}.
                \end{equation*}
                <div class="paragraph">
                  <p>Note that</p>
                </div>
                \begin{equation}\label{eqn:plant_size}
                \begin{aligned}
                \frac{d N_p}{dt} &amp;=
                \frac{d}{dt}(S_p + L_p + I_p) = 0,
                \\
                \frac{d N_v}{dt} &amp;=
                \mu - ( \gamma + \gamma_f) N_v.
                \end{aligned}
                \end{equation}
                <div class="paragraph">
                  <p>Thus the whole plant population $N_p(t)$ remains
                    constant for all $t\geq 0$. In addition,
                    \begin{equation}
                    \label{eqn:vector_size}
                    N_v(t)= N^{\infty}_v<br>
                    (N_v(0) - N^{\infty}_v) \exp(-( \gamma + \gamma_f) t) .
                    \end{equation}
                    From this relation, we conclude that
                    $
                    \lim_{t\to\infty}
                    N_v(t) = N_v^{\infty},
                    $
                    and that population sizes $N_p(t)$, $N_v(t)$ satisfies a
                    conservative law. To summarize, we prove that set $\Gamma$
                    is positive invariant in the sense of the following
                    definition <a class="citation" href="#Meiss2007">[31, P. 106]</a>.</p>
                </div>
                <div class="definition" text="Invariant set ">
                  <p>
                    Let $f:[0,T]\times \Gamma \subset \mathbb{R}^n\to \mathbb{R}^n$.
                    We say that $\Gamma $ is an invariant set under $f$,
                    if $f(t,\Gamma)\subset \Gamma$ for all $t\in [0, T]$.
                  </p>
                </div>
                <div class="theorem" text="Positive-invariance">
                  <p>
                    The set $\Gamma$ is positive invariant.
                  </p>
                </div>
                <div class="proof">
                  <p>
                    Let $(S_p,L_p,I_p,S_v,I_v)^\top \in \Gamma$ and apply the
                    right-hand side of dynamic <a href="#eq:DeterministicSystem">(1)</a>.
                    We have
                    \begin{align*}
                    \frac{dS_p}{dt}
                    &amp;=
                    -\beta_p S_p
                    \frac{I_v}{N_v} + r_1 L_p + r_2 I_p ,
                    \\
                    \frac{dL_p}{dt}
                    &amp;=
                    \beta_p S_p\frac{I_v}{N_v} - (b + r_1) L_p ,
                    \\
                    \frac{dI_p}{dt}
                    &amp;=
                    b L_p - r_2 I_p,
                    \\
                    \frac{dS_v}{dt}
                    &amp;=
                    -\beta_v S_v\frac{I_p}{N_p} -
                    ( \gamma + \gamma_f) S_v+(1-\theta) \mu ,
                    \\
                    \frac{dI_v}{dt}
                    &amp;=
                    \beta_v S_v \frac{I_p}{N_p} - ( \gamma + \gamma_f) I_v
                    + \theta \mu.
                    \end{align*}
                    Now, verifying $\Gamma$ conditions, we get
                    $dS_p / dt + dL_p / dt + dI_p / dt \leq dN_p / dt$.
                    By law conservation
                    <a href="#eqn:plant_size">
                      (2)
                    </a>,
                    $S_p+L_p+I_p\leq N_p$.
                    Similarly,
                    $
                    dS_v / dt +  dI_v / dt \leq \mu  -(\gamma + \gamma_f) (S_v + I_v) \leq
                    \mu - ( \gamma + \gamma_f) N_v
                    $.
                    Further, from relation \eqref{eqn:vector_size}, we obtain
                    $
                    S_v+I_v\leq N^{\infty}_v+(N_v(0)-N^{\infty}_v)\exp{(-
                    ( \gamma + \gamma_f) t)}
                    $.
                    Since this is for all $t$, we have $S_v+I_v\leq N^{\infty}_v$, hence
                    $f(S_p,L_p,I_p,S_v,I_v) \in \Gamma$. Therefore, $\Gamma$ is
                    invariant under
                    <a href="#eq:DeterministicSystem">
                      System (1)
                    </a>
                    <span class="right">
                      $\square$.
                    </span>
                  </p>
                </div>
                <div class="paragraph">
                  <p>System <a href="#eq:DeterministicSystem">(1)</a> has the following fixed points:</p>
                </div>
                Free disase Equilibrium
                $$
                FDE:= \left(
                N_p,0,0,N^{\infty}_v,0
                \right)^{\top}.
                $$
                <div class="paragraph">
                  <p>and the Endemic Equilibrium</p>
                </div>
                $$
                EE:=(S_p^{**},
                L_p^{**},
                I_p^{**},
                S_v^{**},
                I_v^{**}).
                $$
                <div class="paragraph">
                  <p>Here we compute this deterministic fixed points of ODE
                    <a href="#eq:DeterministicSystem">(1)</a>---more details <a class="citation" href="#Salcedo-varela2021">[26]</a> ---and show that its uniqueness.
                    To determine this fixed points, we solve,</p>
                </div>
                <div id="eqn:fixed_points" class="paragraph">
                  <p>\begin{equation}
                    \label{eqn:fixed_points}
                    \begin{aligned}
                    -\beta_p S_p \frac{I_v}{N_v} + r_1L_p+r_2I_p &amp;= 0\\
                    \beta_p S_p \frac{I_v}{N_v} - (b + r_1) L_p &amp;= 0\\
                    b L_p - r_2 I_p &amp;= 0\\
                    -\beta_v S_v \frac{I_p}{N_p} -( \gamma + \gamma_f) S_v +(1-\theta) \mu &amp;= 0\\
                    \beta_v S_v \frac{I_p}{N_p} -( \gamma + \gamma_f) I_v + \theta \mu &amp;= 0.
                    \end{aligned}
                    \end{equation}
                    Note that <a href="#eqn:fixed_points">Eq. (4)</a> has at least
                    two solutions---the free disease $(FDE)$
                    and the endemic $(EE)$ fixed point. To characterize
                    the free disease solution
                    $ FDE:= (S_{p} ^ *, L_{p}^*, I_{p}^*, S_{v}^*, I_{v}^*)$,
                    we assume that
                    ${L^*_p = I_p^* = I_v^* = 0}$.
                    Consequently, from relation <a href="eqn:plant_size">(2)</a>,
                    the first component of <a href="#eqn:fixed_points">Eq. (4)</a>
                    implies ${S_p ^ * = N_p}$. Moreover,
                    from \eqref{eqn:vector_size} we see that
                    $S_v ^ * + I_v ^ * \to N^{\infty}_v $ as
                    $t \to \infty$, then
                    $ S_v^* \to N^{\infty}_v$.
                    Therefore, FDE results
                    $$
                    FDE:= \left(
                    N_p,0,0,N^{\infty}_v,0
                    \right)^{\top}.
                    $$
                    For the endemic fixed point
                    $
                    EE:=(S_p^{**},
                    L_p^{**},
                    I_p^{**},
                    S_v^{**},
                    I_v^{**})
                    $,
                    we assume that components $L_p^{**}$, $I_p^{**}$, $I_v^{**}$
                    are positive and solve the right-hand side of
                    \eqref{sys::DeterministicSystem} in terms of $I_v^{**}$.
                    From <a href="#eqn:fixed_points">Eq. (4)</a> we obtain</p>
                </div>
                \begin{equation*}
                \begin{aligned}
                S^{**}_p &amp;=
                \frac{
                (r_1 L^{**}_p + r_2 I ^ {**} _ p) N_v
                }{
                I^{**}_v \beta_{p}
                }
                \\
                L^{**}_p &amp;=
                \frac{
                \beta_{p} S_p^{**} I_v^{**}
                }{
                N_v \left( b+r_1 \right)
                },
                \\
                I ^ {**} _ p &amp;=
                \frac{b L^{**}_p}{r_2},
                \\
                S^{**}_v &amp;=
                \frac{
                \left(
                1-\theta
                \right)
                \mu\, N_{p}
                }{
                ( \gamma + \gamma_f)\, N_{p} + I^{**}_p
                \beta_{v}
                }.
                \end{aligned}
                \end{equation*}
                <div class="paragraph">
                  <p>Expressing the above relations in terms of $L_p^{**}, I^{**}_p$ results</p>
                </div>
                \begin{equation}
                \label{eqn:endemic_point_components}
                \begin{aligned}
                S^{**}_p &amp;=
                \frac{
                (r_1 L^{**}_p + r_2 I^{**}_p) N_v
                }{
                I^{**}_v \beta_{p}
                },
                \\
                L^{**}_p &amp;=
                \frac{
                r_1L_p^{**} + r_2 I^{**}_p
                }{
                b + r_1
                },
                \\
                I^{**}_p &amp;=
                \frac{
                b (r_1 L_p^{**} + r_2 I_p^{**})
                }{
                \left(
                b + r_1
                \right)
                r_2
                },
                \\
                S^{**}_v &amp;=
                \frac{
                \left(
                1 - \theta
                \right)
                \mu(b + r_1)
                r_2N_p
                }{
                ( \gamma + \gamma_f) N_p(b + r_1)
                r_2 +
                b
                (
                r_1 L_p ^ {**} + r_2
                I_p ^ {**}
                )
                \beta_v
                }.
                \end{aligned}
                \end{equation}
                <div class="paragraph">
                  <p>Substituting the above relations into the $I_v$ component of
                    <a href="#eq:DeterministicSystem">ODE (1)</a> yields</p>
                </div>
                \begin{align*}
                &amp;[
                b ( \gamma + \gamma_f) \beta_p I ^ {**}_v +
                b ( \gamma + \gamma_f) N_v r_2 +
                b \beta_p \beta_v I^{**}_v +
                ( \gamma + \gamma_f) \beta_p r_2 I^{**}_v
                \\
                &amp;+
                ( \gamma + \gamma_f) N_v r_1 r_2
                ] I^{**}_v
                =
                N_v
                (
                b ( \gamma + \gamma_f) \theta \beta_p I^{**}_v +
                b ( \gamma + \gamma_f) \theta N_v r_2
                \\
                &amp;+
                ( \gamma + \gamma_f) \theta \beta_p r_2 I^{**}_v +
                ( \gamma + \gamma_f) \theta N_v r_1 r_2 +
                b \beta_p \beta_v I^{**}_v
                ).
                \end{align*}
                <div class="paragraph">
                  <p>Let</p>
                </div>
                \begin{align*}
                a_1 &amp;:=
                (\gamma + \gamma_f) \beta_p (b + r_2) +
                b \beta_p \beta_v
                ,\\
                a_2 &amp;:=
                (\gamma + \gamma_f) N_v [r_2 (b + r_1) - \beta_p \theta (b + r_2)]
                - N_v b \beta_p \beta_v
                ,
                \\
                a_3 &amp;:=
                (\gamma + \gamma_f) {N_v}^2 r_2 \theta (b + r_1).
                \end{align*}
                <div class="paragraph">
                  <p>So, after some algebraic manipulations, we establish
                    the following quadratic equation</p>
                </div>
                \begin{equation}\label{eqn::QuadraticInfectedFP}
                a_1 I_v^{**2} + a_2 I^{**}_v - a_3 = 0.
                \end{equation}
                <div class="paragraph">
                  <p>According to the discriminant $\Delta = a_2  ^ 2 -  4a_1 a_3$,
                    we get</p>
                </div>
                \begin{equation*}
                \begin{aligned}
                \Delta
                &amp;=
                \{
                (\gamma + \gamma_f) N_v [r_2 (b + r_1)
                - \beta_p \theta (b + r_2)]
                - N_v b \beta_p \beta_v
                \} ^ 2
                \\
                &amp; +
                4[
                ( \gamma + \gamma_f) \beta_p (b + r_2) +
                b \beta_p \beta_v
                )
                (
                (\gamma + \gamma_f) {N_v}^2 r_2 \theta (b + r_1)
                ].
                \end{aligned}
                \end{equation*}
                <div class="paragraph">
                  <p>Note that $\Delta$ is positive, this implies two real solutions</p>
                </div>
                \begin{equation}
                \label{eqn:I_v_positiity}
                I^{**}_{v} =
                \frac{
                -a_2 +
                \sqrt{a_2^2 - 4a_1 a_3}}%
                {2a_1}.
                \end{equation}
                <div class="paragraph">
                  <p>From relations \eqref{eqn:endemic_point_components} and \eqref{eqn:I_v_positiity}
                    we obtain endemic fixed point $EE$.
                    And using van den Driessche’s <a class="citation" href="#VandenDriessche2017a">[32]</a> definition of basic reproductive
                    number to compute basic reproductivenumber
                    $\mathcal{R}^d_0$ as the spectral radius of the next-generation matrix.
                    For the system <a href="#eq:DeterministicSystem">(1)</a> next-generation matrix results</p>
                </div>
                \begin{equation*}\label{Mtx::NextGenerationMatrix}
                K:=
                \begin{bmatrix}
                0 &amp; 0 &amp;\frac{\beta_p S^*_p}{N^{\infty}_v ( \gamma + \gamma_f)}
                \\
                0 &amp; 0 &amp; 0
                \\
                \frac{\beta_v S^*_v b}{N_p (b+r_1) r_2} &amp;
                \frac{\beta_v S^*_v}{N_p r_2} &amp; 0
                \\
                \end{bmatrix}.
                \end{equation*}
                <div class="paragraph">
                  <p>Matrix $K$  has $(i, j)$ entry equal to the expected number of secondary
                    infections in the compartment $i$ produced by an infected individual introduced
                    in the compartment $j$. Calculating the basic reproductive number for the system
                    <a href="#eq:DeterministicSystem">(1)</a> we have</p>
                </div>
                \begin{equation*}%\label{eqn::DeterministicBRN}
                \mathcal{R}^d_0:=\frac{\beta_p \beta_v b}{(b + r_1 ) ( \gamma + \gamma_f) r_2}.
                \end{equation*}
                <div class="paragraph">
                  <p>The following result give a criteria for the stability of fixed points using basic
                    reproductive number $\mathcal{R}^d_0$ <a class="citation" href="#VandenDriessche2017a">[32]</a>.</p>
                </div>
                <div class="theorem" text="Thm.1, p.292 VandenDriessche(2017)">
                  If $\mathcal{R}^d_0 &lt; 1$, then the fixed point FDE of system
                  (1) is locally asymptotically stable.
                </div>
                <div class="paragraph">
                  <p>Besides,  when $\mathcal{R}^d_0&gt;1$ we have the instability
                    of free disease fixed point, but we obtain the stability
                    of the endemic fixed point, and we establish the following result.</p>
                </div>
                <div class="theorem">
                  <p>
                    If $\mathcal{R}^d_0&gt;1$, then the fixed point EE of
                    <a href="#eq:DeterministicSystem">
                      System (1)
                    </a>
                    is locally asymptotically stable.
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div class="sect1">
            <h2 id="sec:stochastic_extension">Stochastic Extension</h2>
            <div class="sectionbody">
              <div class="paragraph">
                <p>In this section, we derive our stochastic differential equation from
                  system <a href="#eq:DeterministicSystem">(1)</a>. Following ideas from
                  <a class="citation" href="#Schurz2015">[33]</a>,
                  we quantify uncertainty in plant-replanting $r_1$, $r_2$, and vector-die
                  by fumigation $\gamma_f$
                  using a standard Brownian process and the suitable smooth functions.
                  Let $x:=(S_p, I_p, L_p, S_v,I_v)^{\top}$, then we consider the functions
                  $F_i: \mathbb{R} \to \mathbb{R}$, $i \in\{ r_1, r_2, \gamma_f \}$
                  defined by</p>
              </div>
              <div class="paragraph">
                <p>\begin{equation}
                  F_{r_1}(x)
                  :=
                  \sigma_L
                  \frac{S_p}{N_p},
                  \qquad
                  F_{r_2}(x)
                  :=
                  \sigma_I
                  \frac{S_p}{N_p},
                  \qquad
                  F_{\gamma_f}(x)
                  :=
                  \sigma_v.
                  \end{equation}</p>
              </div>
              <div class="paragraph">
                <p>Where $F_i$ are locally Lipschitz-continuous on $\Gamma$. Let
                  $B(t) = (B_p(t), B_v(t))^{\top}$ be a 2-dimensional standard Brownian process
                  defined on a filtered complete probability space
                  $(\Omega,\mathcal{F},\{\mathcal{F}_t\}_{t\geq 0},\P)$.
                  %, and $dB(t)=B(t+dt)-B(t)$ denotes the increment of a standard Brownian process.
                  Thus, we perturb the deterministic model <a href="#eq:DeterministicSystem">(1)</a>
                  replacing the rates $r_1$, $r_2$, and $\gamma_f$ by</p>
              </div>
              <div id="eqn:NoisePerturbation" class="paragraph">
                <p>\begin{equation}
                  \label{eqn:NoisePerturbation}
                  \begin{aligned}
                  r_1 dt \rightsquigarrow r_1 dt
                  + \sigma_L \frac{S_p}{N_p} dB_p(t),
                  \\
                  r_2 dt \rightsquigarrow r_2 dt
                  + \sigma_I \frac{S_p}{N_p} dB_p(t),
                  \\
                  \gamma_f dt \rightsquigarrow \gamma_f dt + \sigma_v dB_v(t).
                  \end{aligned}
                  \end{equation}</p>
              </div>
              <div class="paragraph">
                <p>Note that the right-hand side of <a href="#eqn:NoisePerturbation">(4)</a> denotes a
                  stochastic perturbation of $r_1$, $r_2$, $\gamma_f$.</p>
              </div>
              <div class="paragraph">
                <p>So, we deduce a stochastic extension from deterministic system
                  <a href="#eq:DeterministicSystem">(1)</a> given the following Itô SDE:</p>
              </div>
              <div class="paragraph">
                <p>\begin{equation}
                  \label{sys::StochasticSystem}
                  \begin{aligned}
                  d S_p &amp;=
                  \left(
                  -\beta_p S_p \frac{I_v}{N_v} + r_1 L_p + r_2 I_p
                  \right)dt
                  + \frac{S_p(\sigma_L L_p
                  +
                  \sigma_I I_p)}{N_p}dB_p(t),
                  \\
                  dL_p &amp;=
                  \left(
                  \beta_p S_p \frac{I_v}{N_v} - (b + r_1) L_p
                  \right) dt
                  - \sigma_L \frac{S_pL_p}{N_p} dB_p(t),
                  \\
                  d I_p &amp;=
                  \left(
                  b L_p - r_2 I_p
                  \right) dt
                  - \sigma_I \frac{S_pI_p}{N_p} dB_p(t),
                  \\
                  dS_v &amp;=
                  \left(
                  -\beta_v S_v \frac{I_p}{N_p} - (\gamma + \gamma_f) S_v <br>
                  (1-\theta) \mu	\right)dt - \sigma_v S_v dB_v(t),
                  \\
                  d I_v &amp;=
                  \left(
                  \beta_v S_v \frac{I_p}{N_p} - (\gamma + \gamma_f) I_v + \theta \mu
                  \right) dt
                  - \sigma_v I_v dB_v(t),
                  \\
                  &amp; S_p(0) = S_{p}^{0}, \quad
                  L_p(0) = L_{p}^{0}, \quad
                  I_p(0) = I_{p}^{0}, \quad
                  \\
                  &amp; S_v(0) = S_{v}^{0},
                  I_v(0) = I_{v}^{0}.
                  \end{aligned}
                  \end{equation}</p>
              </div>
            </div>
          </div>
          <div class="sect1">
            <h2 id="sec:solution_existence">Existence and uniqueness of positive Solution</h2>
            <div class="sectionbody">
              <p>
                Here, we prove the existence of a unique positive solution.
                Since we study the asymptotic behavior of a biological model,
                we have to assure the existence of unique-globally-positive
                invariant solution of SDE
                \eqref{sys::StochasticSystem}. The following result prove that
                this set is positive invariant-- see supplementary material.
              </p>
              <div class="theorem" id="thm::existence-unique">
                For any initial values
                $
                (S_p(0), L_p(0), I_p(0), S_v(0), I_v(0))^{\top}
                \in \Gamma
                $,
                exists a unique a.s. invariant global positive solution to SDE
                \eqref{sys::StochasticSystem} in $\Gamma$, that is, the solution
                $x(t) = (S_p(t), L_p(t), I_p(t), S_v(t), I_v(t)) ^\top$, satisfies
                \begin{equation*}
                \mathbb{P}
                \Big\{
                x(t)
                \in
                \Gamma, \
                \forall t \geq 0
                \Big\} = 1.
                \end{equation*}
              </div>
              <div class="proof">
                <p>
                  To prove existence of global solution, note that the right-hand side of
                  \eqref{sys::StochasticSystem} are second-order polynomials, are locally
                  Lipschitz continuous. By Theorem 3.4 <a class="citation" href="#Mao2008">[34, P. 56]</a>,
                  there exists a unique local solution
                  $(S_p(t),L_p(t),I_p(t),S_v(t),I_v(t))^\top$
                  defined on $[0,\tau_e)$, where $\tau_e$ is the explosion time.
                  We will show $\tau_e = \infty$ a.s.
                </p>
                <p>
                  Let $k_0&gt;0$ such that
                  $$
                  (S_p(0),L_p(0),I_p(0),S_v(0),I_v(0))^\top \in
                  \left(
                  \frac{1}{k_0},k_0\right)
                  \times
                  \dots
                  \times
                  \left(\frac{1}{k_0},k_0
                  \right)=
                  \left(\frac{1}{k_0},k_0\right)^5.
                  $$
                  For any $k&gt;0$ with $k&gt;k_0$, define
                  \begin{equation*}
                  \tau_k :=
                  \inf
                  \left \{
                  t\in [0,\tau_e)
                  :(S_p(t),L_p(t),I_p(t),S_v(t),I_v(t))^\top
                  \not \in
                  \left(
                  \frac{1}{k},k
                  \right)^5
                  \right\}.
                  \end{equation*}
                  Note that $\tau_k$ is an increasing sequence, because
                  $\left(1/k,k\right)$ is an increasing sequence. Moreover,
                  $\tau_k \leq \tau_e$. Then $\tau_k$ has a limit, denote by
                  $\tau_{\infty}$. We will show that $\tau_{\infty}=\infty$;
                  this implies $\tau_e = \infty$ and we conclude hat system
                  \eqref{sys::StochasticSystem} has a unique solution and will
                  remain in $\Gamma$ with probability one.
                </p>
                <p>
                  We proceed by a contradiction. Suppose $\tau_{\infty}&lt;\infty$.
                  Thus, there exists $T&gt;0$ such that
                  $$
                  \mathbb{P}
                  \Big\{
                  \tau_{\infty} &lt; T
                  \Big\}
                  &gt;\epsilon, \quad \forall
                  \epsilon \in (0,1).
                  $$
                  Thus, there exists $k_1&gt;k_0$ such that
                  $\mathbb{P} \Big\{ \tau_k &lt; T \Big\} &gt;\epsilon$ for all
                  $k \geq k_1$.
                </p>
                <p>
                  Consider the following Lyapunov function $V:\mathbb{R}^5 \to \mathbb{R}$
                  defined by
                  \begin{equation*}
                  \begin{aligned}
                  &amp; V (S_p, L_p, I_p, S_v, I_v) :=
                  \left(S_p - N_p - N_p \log{\frac{S_p}{N_p}}\right) +
                  \left(L_p - N_p - N_p \log{\frac{L_p}{N_p}}\right)
                  \\
                  &amp;+
                  \left(I_p - N_p - N_p \log{\frac{I_p}{N_p}}\right) +
                  \left(S_v - N^{\infty}_v - N^{\infty}_v
                  \log{\frac{S_v}{N^{\infty}_v}}\right)
                  \\
                  &amp;+
                  \left(I_v - N^{\infty}_v - N^{\infty}_v
                  \log{\frac{I_v}{N^{\infty}_v}}\right).
                  \end{aligned}
                  \end{equation*}
                  Applying the diffusion operator $\mathcal{L}$ see <a class="citation" href="#Mao2008">[34, P. 12]</a>, to
                  $$
                  f(x) = x - N - N\
                  \log{
                  \left( \frac{x}{N} \right)
                  },
                  $$
                  we get
                  \begin{align*}
                  &amp;\mathcal{L}[V(S_p,L_p,I_p,S_v,I_v)] =
                  \left(
                  1 - \frac{N_p}{S_p}
                  \right) \left(
                  -\frac{\beta_p}{N_v} S_p I_v + r_1 L_p + r_2 I_p
                  \right)
                  \\
                  &amp;+
                  \left(
                  1 - \frac{N_p}{L_p}
                  \right) \left(
                  \frac{\beta_p}{N_v} S_p I_v -(b + r_1) L_p
                  \right) + \left(
                  1 - \frac{N_p}{I_p}
                  \right) \left(
                  b L_p - r_2 I_p
                  \right)
                  \\
                  &amp;+
                  \left(
                  1 - \frac{N^{\infty}_v}{S_v}
                  \right) \left(
                  -\frac{\beta_v}{N_p} S_v I_p - (\gamma + \gamma_f) S_v +
                  (1- \theta) \mu
                  \right)
                  \\
                  &amp;+ \left(
                  1 - \frac{N^{\infty}_v}{I_v}
                  \right) \left(
                  -\frac{\beta_v}{N_p} S_v I_p -
                  (\gamma + \gamma_f) I_v + \theta \mu
                  \right)
                  \\
                  &amp;+
                  \frac{1}{2}\frac{S_p ^2}{N_p ^ 2}
                  \frac{N_p}{S_p ^ 2} (\sigma_L L_p + \sigma_I I_p) ^ 2
                  + \frac{1}{2} \sigma_L ^2
                  \frac{S_p ^2}{N_p ^2} L_p^2 \frac{N_p}{L_p ^2}
                  + \frac{1}{2} \sigma_I ^2
                  \frac{S_p ^2}{N_p ^2} I_p^2 \frac{N_p}{I_p ^2}
                  \\
                  &amp;+
                  \frac{1}{2} \sigma_v ^2 S_v ^2 \frac{N^{\infty}_v}{S_v ^2}
                  + \frac{1}{2} \sigma_v ^2 I_v ^2 \frac{N^{\infty}_v}{I_v ^2}.
                  \end{align*}
                  Bounding the above equality by positive terms, we have
                  \begin{equation}\label{eqn::boudn1positvesol}
                  \begin{aligned}
                  &amp;\mathcal{L}[V(S_p,L_p,I_p,S_v,I_v)] \leq
                  \frac{\beta_p}{N_v} S_p I_v + r_1 L_p + r_2 I_p
                  + \frac{\beta_p}{N_v} S_p I_v
                  + (b + r_1) L_p
                  \\
                  &amp;+
                  b L_p + r_2 N_p + (1 - \theta) \mu
                  + \frac{\beta_v}{N_p} S_v I_p
                  + (\gamma + \gamma_f) N^{\infty}_v
                  + \frac{\beta_v}{N_p} S_v I_p + \theta \mu
                  \\
                  &amp;+ (\gamma + \gamma_f) N^{\infty}_v
                  + \frac{1}{2} \frac{1}{N_p} (\sigma_L L_p + \sigma_I I_p)^2
                  + \frac{1}{2}\sigma_L ^2
                  \frac{S_p ^2}{N_p} + \frac{1}{2}\sigma_I ^2
                  \frac{S_p ^2}{N_p}
                  \\
                  &amp;+ \frac{1}{2}\sigma_v ^2 N^{\infty}_v
                  + \frac{1}{2} \sigma_v ^2 N^{\infty}_v.
                  \end{aligned}
                  \end{equation}
                  Using size population, we bound \eqref{eqn::boudn1positvesol} by
                  \begin{equation}\label{eqn::boudn2positvesol}
                  \begin{aligned}
                  &amp;\mathcal{L}[V(S_p,L_p,I_p,S_v,I_v)] \leq
                  \beta_p N_p + r_1 N_p + r_2 N_p + \beta_p N_p
                  + (b + r_1) N_p
                  \\
                  &amp;+
                  b N_p + r_2 N_p + (\gamma + \gamma_f) N^{\infty}_v
                  + \beta_v N^{\infty}_v
                  + (\gamma + \gamma_f) N^{\infty}_v +\beta_v N^{\infty}_v +
                  (\gamma + \gamma_f) N^{\infty}_v
                  \\
                  &amp;+ (\gamma + \gamma_f) N^{\infty}_v
                  + (\gamma + \gamma_f) N^{\infty}_v
                  + \frac{1}{2} (\sigma_L + \sigma_I)^2 N_p
                  + \frac{1}{2}\sigma_L ^2
                  N_p + \frac{1}{2} \sigma_I ^2 N_p
                  \\
                  &amp;+
                  \frac{1}{2}\sigma_v ^2 N^{\infty}_v
                  + \frac{1}{2} \sigma_v ^2 N^{\infty}_v.
                  \end{aligned}
                  \end{equation}
                  Factorizing $N_p$ and $N^{\infty}_v$ in relation \eqref{eqn::boudn2positvesol},
                  we get
                  \begin{equation*}\label{eqn::boudn3positvesol}
                  \begin{aligned}
                  \mathcal{L}[V(S_p,L_p,I_p,S_v,I_v)] &amp;
                  \\
                  \leq &amp;
                  \left(
                  2 \beta_p + 2 r_1 + 2 r_2 + 2 b +
                  \frac{1}{2} (\sigma_L + \sigma_I)^2
                  + \frac{1}{2}\sigma_L ^2
                  + \frac{1}{2} \sigma_I ^2
                  \right) N_p
                  \\
                  + &amp;
                  [
                  2 \beta_v  + 4 (\gamma + \gamma_f) + \sigma_v ^2
                  ]
                  N^{\infty}_v.
                  \end{aligned}
                  \end{equation*}
                  Define
                  \begin{equation*}\label{eqn::boudn4positvesol}
                  \begin{aligned}
                  C_1  := &amp;
                  \left(
                  2 \beta_p + 2 r_1 + 2 r_2 + 2 b +
                  \frac{1}{2} (\sigma_L + \sigma_I)^2
                  + \frac{1}{2}\sigma_L ^2
                  + \frac{1}{2} \sigma_I ^2
                  \right) N_p
                  \\
                  + &amp;
                  [
                  2 \beta_v  + 4 (\gamma + \gamma_f) + \sigma_v ^2
                  ] N^{\infty}_v.
                  \end{aligned}
                  \end{equation*}
                  Then by Ito's formula, we obtain
                  \begin{equation}\label{eqn::boudn5positvesol}
                  \begin{aligned}
                  &amp;d[V(S_p,L_p,I_p,S_v,I_v)] \leq
                  C_1 dt + \left(
                  1- \frac{N_p}{S_p}
                  \right)\left[\frac{S_p}{N_p}(
                  \sigma_L L_p + \sigma_I I_p)\right]dB_p(t)
                  \\
                  &amp;+
                  \left(1 - \frac{N_p}{L_p}\right)
                  \left(
                  - \sigma_L \frac{S_p}{N_p} L_p
                  \right)dB_p(t)
                  + \left(1 - \frac{N_p}{I_p}\right)
                  \left(
                  - \sigma_I \frac{S_p}{N_p} I_p
                  \right)dB_p(t)
                  \\
                  &amp;+
                  \left(1 - \frac{N^{\infty}_v}{S_v}\right)
                  \left(- \sigma_v S_v\right)dB_v(t) +
                  \left(1 - \frac{N^{\infty}_v}{I_v}\right)
                  \left(- \sigma_v I_v\right)dB_v(t).
                  \end{aligned}
                  \end{equation}
                </p>
                <p>
                  Bounding \eqref{eqn::boudn5positvesol} by positive terms, we obtain
                  \begin{equation}\label{eqn::boudn6positvesol}
                  \begin{aligned}
                  &amp;d[V(S_p,L_p,I_p,S_v,I_v)] \leq
                  C_1 dt + \left[\frac{S_p}{N_p}
                  (\sigma_L L_p + \sigma_I I_p)\right]dB_p(t)
                  \\
                  &amp;+
                  (\sigma_L + \sigma_I) dB_p(t)
                  + 2 \sigma_v N^{\infty}_v dB_p(t).
                  \end{aligned}
                  \end{equation}
                  Integrating both sides of \eqref{eqn::boudn6positvesol} on
                  $(0,\tau_k \wedge T)$, and taking expectation, we have
                  \begin{equation*}\label{eqn::boudn7positvesol}
                  \begin{aligned}
                  &amp;\E{V(S_p,L_p,I_p,S_v,I_v)} \leq
                  V(S_p(0),L_p(0),I_p(0),S_v(0),I_v(0))
                  \\
                  &amp;+
                  \E{\int_0^{\tau_k \wedge T} C_1 dt}
                  + \E{\int_0^{\tau_k \wedge T}
                  \left[\frac{S_p}{N_p}
                  (\sigma_L L_p + \sigma_I I_p)\right]dB_p(t)}
                  \\
                  &amp;+
                  \E{\int_0^{\tau_k \wedge T}
                  (\sigma_L + \sigma_I) dB_p(t)} +
                  2 \E{ \int_0^{\tau_k \wedge T}
                  \sigma_v N^{\infty}_v dB_p(t)}.
                  \end{aligned}
                  \end{equation*}
                  Since $G \in L^2(0, T)$, then $\E{\int_0^T G dB(s)}=0$. Using the above argument, we obtain
                  \begin{equation*}\label{eqn::boudn8positvesol}
                  \begin{aligned}
                  &amp;\E{V(S_p,L_p,I_p,S_v,I_v)} \leq
                  V(S_p(0),L_p(0),I_p(0),S_v(0),I_v(0))
                  + C_1 T.
                  \end{aligned}
                  \end{equation*}
                  For $k \geq 1$ such that $k \geq k_0$, let
                  $\Omega_k = \{ \tau_k \leq T\}$. Then,	$\probX{}{\Omega_k} &gt; \epsilon$.
                  If $t \in \Omega_k$, thus at least one of the following	will hold true:
                  \begin{align*}
                  S_p(t) \not \in &amp;\left( \frac{1}{k}, k\right), \qquad L_p(t) \not \in
                  \left( \frac{1}{k}, k\right), \qquad  I_p(t) \not \in
                  \left( \frac{1}{k}, k\right), \\
                  &amp;S_v(t) \not \in \left( \frac{1}{k}, k\right), \qquad  I_v(t) \not \in
                  \left( \frac{1}{k}, k\right).
                  \end{align*}
                  Note that, $f(x) = x - N_{\bullet} - N_{\bullet}\log{x/N_{\bullet}}$ is
                  increasing on $(N_{\bullet},\infty)$ and decreasing on $(0,N_{\bullet})$
                  it follows
                  \begin{align*}
                  &amp;V(S_p(\tau_k \wedge T),L_p(\tau_k \wedge T),I_p(\tau_k \wedge T),
                  S_v(\tau_k \wedge T),I_v(\tau_k \wedge T)) \geq
                  \\
                  &amp;
                  \left(k - N_p - N_p \log{\frac{k}{N_p}}\right)^3
                  \wedge \left(\frac{1}{k} - N_p - N_p
                  \log{\frac{1}{N_p k}}\right)^3 \wedge
                  \\
                  &amp;
                  \left(k - N^{\infty}_v - N^{\infty}_v
                  \log{\frac{k}{N^{\infty}_v}}\right)^2
                  \wedge \left(\frac{1}{k} - N^{\infty}_v - N^{\infty}_v
                  \log{\frac{1}{N^{\infty}_v k}}\right)^2.
                  \end{align*}
                  Now we have
                  \begin{align*}
                  &amp; V(S_p(0),L_p(0),I_p(0),S_v(0),I_v(0))
                  + C_1 T
                  \geq \E{V(S_p,L_p,I_p,S_v,I_v)}
                  \\
                  &amp; \geq
                  \epsilon
                  \left(
                  k - N_p - N_p
                  \log{\frac{k}{N_p}}
                  \right)^3
                  \wedge
                  \left(
                  \frac{1}{k} - N_p
                  - N_p \log{\frac{1}{N_p k}}
                  \right)^3 \wedge
                  \\
                  &amp;
                  \left(
                  k - N^{\infty}_v - N^{\infty}_v
                  \log{\frac{k}{N^{\infty}_v}}
                  \right)^2
                  \wedge
                  \left(
                  \frac{1}{k} - N^{\infty}_v - N^{\infty}_v
                  \log{\frac{1}{N^{\infty}_v k}}
                  \right)^2.
                  \end{align*}
                  Finally, letting $ k \to \infty$ we obtain
                  \begin{equation*}
                  \infty &gt; V(S_p(0),L_p(0),I_p(0),S_v(0),I_v(0)) + C_1 T \geq \infty,
                  \end{equation*}
                  which is a contradiction. Thus, we conclude that $\tau_\infty = \infty$.
                  <span class="right">
                    $\square$
                  </span>
                </p>
              </div>
            </div>
          </div>
          <div class="sect1">
            <h2 id="sec:extinction">Extinction</h2>
            <div class="sectionbody">
              <div class="paragraph">
                <p>In this section, we deduce conditions over the noise amplitude and
                  a parameter threshold such that the disease will be extinguished with
                  probability one. According to the deterministic threshold, if
                  $\mathcal{R}^d_0&lt;1$, then  the solution tends to free-disease fixed
                  point. In the other hand, the solution process of our SDE can reach
                  this fixed point at least in two different ways. First, we obtain
                  conditions for the noise amplitude.</p>
              </div>
              <div class="definition" id="def::ExponentialStability">
                <p>
                  The free-disease fixed point of system
                  \eqref{sys::StochasticSystem}
                  is almost surely exponentially stable if
                  \begin{equation}
                  \label{eqn::ExponentialStability}
                  \begin{aligned}
                  \limsup_{t \to \infty}
                  \frac{1}{t}
                  \ln(L_p + I_p) &lt; 0 \quad \mathrm{and} \quad
                  &amp;
                  \limsup_{t \to \infty}
                  \frac{1}{t}\ln(I_v) &lt; 0 \qquad \mathrm{a.s.}
                  \end{aligned}
                  \end{equation}
                </p>
              </div>
              <div class="paragraph">
                <p>The following result gives conditions on the parameters to obtain
                  disease extinction.</p>
              </div>
              <div class="theorem" id="thm::NoiseExtinction">
                <p>
                  For all initial condition
                  $(S_p(0), L_p(0), I_p(0), S_v(0), I_v(0)) ^ \top \in \Gamma$. If
                  \begin{equation}\label{eqn::NoiseCond}
                  \begin{aligned}
                  \sigma^2_I &gt; \frac{\sigma^2_L
                  r^2_2}{2\sigma^2_L(r_1-2\beta_p)-\beta^2_p},
                  \qquad
                  \sigma^2_v &gt; \frac{\beta^2_v}
                  {2[(\gamma + \gamma_f)-(\theta\mu+\beta_v)]},
                  \end{aligned}
                  \end{equation}
                  then, the disease will exponentially extinguish with probability
                  one. That is,
                  \begin{equation}
                  \begin{aligned}
                  \limsup_{t \to \infty}
                  \frac{1}{t}
                  \ln(L_p + I_p) &lt; 0 \quad \mathrm{and} \quad
                  &amp;
                  \limsup_{t \to \infty}
                  \frac{1}{t}\ln(I_v) &lt; 0 \qquad \mathrm{a.s.}
                  \end{aligned}
                  \end{equation}
                </p>
              </div>
              <div class="proof">
                <p>
                  The main idea is to apply It\^{o}'s formula to a convenient function and
                  deduce conditions. Let
                  $
                  V(S_p, L_p, I_p) = \ln(L_p + I_p)
                  $, then the It\^{o}'s formula gives
                  \begin{align*}
                  d \ln(L_p+I_p)
                  =&amp;
                  \left(
                  \frac{1}{L_p + I_p}
                  \right)
                  \left(
                  \frac{\beta_p}{N_v}
                  S_p I_v - (b + r_1) L_p
                  -\frac{1}{2}
                  \sigma_L^2 \frac{L_p^2}{(L_p+I_p)^2}
                  \right)dt
                  \\
                  &amp;-
                  \sigma_L \frac{L_p}{L_p+I_p}dB_p(t)
                  \\
                  &amp;\leq
                  \left(
                  \frac{1}{L_p+I_p}
                  \right)
                  \left(
                  \beta_p S_p - (b + r_1) -
                  \frac{1}{2}
                  \sigma_L^2
                  \frac{L_p^2}{(L_p+I_p)^2}
                  \right)dt
                  \\
                  &amp;-
                  \sigma_L \frac{L_p}{L_p + I_p} dB_p(t).
                  \\
                  \end{align*}
                  Let $x:=\dfrac{L_p}{L_p + I_p}$, then
                  \begin{align*}
                  d \ln(L_p + I_p)
                  &amp;\leq
                  \left(
                  \beta_p
                  \frac{S_p}{L_p + I_p} -
                  (b + r_1) -
                  \frac{1}{2}
                  \sigma_L ^ 2 x^2
                  \right)
                  dt - \sigma_L x dB_p(t)
                  \\
                  &amp;\leq
                  \left(\beta_p
                  \frac{N_p}{L_p + I_p} - (b+r_1) -
                  \frac{1}{2}
                  \sigma_L^2 x^2
                  \right) dt -
                  \sigma_L xdB_p(t)
                  \\
                  &amp;
                  \leq
                  \left(
                  \beta_p x + 2\beta_p -
                  (b + r_1) -
                  \frac{1}{2}
                  \sigma_L ^ 2 x^2
                  \right) dt -
                  \sigma_L xdB_p(t)
                  \\
                  &amp;=
                  \left(
                  -\frac{1}{2}
                  \sigma_L ^ 2 x ^ 2 +
                  \beta_p x + 2 \beta_p -
                  (b + r_1)
                  \right) dt -\sigma_L x
                  dB_p(t).
                  \end{align*}
                  Hence,
                  \begin{align*}
                  \ln(L_p+I_p)
                  &amp;\leq
                  -\frac{\sigma_L ^ 2}{2}
                  \int_{0} ^ {t}
                  \left[
                  \left(
                  x -
                  \frac{\beta_p}{\sigma_L ^ 2}
                  \right) ^ 2 +
                  \frac{\beta_p ^ 2}{2 \sigma_L ^ 2} +
                  2 \beta_p - (b + r_1)
                  \right] du
                  \\
                  &amp;-
                  \int_{0} ^ {t}
                  \sigma_L x dB_p(u) +
                  \ln(L_p(0) + I_p(0)),
                  \end{align*}
                  which implies,
                  \begin{equation}
                  \label{eqn::ItoForBound}
                  \begin{aligned}
                  \frac{1}{t}\ln(L_p+I_p)
                  &amp;\leq
                  -\frac{\sigma_L^2}{2t}
                  \int_{0}^{t}
                  \left(
                  x -
                  \frac{\beta_p}{\sigma_L^2}
                  \right) ^ 2 du +
                  \frac{\beta_p^2}{2\sigma_L^2} -
                  (b + r_1) + 2\beta_p
                  \\
                  &amp;-
                  \frac{1}{t}
                  \int_{0}^{t}
                  \sigma_L x dB_p(u) +
                  \frac{1}{t} \ln(S_p(0)+L_p(0)+I_p(0)),
                  \end{aligned}
                  \end{equation}
                  let
                  $$
                  M_t :=
                  \frac{1}{t}
                  \int_{0}^{t}
                  \sigma_L x dB_p(t) +
                  \frac{1}{t} \ln(L_p(0)+I_p(0)) .
                  $$
                  Since the integral in the term $M_t$ is a martingale, the strong law of
                  large numbers for martingales <a class="citation" href="#Mao2008">[34, Pp. 12-14 Thm. 3.4]</a>
                  implies that
                  \begin{equation*}
                  \lim
                  \limits_{t \to \infty} M_t = 0\,\,
                  \mbox{a.s.}
                  \end{equation*}
                  Thus, from relation \eqref{eqn::ItoForBound} we obtain
                  \begin{align}
                  \label{eqn::Bound1}
                  \limsup_{t\infty \to \infty}
                  \frac{1}{t}
                  \ln(L_p + I_p) &lt;
                  \frac{\beta_p^2}{2\sigma_L^2} +
                  2\beta_p - (b + r_1).
                  \end{align}
                  A similar argument also shows that
                  \begin{align}\label{eqn::Bound2}
                  \limsup_{t\infty \to \infty}
                  \frac{1}{t}
                  \ln(L_p + I_p) &lt;
                  \frac{r_2 ^ 2}{2 \sigma_I ^ 2} + b.
                  \end{align}
                  Through by \eqref{eqn::Bound1} and \eqref{eqn::Bound2}, we obtain
                  \begin{align*}
                  \limsup_{t\infty \to \infty}
                  \frac{1}{t}
                  \ln(L_p + I_p)
                  &lt;
                  \frac{\beta_p^2}{2\sigma_L^2} +
                  \frac{r_2^2}{2 \sigma_I ^ 2} +
                  2\beta_p - r_1,
                  \end{align*}
                  and for infected vector we obtain
                  \begin{align*}
                  \limsup_{t\infty \to \infty}
                  \frac{1}{t}
                  \ln(I_v)
                  &lt;
                  \frac{\beta_v^2}{2\sigma_v^2}
                  +
                  \beta_v - (\gamma + \gamma_f) + \theta \mu .
                  \end{align*}
                  Therefore by relation \eqref{eqn::NoiseCond}, implies that
                  \begin{equation*}
                  \limsup_{t \to \infty}
                  \frac{1}{t}
                  \ln(L_p + I_p) &lt; 0 \quad \mathrm{ and } \quad
                  \limsup_{t \to \infty}
                  \frac{1}{t}\ln(I_v)&lt; 0\qquad \mathrm{a.s.}
                  \end{equation*}
                </p>
                <span class="right">
                  $\square$
                </span>
              </div>
              <div class="remark">
                <p>
                  The
                  <a href="#thm::NoiseExtinction">
                    above Theorem
                  </a>
                  shows that if the noise intensities are large enough.
                  Then the infected population tends to die out.
                </p>
              </div>
              <p>
                Now, we define a stochastic version of basic reproductive number, we use ideas from
                <a class="citation" href="#Agarwal2019">[14]</a>. We postulate the following parameter \eqref{sys::StochasticSystem}
                as,
                \begin{equation}
                \label{eqn::StochasticBRN}
                \mathcal{R}^s_0 :=
                \mathcal{R}^{d}_0
                -\frac{1}{2}
                \left[
                \left(
                \sigma_L
                +\sigma_I
                \right)^2
                - \frac{\sigma^2_v} {
                \beta_v + \theta (\gamma + \gamma_f) + \sigma_v^2
                }
                \right].
                \end{equation}
              </p>
              <p>
                From \eqref{eqn::StochasticBRN}, we deduce two cases that depend on
                noise intensities:
                $\mathcal{R}^{s}_0 &lt; \mathcal{R}^{d}_0$ and
                ${\mathcal{R}^{s}_0 &gt; \mathcal{R}^{d}_0}$.
                We define these cases of our stochastic reproductive number according
                to the deterministic threshold let
                $\mathcal{R}^s_0$ as follows:
              </p>
              <ol>
                <li>
                  $\mathcal{R}^s_s$,
                  when $\mathcal{R}_0^s &lt; \mathcal{R}_0^d$,
                </li>
                <li>
                  and $\mathcal{R}^s_a$,
                  when $\mathcal{R}_0^s &gt; \mathcal{R}_0^d$.
                </li>
              </ol>
              From here $\bullet$ denotes the index $\{a, s\}$. For example,
              in the following theorem, if ${\mathcal{R}^s_{\bullet} &lt; 1}$,
              then infected populations tend to become extinct.
              <div class="theorem" id="thm::Rs0Extinction">
                Let $(S_p(t),L_p(t),I_p(t), S_v(t),I_v(t))^\top$
                be the solution of SDE \eqref{sys::StochasticSystem}
                with initial values
                $
                (S_p(0),L_p(0),I_p(0),S_v(0),\\ I_v(0)) ^ \top \in \Gamma
                $,
                \quad
                $r_2 \geq r_1$,
                and \quad
                $\beta_p \beta_v &gt; 2 \mu r_2$.
                If \quad ${0 \leq \mathcal{R}^s_{\bullet}&lt;1}$,
                then the infection tends to zero exponentially, a.s.
              </div>
              <div class="proof">
                Consider the following Lyapunov function:
                \begin{equation*}
                \begin{aligned}
                V(S_p,L_p,I_p,S_v,I_v)
                &amp;=
                w_0 \left(
                S_p-N_p-N_p\log{\frac{S_p}{N_p}}
                \right)
                +w_1 L_p + w_2 I_p
                \\
                &amp;+
                w_3 \left(S_v-N^{\infty}_v-N^{\infty}_v
                \log{\frac{S_v}{N^{\infty}_v}}\right)
                + w_4 I_v,
                \end{aligned}
                \end{equation*}
                where
                \begin{align*}
                &amp;
                w_0 = w_1 = \frac{1}{N_p},
                \\
                &amp;
                w_2 =
                \frac{1}{b}
                \left\{
                \frac{ \mathcal{R}^{d}_0-1} {2 N_p}
                -\frac{1}{N_p}
                \left[
                r_2+ b +\beta_p +
                \left(
                \sigma_L
                + \sigma_I
                \right) ^ 2
                \right]
                \right\},
                \\
                &amp;
                w_3 =
                w_4 =
                \frac{\mathcal{R}^{d}_0 - 1}{2 N^{\infty}_v }
                \frac{1}{\beta_v + \theta (\gamma + \gamma_f) + \sigma_v^2}.
                \end{align*}
                Computing the diffusion operator $\mathcal{L} V$
                <a class="citation" href="#Mao2008">[34, P. 110]</a>, we obtain
                \begin{equation*}
                \begin{aligned}
                \mathcal{L}V
                &amp;=
                w_0 \left(
                \frac{S_p-N_p}{S_p}
                \right)
                \left(
                -\frac{\beta_p}{N_v} S_p I_v
                + r_1 L_p + r_2 I_p
                \right)
                \\
                &amp;+
                w_1 \left[
                \frac{\beta_p}{N_v}S_p I_v -(b+r_1)L_p
                \right]
                +
                w_2 \left(
                b L_p -r_2 I_p
                \right)
                \\
                &amp;+
                w_3 \left(
                \frac{S_v-N^{\infty}_v}{S_v}
                \right)
                \left[
                -\frac{\beta_v}{N_p}S_v I_p
                -(\gamma + \gamma_f) S_v + (1 - \theta) \mu
                \right]
                \\
                &amp;+
                w_4 \left[
                \frac{\beta_v}{N_p}S_v I_p -
                (\gamma + \gamma_f) I_v +\theta \mu
                \right]
                +
                \frac{w_0}{2}
                \left[
                \frac{S_p}{N_p}
                (\sigma_L L_p+\sigma_I I_p)
                \right]^2
                \\
                &amp;+
                \frac{w_3}{2} \sigma_v^2 N ^ {\infty}_{v}.
                \end{aligned}
                \end{equation*}
                By conservation law \eqref{eqn:plant_size},
                $L_p = N_p-S_p-I_p$, we have
                \begin{equation*}
                \begin{aligned}
                \mathcal{L}V
                &amp;=
                w_0 \left(
                \frac{S_p-N_p}{S_p}
                \right)
                \left[
                -\frac{\beta_p}{N_v}
                S_p I_v + r_1 N_p - r_1 S_p
                + (r_2 - r_1) I_p
                \right]
                \\
                &amp;+
                w_1 \left[
                \frac{\beta_p}{N_v}S_p I_v
                - (b+r_1) (N_p - S_p -I_p)
                \right]
                \\
                &amp;+
                w_2
                \left[
                b ( N_p - S_p -I_p) -r_2 I_p
                \right]
                \\
                &amp;+
                w_3
                \left(
                \frac{S_v-N_v}{S_v}
                \right)
                \left[
                -\frac{\beta_v}{N_p} S_v I_p
                -(\gamma + \gamma_f) S_v + (1-\theta) \mu
                \right]
                \\
                &amp;+
                w_4
                \left[
                \frac{\beta_v}{N_p} S_v I_p
                -(\gamma + \gamma_f) I_v +\theta \mu
                \right]
                \\
                &amp;+
                \frac{w_0}{2}
                \left\{
                \frac{S_p}{N_p}[\sigma_L ( N_p - S_p -I_p)
                +\sigma_I I_p]\right\}^2
                + \frac{w_3}{2} \sigma_v^2 N^{\infty}_v.
                \end{aligned}
                \end{equation*}
                Expanding the above equality, we get
                \begin{equation*}
                \begin{aligned}
                \mathcal{L} V
                &amp;\leq
                - w_0 \left(
                S_p-N_p
                \right)\frac{\beta_p}{N_v} I_v  -r_1 w_0
                \frac{\left(S_p-N_p\right)^2}{S_p}
                + w_0 (r_2-r_1) \left(
                \frac{S_p-N_p}{S_p}
                \right) I_p
                \\
                &amp;+
                w_1
                \frac{\beta_p}{N_v}S_p I_v
                - w_1(b+r_1) (N_p - S_p -I_p)
                +
                w_2 b ( N_p - S_p -I_p) - w_2 r_2 I_p
                \\
                &amp;-
                w_2 r_2 I_p -
                w_3 \left(
                S_v - N^{\infty}_v
                \right)\frac{\beta_v}{N_p} I_p -(\gamma + \gamma_f) w_3
                \frac{\left(S_v- N^{\infty}_v\right)^2}{S_v}
                + w_4 \frac{\beta_v}{N_p}S_v I_p
                \\
                &amp;-
                w_4(\gamma + \gamma_f) I_v
                + w_4\theta \mu
                + \frac{w_0}{2}
                \left(
                \sigma_L + \sigma_I
                \right)^2 N_p + \frac{w_3}{2} \sigma_v^2 N^{\infty}_v.
                \end{aligned}
                \end{equation*}
                Factorizing similar terms, we obtain
                \begin{equation*}
                \begin{aligned}
                \mathcal{L}V
                &amp;\leq
                -r_1 w_0
                \frac{\left(S_p-N_p\right)^2}{S_p}
                -(\gamma + \gamma_f) w_3
                \frac{\left(S_v-N^{\infty}_v\right)^2}{S_v} +
                (w_1- w_0) \frac{\beta_p}{N_v} S_p I_v
                \\
                &amp;+
                (w_1- w_0) \frac{\beta_p}{N_v} S_p I_v+
                (w_4- w_3) \frac{\beta_v}{N_p} S_v I_p
                \\
                &amp;+
                \left[w_0(r_2-r_1) + w_1(b+r_1)- w_2(b+r_2)
                + w_3 \frac{\beta_v N^{\infty}_v}{N_p}\right]I_p
                \\
                &amp;+
                \left[w_0\frac{\beta_p}{N_v}N_p
                -( \gamma + \gamma_f) w_4
                \right] I_v + [w_2 b - w_1 (b+ r_1)](N_p-S_p)
                \\
                &amp;-
                w_0 (r_2- r_1)\frac{N_p}{S_p} I_p
                + w_4 \theta \mu +
                \frac{w_0}{2}
                \left(
                \sigma_L
                +\sigma_I \right)^2 N_p
                + \frac{w_3}{2} \sigma_v^2 N^{\infty}_v.
                \end{aligned}
                \end{equation*}
                Since $w_0 = w_1$, $w_3 = w_4$, above inequality reduce to
                \begin{equation*}
                \begin{aligned}
                \mathcal{L}V
                &amp;\leq
                -r_1 w_0
                \frac{\left(S_p-N_p\right)^2}{S_p}
                -(\gamma + \gamma_f) w_3
                \frac{\left(S_v-N^{\infty}_v\right)^2}{S_v}
                \\
                &amp;+
                \left[w_1(r_2+ b )- w_2(b+r_2)
                + w_3 \frac{\beta_v N^{\infty}_v}{N_p}\right]I_p
                \\
                &amp;+
                \left[w_1\frac{\beta_p}{N_v} N_p
                - (\gamma + \gamma_f) w_3\right]I_v
                +
                \left[
                w_2 b - w_1 (b+ r_1)
                \right] (N_p - S_p)
                \\
                &amp; -
                w_1 (r_2 - r_1)
                \frac{N_p}{S_p} I_p+
                w_3 \theta \mu +
                \frac{w_1}{2}
                \left(
                \sigma_L
                +\sigma_I \right)^2 N_p
                + \frac{w_3}{2} \sigma_v^2 N^{\infty}_v.
                \end{aligned}
                \end{equation*}
                Now, we bound by positive terms, and sizes populations we obtain
                \begin{equation*}
                \begin{aligned}
                \mathcal{L}V
                &amp;\leq
                \left\{w_1\left[r_2+ b +\beta_p+
                \left(
                \sigma_L
                +\sigma_I \right)^2 \right]
                + w_2 b\right\}N_p
                \\
                &amp;+
                \left[ \beta_v + \theta (\gamma + \gamma_f)
                + \sigma_v^2\right] w_3 N^{\infty}_v
                - \frac{1}{2} w_1 \left(
                \sigma_L
                +\sigma_I \right)^2 N_p
                -\frac{1}{2} w_3 \sigma_v^2 N^{\infty}_v.
                \end{aligned}
                \end{equation*}
                Substituting $w_2$ and $w_3$, the above inequality becomes
                \begin{equation*}
                \begin{aligned}
                \mathcal{L}V
                &amp;\leq
                \mathcal{R}^{d}_0
                -\frac{1}{2}
                \left[
                w_1
                \left(
                \sigma_L + \sigma_I
                \right)^2 N_p
                +
                \frac{\mathcal{R}^{d}_0-1}{2}
                \frac{\sigma_v^2}
                {
                \beta_v + \theta (\gamma + \gamma_f)
                + \sigma_v^2
                }
                \right] - 1.
                \end{aligned}
                \end{equation*}
                We bound $ \mathcal{L} V$ as
                \begin{equation*}
                \begin{aligned}
                \mathcal{L} V
                &amp;\leq
                \left(
                1 - \frac{1}{2} \frac{\sigma^2_v}
                {\beta_v + \theta (\gamma + \gamma_f) + \sigma_v^2}
                \right)\mathcal{R}^{d}_0
                \\
                &amp;-
                \frac{1}{2}
                \left[
                \left(
                \sigma_L + \sigma_I
                \right)^2
                - \frac{\sigma^2_v}
                {\beta_v + \theta (\gamma + \gamma_f) + \sigma_v^2}
                \right]-1
                \\
                &amp;\leq
                \mathcal{R}^{d}_0
                -\frac{1}{2}\left[
                \left(
                \sigma_L + \sigma_I
                \right)^2
                - \frac{\sigma^2_v}
                {\beta_v + \theta (\gamma + \gamma_f) + \sigma_v^2}
                \right] - 1.
                \end{aligned}
                \end{equation*}
                Using definition of $\mathcal{R}^s_{\bullet}$, we get
                \begin{equation*}
                \begin{aligned}
                \mathcal{L}V
                &amp;\leq
                \mathcal{R}^{s}_{\bullet} - 1.
                \qquad \mbox{a.s.}
                \end{aligned}
                \end{equation*}
                Integrating diffusion operator $\mathcal{L} V$ from 0 to $t$,
                dividing for $t$, and taking expectation, we have
                \begin{equation*}
                \lim_{t \to \infty} \E{\frac{V(t)}{t}}
                \leq \mathcal{R}^{s}_{\bullet} - 1.
                \end{equation*}
                Letting $t \to \infty$ we obtain
                \begin{align*}
                \lim\limits_{t \to\infty}
                (S_p, L_p, I_p, S_v, I_v )^{\top}_{t}
                =
                (N_p, 0, 0, N^{\infty}_v, 0)
                \end{align*}
                exponentially a.s.
                <span class="right">
                  $\square$
                </span>
              </div>
              <div class="remark">
                <p>
                  If the stochastic reproductive number
                  is below one, then
                  <a href="#thm::Rs0Extinction">
                    this theorem
                  </a>
                  shows that infection will extinguish.
                </p>
              </div>
            </div>
          </div>
          <div class="sect1">
            <h2 id="sec:persistence">Persistence</h2>
            <div class="sectionbody">
              <p>
                In this section, we obtain conditions under the solution of the system
                \eqref{sys::StochasticSystem} is disease-persistent. In the deterministic
                case, it is sufficient to obtain $\mathcal{R}^d_0&gt;1$, but our stochastic
                dynamics lacks of endemic fixed point. Then, we will prove that the
                solution of system \eqref{sys::StochasticSystem} oscillates around the
                fixed point $(S_p^{**}, L_p^{**}, I_p ^ {**},S_v^{**},I_v^{**})^\top$,
                with an amplitude depending on the stochastic intensity of the perturbation.
              </p>
              <p>
                We use the ideas, similar to <a class="citation" href="#Zhao2015a">[35]</a>, to obtain oscillations.
                First, we bound the distance between the solution and endemic fixed point
                $EE$. Then, using \eqref{eqn:UpperBound} and Assumption 1, we will prove
                that the inferior limit of time average of system \eqref{sys::StochasticSystem}
                is positive. In addition, let us finish by applying the definition of persistence
                in mean.
              </p>
              <p>
                We use the definition given in (missing reference) to prove the persistence
                of the disease.
              </p>
              <div class="definition" id="def::persistence">
                <p>
                  Let $x = \{S_p,L_p,I_p,S_v,I_v\}$ be the solution of
                  \eqref{sys::StochasticSystem}.
                  The solution process $x$ is persistent in mean if
                  \begin{equation*}
                  \liminf_{t \to \infty}\frac{1}{t}\int_0^t x(r) dr &gt;0,\qquad a.s.
                  \end{equation*}
                </p>
              </div>
              <p>
                We use the following conditions to prove that system
                \eqref{sys::StochasticSystem} is persistence in mean.
              </p>
              <div class="assumption" id="ass:perisistence">
                <p>
                  With notation of system \eqref{sys::StochasticSystem} we ask the following.
                </p>
                <ol class="custom">
                  <li value="(ASS-1)">
                    According to
                    <a href="thm::NoiseExtinction">
                      Thmeorem *.*
                    </a>,
                    we require the following conditions to
                    avoid extinction by noise.
                    \begin{equation}
                    \label{eqn::NoiseBreak}
                    \begin{aligned}
                    \sigma^2_I &lt;
                    \frac{\sigma^2_L r^2_2}{2\sigma^{2}_{L}( r_1 - 2 \beta_p)- \beta^{2}_p},
                    \qquad
                    \sigma^2_v &lt;
                    \frac{\beta^{2}_v}{
                    2 [
                    (\gamma  + \gamma_f)
                    -
                    (\theta\mu + \beta_v)
                    ]
                    },
                    \end{aligned}
                    \end{equation}
                  </li>
                  <li value="(ASS-2)">
                    Case $\mathcal{R}^s_s$
                    \begin{equation*}
                    \label{eqn::RslessthanRd}
                    (\sigma_L+\sigma_I)^2 &gt; \frac{\sigma^2_v}
                    {\beta_v + \theta (\gamma +\gamma_f) + \sigma_v^2}.
                    \end{equation*}
                  </li>
                  <li>
                    Case $\mathcal{R}^s_a$
                    \begin{equation*}\label{eqn::RsgreatherthanRd}
                    (\sigma_L+\sigma_I)^2 &lt; \frac{\sigma^2_v}
                    {\beta_v + \theta (\gamma + \gamma_f)
                    + \sigma_v^2}.
                    \end{equation*}
                  </li>
                  <li>
                    Define
                    \begin{equation*}
                    \begin{aligned}
                    &amp; \alpha_1 =
                    \frac{1}{N_p^2}
                    \left(
                    r_2-\frac{r_2}{\rho_1}
                    \right),
                    \quad
                    \alpha_2 =
                    \frac{1}{N_p^2}
                    \left(
                    b + r_2 -
                    \rho_1 \frac{b + r_2}{2}
                    \right),
                    \\
                    &amp; \alpha_3 =
                    \frac{(\gamma +\gamma_f)}{ (N^{\infty}_v) ^ 2}
                    \left(
                    1 - \frac{1}{2 \rho_2}
                    \right),
                    \quad
                    \alpha_4 =
                    \frac{(\gamma +\gamma_f)}{(N^{\infty}_v)^2}
                    \left(
                    1 - \rho_2
                    \right),
                    \\
                    &amp;
                    C = 2  \frac{\sigma_L}{ N_p} \beta_p
                    +
                    \frac{1}{2}
                    \sigma_L^3
                    \frac{L_p^{**}}{N_p ^ 2}
                    +
                    \frac{1}{2}
                    \sigma_I^2
                    +
                    \frac{1}{2}
                    \sigma_v ^2,
                    \end{aligned}
                    \end{equation*}
                    and conforming to the deterministic endemic
                    point $EE$,
                    suppose that
                    \begin{equation}
                    \label{eqn::mimi}
                    \min
                    \left \{
                    \alpha_1 (S_p ^{**}) ^ 2,
                    \alpha_2 (L_p ^{**}) ^ 2,
                    \alpha_3 (S_v ^{**} ) ^ 2,
                    \alpha_4 (I_v ^{**} ) ^ 2
                    \right\}
                    \geq C.
                    \end{equation}
                  </li>
                </ol>
              </div>
              <p>
                Condition (ASS-1) gives relations for which extinction by noise are broken.
                Condition (ASS-2), assure the condition to use $\mathcal{R}^s_s$. Similar,
                condition (ASS-3), gives a relation to deduce $\mathcal{R}^s_a$. Finally,
                condition (ASS-4), is a lower bound used to prove that system
                \eqref{sys::StochasticSystem} is persistence in mean.
                <a href="#thm::UpperBound">
                  Theorem *.*
                </a>
                characterizes conditions for which superior limit of time
                average of distance between solution and endemic fixed point is bounded.
              </p>
              <div class="theorem" id="thm::UpperBound">
                <p>
                  Let conditions
                  <a href="#ass:perisistence">
                    <strong>
                      (ASS-1)-(ASS-2)
                    </strong>
                  </a>
                  holds, and $\rho_1\in \left( 1,2\right)$,
                  $\rho_2 \in (\frac{1}{2}, 1)$. According to deterministic endemic
                  fixed point	$(S_p^{**}, L_p^{**}, I_p^{**}, S_v^{**}, I_v^{**}) ^ \top$,
                  if $\mathcal{R}^s_{\bullet}&gt;1$, then
                  \begin{equation}\label{eqn:UpperBound}
                  \begin{aligned}
                  &amp;
                  \limsup
                  \limits_{t \to 	\infty}
                  \displaystyle\frac{1}{t}
                  \int_{0}^{t}
                  \left\{
                  \alpha_1
                  (S_p - S_p ^{**}) ^ 2 +
                  \alpha_2
                  (L_p - L_p ^{**}) ^ 2
                  \right.
                  \\
                  &amp;\left.
                  + \alpha_3
                  (S_v - S_v^{**})^2
                  +\alpha_4(I_v-I_v^{**})^2
                  \right\}
                  dr 	\leq C
                  \qquad
                  \mathrm{a.s.,}
                  \end{aligned}
                  \end{equation}
                  where
                  $
                  C = 2 \frac{\sigma_L}{ N_p} \beta_p +
                  \frac{\sigma_L^3}{2}\frac{L^{**}_p}{N_p ^2}
                  + \frac{1}{2}\sigma^2_I + \frac{1}{2} \sigma_v ^2
                  $ .
                </p>
              </div>
              <div class="proof" id="prf:persistence_upper">
                <p>
                  Using the following Lyapunov function,
                  \begin{equation}\label{eqn::Lyapunov1}
                  \begin{aligned}
                  V(S_p,L_p,S_v,I_v)
                  &amp;=
                  c_1
                  \left[
                  L_p - L_p^{**} -
                  L_p^{**}\log
                  \left(
                  \frac{L_p}{L_p^{**}}
                  \right)
                  \right]
                  \\
                  &amp;+
                  \frac{c_2}{2}
                  \left[
                  (S_p - S_p^{**}) +
                  (L_p - L_p^{**})
                  \right]^2
                  \\
                  &amp;+
                  \frac{c_3}{2}
                  \left[
                  (S_v - S_v^{**}) +
                  (I_v - I_v^{**})
                  \right]^2,
                  \end{aligned}
                  \end{equation}
                  where $c_1 = \sigma_L/N_p^2$, $c_2 = 1/N_p^2$, and
                  $c_3 = 1/{N^{\infty}_v}^2$.
                </p>
                <p>
                  We will show that the distance between the solution and the endemic
                  fixed point is bounded.
                </p>
                <p>
                  Since system \eqref{sys::DeterministicSystem} has constant plant population,
                  let $I_p = N_p-S_p-L_p$, then we rewrite system
                  \eqref{sys::DeterministicSystem} as
                  \begin{equation}\label{sys::Stc4dSystem}
                  \begin{aligned}
                  dS_p
                  &amp;=
                  \left(
                  -\beta_pS_p\frac{I_v}{N_v}+r_2 N_p
                  - r_2 S_p + (r_1-r_2)L_p
                  \right)dt
                  \\
                  &amp;+
                  \frac{S_p}{N_p}\left\{
                  \sigma_L L_p+\sigma_I[N_p-(S_p+L_p)]
                  \right\}dB_p,\\
                  dL_p
                  &amp;=
                  \left(
                  \beta_pS_p\frac{I_v}{N_v}-(b+r_1)L_p
                  \right)dt
                  -\sigma_L\frac{S_p}{N_p}L_pdB_p,
                  \\
                  dS_v
                  &amp;=
                  \left( -\frac{\beta_v}{N_p}S_v(N_p-S_p-L_p)
                  -(\gamma +\gamma_f) S_v + (1-\theta)\mu\right)dt
                  -\sigma_v S_vdB_v,
                  \\
                  dI_v
                  &amp;=
                  \left( \frac{\beta_v}{N_p}S_v(N_p-S_p-L_p)
                  - (\gamma +\gamma_f)  I_v +\theta \mu \right) dt
                  - \sigma_v S_v dB_v.
                  \end{aligned}
                  \end{equation}
                </p>
                <p>
                  For the endemic fixed point of system \eqref{sys::DeterministicSystem},
                  $
                  (S_p^{**}, L_p^{**}, I_p^{**}, S_v^{**}, I_v^{**}) ^ \top
                  $,
                  we have the following relations:
                  \begin{align}
                  -&amp;
                  \frac{\beta_p}{N_v}S^{**}_p I^{**}_v +
                  r_2 N_p - r_2 S^{**}_p + (r_1 - r_2) L^{**}_p
                  =0,\label{eqn::efp1}
                  \\
                  &amp;
                  \frac{\beta_p}{N_v}S^{**}_pI^{**}_v
                  -(b+r_1)L^{**}_p
                  = 0,\label{eqn::efp2}
                  \\
                  -&amp;
                  \frac{\beta_v}{N_p}S^{**}_v(N_p-S^{**}_p-L^{**}_p)-
                  (\gamma + \gamma_f) S^{**}_v
                  + (1-\theta)\mu=0,\label{eqn::efp3}
                  \\
                  &amp;
                  \frac{\beta_v}{N_p}S_v(N_p-S^{**}_p-L^{**}_p)-
                  (\gamma + \gamma_f) I^{**}_v
                  +\theta\mu=0 \label{eqn::efp4}.
                  \end{align}
                  In short notation, we rewrite the Lyapunov function \eqref{eqn::Lyapunov1} as
                  \begin{equation*}
                  V(S_p,L_p,S_v,I_v) = c_1 V_1 + c_2 V_2 + c_3 V_3.
                  \end{equation*}
                </p>
                <p>
                  We work with each component of the previous function individually. Applying the
                  diffusion operator to $V_1$, we have
                  \begin{equation*}
                  V_{1_{x}} = \left(1-\frac{L^{**}_p}{L_p}\right),\qquad
                  V_{1_{xx}} = \frac{L^{**}_p}{L_p},
                  \end{equation*}
                  using the equation $dL_p$ of \eqref{sys::Stc4dSystem}, and the
                  diffusion operator we obtain
                  \begin{align*}
                  \mathcal{L} V_1
                  &amp;=
                  \left(
                  1 -
                  \frac{L_P^{**}}{L_p}
                  \right)
                  \left(
                  \frac{\beta_p}{N_v}S_pI_v -
                  (b+r_1) L_p
                  \right) +
                  \frac{1}{2}
                  \frac{\sigma_L^2S_p^2L_p^{**}}{N_p^2}.
                  \end{align*}
                  Since \eqref{eqn::efp2} satisfies,
                  $\beta_p S_p^{**} I^{**}_v /N_v = (b + r_1) L^{**}_p$, then
                  \begin{equation*}\label{eqn::Dv1}
                  \begin{aligned}
                  \mathcal{L} V_1
                  &amp;=
                  \left(
                  1 - \frac{L_P^{**}}{L_p}
                  \right)
                  \left(
                  \frac{\beta_p}{N_v} S_p I_v -
                  \frac{\beta_p}{N_v} S_p ^ {**} I_v ^ {**}
                  \frac{L_p}{L_p^{**}}
                  \right) +
                  \frac{1}{2}
                  \frac{\sigma_L^2S_p^2L_p^{**}}{N_p^2}
                  \\
                  &amp;=
                  \frac{\beta_p}{N_v}
                  \left(
                  1 -
                  \frac{L_P^{**}}{L_p}
                  \right)
                  \left(
                  S_p I_v -
                  S_p ^ {**} I_v ^ {**}
                  \frac{L_p}{L_p ^ {**}}
                  \right) +
                  \frac{1}{2}
                  \frac{\sigma_L^2S_p^2L_p^{**}}{N_p^2}
                  \\
                  &amp;=
                  \frac{\beta_p}{L_p N_v}
                  \left(
                  L_p - L_p^{**}
                  \right)
                  \left(
                  S_p I_v - S_p ^{**} I_v^{**}
                  \frac{L_p}{L_p^{**}}
                  \right) +
                  \frac{1}{2}
                  \frac{\sigma_L^2 S_p^2 L_p^{**}}{N_p^2}
                  \\
                  &amp;=
                  \frac{\beta_p}{N_v}
                  \left(
                  S_pI_v - S_p ^{**} I_v^{**}\frac{L_p}{L^{**}_p}
                  -S_pI_v\frac{L^{**}_p}{L_p}+S^{**}_pI^{**}_v
                  \right) +
                  \frac{1}{2}
                  \frac{\sigma_L^2 S_p^2 L_p^{**}}{N_p^2}.
                  \end{aligned}
                  \end{equation*}
                </p>
                <p>
                  Omitting negative terms and using population size, we get
                  \begin{equation}\label{eqn::LV1}
                  \mathcal{L} V_1\leq 2\beta_pN_p+\frac{\sigma_L^2 }{2}L_p^{**}.
                  \end{equation}
                </p>
                <p>
                  Now, we work with $V_2$, According to relation \eqref{eqn::Lyapunov1}, we have
                  \begin{equation*}
                  V_{2_{x}} = [(S_p - S_p ^{**} ) +
                  (L_p - L_p ^{**})
                  ]
                  \left(
                  1,1
                  \right)^\top,\quad
                  V_{2_{xx}} = \begin{bmatrix}  1 &amp; 1\\ 1 &amp; 1 \end{bmatrix}.
                  \end{equation*}
                  Applying the diffusion operator to $V_2$, we obtain
                  \begin{align*}
                  \mathcal{L}V_2
                  &amp;=
                  \left[
                  (S_p - S_p ^{**} ) +
                  (L_p - L_p ^{**})
                  \right]
                  \left(
                  -\frac{\beta_p}{N_v}S_pI_v + r_2N_p+
                  (r_1-r_2)L_p
                  \right.\\
                  &amp;-
                  \left.
                  r_2 S_p+\frac{\beta_p}{N_v}
                  S_pI_v - (b+r_1)L_p
                  \right) +\frac{1}{2}
                  \left\{\frac{S_p}{N_p}
                  \left[
                  \sigma_I(N_p-S_p-L_p)
                  \right]
                  \right\}^2
                  \\
                  &amp;=
                  \left[
                  (S_p - S_p^{**}) +
                  (L_p - L_p^{**})
                  \right]
                  \left[
                  (r_1-r_2)L_p+r_2N_p-r_2S_p-(b+r_1)L_p
                  \right]
                  \\
                  &amp;+
                  \frac{1}{2}
                  \left\{\frac{S_p}{N_p}
                  \left[
                  \sigma_I(N_p-S_p-L_p)
                  \right]
                  \right\}^2.
                  \end{align*}
                  By \eqref{eqn::efp1} and \eqref{eqn::efp2} yields,
                  \begin{align*}
                  r_2 N_p &amp;=
                  r_2 S ^ {**}_p - (r_1 - r_2)
                  L^{**}_p +
                  \frac{\beta_p S^{**}_p I^{**}_v}{N_v},
                  \\
                  \beta_p S^{**}_p I^{**}_v / N_v &amp;= (b + r_1) L^{**}_p.
                  \end{align*}
                  Thus
                  \begin{align*}
                  \mathcal{L}V_2
                  &amp;=
                  \left[
                  (S_p - S_p^{**}) +
                  (L_p - L_p^{**})
                  \right]
                  \left[
                  (r_1-r_2)L_p-r_2S_p-(b+r_1)L_p+r_2S^{**}_p
                  \right.
                  \\
                  &amp;-\left.
                  (r_1-r_2)L^{**}_p+(b+r_1)L^{**}_p
                  \right]
                  +
                  \frac{1}{2}
                  \left\{\frac{S_p}{N_p}
                  \left[
                  \sigma_I(N_p-S_p-L_p)
                  \right]
                  \right\}^2
                  \\
                  &amp;=
                  \left[
                  (S_p - S_p^{**}) +
                  (L_p - L_p^{**})
                  \right]
                  \left[
                  -r_2(S_p-S_p^{**})-(b+r_2)(L_p-L^{**}_p)
                  \right]
                  \\
                  &amp;+
                  \frac{1}{2}
                  \left\{\frac{S_p}{N_p}
                  \left[
                  \sigma_I(N_p-S_p-L_p)
                  \right]
                  \right\}^2.
                  \end{align*}
                  Expanding the above equation, we have
                  \begin{equation}\label{eqn::LV2}
                  \begin{aligned}
                  \mathcal{L}V_2
                  &amp;=
                  -r_2(S_p-S_p^{**})^2
                  -(b+2r_2)(S_p - S^{**}_p)(L_p - L^{**}_p)
                  \\
                  &amp;-
                  (b+r_2)(L_p - L^{**}_p)^2 +
                  \frac{1}{2}
                  \left\{\frac{S_p}{N_p}
                  \left[
                  \sigma_I(N_p-S_p-L_p)
                  \right]
                  \right\}^2.
                  \end{aligned}
                  \end{equation}
                </p>
                <p>
                  Now, we bound
                  $
                  T_1 =
                  -(b + 2r_2)
                  (S_p - S ^ {**}_p)
                  (L_p - L^{**}_p)
                  $.
                  Young's inequality implies
                  \begin{equation}\label{eqn::YoungLV1}
                  \begin{aligned}
                  T_1
                  &amp;\leq
                  (b+2r_2)|S_p-S^{**}_p||L_p-L^{**}_p|\frac{\sqrt{\rho_1}}{\sqrt{\rho_1}}
                  \\
                  &amp;\leq
                  (b+2r_2)
                  \left[
                  \frac{(S_p-S^{**}_p)^2}{2\rho_1}
                  +\frac{\rho_1(L_p-L^{**}_p)^2}{2}
                  \right].
                  \end{aligned}
                  \end{equation}
                  Combining \eqref{eqn::YoungLV1}, and \eqref{eqn::LV2}, we have
                  \begin{equation}\label{eqn::LV2*}
                  \begin{aligned}
                  \mathcal{L}V_2
                  &amp;\leq
                  -\left[
                  r_2-\frac{(b+2r_2)}{2\rho_1}
                  \right](S_p-S_p^{**})^2
                  \\
                  &amp;-
                  \left[(b + r_2)
                  -\frac{(b + 2r_2) \rho_1}{2}
                  \right](L_p - L^{**}_p)^2
                  +\frac{1}{2}\sigma^2_I N^2_p
                  \\
                  &amp;\leq
                  -\left[
                  r_2-\frac{r_2}{\rho_1}
                  \right](S_p-S_p^{**})^2
                  \\
                  &amp;-
                  \left[(b + r_2)
                  -\frac{(b + r_2) \rho_1}{2}
                  \right](L_p - L^{**}_p)^2
                  +\frac{1}{2}\sigma^2_I N^2_p.
                  \end{aligned}
                  \end{equation}
                </p>
                <p>
                  Let
                  $
                  \rho_1$ such that $\rho_1 \in
                  \left(1, 2\right)
                  $.
                  Finally, we apply diffusion operator to $V_3$, thus
                  \begin{equation*}
                  V_{3_{x}} = \left[	(S_v - S_v ^{**} ) +
                  (I_p - I_p ^{**})
                  \right](1,1)^\top,\quad
                  V_{3_{xx}} = \begin{bmatrix} 1 &amp; 1\\ 1&amp; 1 \end{bmatrix}.
                  \end{equation*}
                  Using $dS_v$, and $dI_v$ of \eqref{sys::Stc4dSystem}, we get
                  \begin{align*}
                  \mathcal{L}V_3
                  =&amp;
                  \left[
                  (S_v - S_v ^{**}) +
                  (I_v - I_v ^{**})
                  \right]
                  \left[ -
                  \frac{\beta_v}{N_p} S_vI_p -
                  (\gamma +\gamma_f) S_v +
                  (1 - \theta)
                  \mu
                  \right.
                  \\
                  &amp;+
                  \left.
                  \frac{\beta_v}{N_p}
                  S_v I_p -
                  (\gamma +\gamma_f) I_v +
                  \theta \mu
                  \right] +
                  \frac{1}{2}
                  \sigma_v ^ 2 {N^{\infty}_v }^ 2.
                  \end{align*}
                  Since \eqref{eqn::efp3} and \eqref{eqn::efp4} satisfies, and
                  $
                  \mu = (\gamma +\gamma_f) N^{\infty}_v\leq (
                  \gamma +\gamma_f) ( S_v^{**} + I_v^{**})
                  $,
                  we obtain
                  \begin{equation}\label{eqn::LV3}
                  \begin{aligned}
                  \mathcal{L}V_3
                  &amp;\leq
                  \left[
                  (S_v - S_v^{**}) +
                  (I_v - I_v^{**})
                  \right]
                  [
                  - (\gamma +\gamma_f)  S_v
                  + (\gamma +\gamma_f) S_v^{**}
                  \\
                  &amp;-
                  (\gamma +\gamma_f) I_v
                  + (\gamma +\gamma_f) I_v^{**}
                  ]
                  +
                  \frac{1}{2}
                  \sigma_v ^ 2 {N^{\infty}_v} ^ 2
                  \\
                  &amp;=
                  \left[
                  (S_v - S_v ^{**}) +
                  (I_v - I_v ^{**})
                  \right]
                  [
                  -(\gamma +\gamma_f) (S_v - S_v^{**})
                  \\
                  &amp;-
                  (\gamma +\gamma_f) (I_v - I_v^{**})
                  ] +
                  \frac{1}{2}
                  \sigma_v ^ 2 {N^{\infty}_v }^ 2
                  \\
                  &amp;=
                  -(\gamma +\gamma_f) (S_v - S_v ^{**})^2
                  -2(\gamma +\gamma_f) (S_v - S_v ^{**})(I_v - I_v^{**})
                  \\
                  &amp;- (\gamma +\gamma_f) (I_v - I_v^{**})^2
                  + \frac{1}{2}\sigma_v ^2 {N^{\infty}_v}^2.
                  \end{aligned}
                  \end{equation}
                </p>
                <p>
                  We bound $T_2 =-2(\gamma +\gamma_f) (S_v - S_v ^{**})(I_v - I_v^{**})$
                  using Young's  inequality <a class="citation" href="#Hardy-Littlewood-Polya1952">[36, P. 61]</a> as follows
                  \begin{equation}\label{eqn::YoungLV2}
                  \begin{aligned}
                  T_2
                  &amp;\leq
                  2(\gamma +\gamma_f)|S_v - S_v ^{**}||I_v - I_v^{**}|
                  \\
                  &amp;\leq
                  2 (\gamma +\gamma_f)
                  \left[
                  \frac{(S_v-S^{**}_v)^2}{2\rho_2}
                  +
                  \frac{\rho_2(I_v-I^{**}_v)^2}{2}
                  \right].
                  \end{aligned}
                  \end{equation}
                </p>
                <p>
                  Substituting \eqref{eqn::YoungLV2} into \eqref{eqn::LV3} yields
                  \begin{equation}\label{eqn::LV4}
                  \begin{aligned}
                  \mathcal{L}V_3
                  &amp;\leq
                  -(\gamma +\gamma_f)
                  \left(1-\frac{1}{2\rho_2}\right) (S_v - S_v ^{**})^2
                  \\
                  &amp;-
                  (\gamma +\gamma_f)\left(1-\rho_2\right) (I_v - I_v^{**})^2
                  + \frac{1}{2} \sigma_v ^2 {N^{\infty}_v}^2,
                  \end{aligned}
                  \end{equation}
                  with $\rho_2 \in \left( \frac{1}{2},1\right )$.	Combining \eqref{eqn::LV1},
                  \eqref{eqn::LV2*}, and \eqref{eqn::LV4} we bound the diffusion operator in this way
                  \begin{equation}\label{eqn::LV-1}
                  \begin{aligned}
                  \mathcal{L}V
                  &amp;\leq
                  -c_2\left[r_2-\frac{r_2}{\rho_1}\right](S_p-S_p^{**})^2
                  \\
                  &amp; -
                  c_2\left[(b+r_2)-\frac{(b+r_2)\rho_1}{2}\right](L_p-L^{**}_p)^2
                  \\
                  &amp;-
                  c_3(\gamma +\gamma_f) \left(1-\frac{1}{2\rho_2} \right)
                  (S_v - S_v ^{**})^2
                  \\
                  &amp;-
                  c_3(\gamma +\gamma_f) \left(1-\rho_2\right)(I_v - I_v^{**})^2
                  \\
                  &amp;+
                  2c_1\beta_p N_p + c_1\frac{\sigma_L^2}{2}L^{**}_p
                  +c_2\frac{1}{2}\sigma^2_I N^2_p+c_3\frac{1}{2}
                  \sigma_v ^2 {N^{\infty}_v}^2.
                  \end{aligned}
                  \end{equation}
                </p>
                <p>
                  Define
                  $C = 2 c_1 \beta_p N_p + c_1 \sigma_L^2 L^{**}_p / 2
                  +c_2 \sigma^2_I N^2_p / 2+c_3 \sigma_v ^2 {N^{\infty}_v}^2 /2$.
                  Then, we bound \eqref{eqn::LV-1} in this fashion
                  \begin{equation*}\label{eqn::LV-2}
                  \begin{aligned}
                  \mathcal{L}V
                  &amp;\leq
                  -c_2\left[
                  r_2-\frac{r_2}{\rho_1}
                  \right](S_p-S_p^{**})^2
                  - c_2\left[
                  (b+r_2)-\frac{(b+r_2)\rho_1}{2}
                  \right](L_p-L^{**}_p)^2
                  \\
                  &amp;-
                  c_3(\gamma +\gamma_f) \left(
                  1-\frac{1}{2\rho_2}
                  \right) (S_v - S_v ^{**})^2
                  - c_3(\gamma +\gamma_f)
                  \left(
                  1-\rho_2
                  \right)(I_v - I_v^{**})^2 + C.
                  \end{aligned}
                  \end{equation*}
                </p>
                <p>
                  Define $F(t)$ as
                  \begin{align*}
                  F(t)
                  &amp;:=
                  -c_2\left[
                  r_2-\frac{r_2}{\rho_1}
                  \right]
                  (S_p-S_p^{**})^2
                  -c_2\left[
                  (b+r_2)-\frac{(b+r_2)\rho_1}{2}
                  \right]
                  (L_p-L^{**}_p)^2
                  \\
                  &amp;-
                  c_3(\gamma +\gamma_f) \left(
                  1-\frac{1}{2\rho_2}
                  \right) (S_v - S_v ^{**})^2
                  -c_3 (\gamma +\gamma_f)
                  \left(
                  1-\rho_2
                  \right)(I_v - I_v^{**})^2 + C.
                  \end{align*}
                  Applying Ito's formula, we obtain
                  \begin{align*}
                  dV
                  &amp;\leq
                  F(t)dt +
                  \left(
                  1-\frac{L^{**}_p}{L_p}
                  \right)
                  \left[
                  -\sigma_L
                  \frac{S_p}{N_p}
                  L_p +
                  \sigma_I
                  \frac{S_p}{N_p}
                  (N_p - S_p - L_p)
                  \right]
                  dB_p(t)
                  \\
                  &amp;-
                  \sigma_v N^{\infty}_v
                  [
                  (S_v-S_v^{**})+(I_v-I_v^{**})
                  ] dB_v(t).
                  \\
                  &amp;\leq
                  F(t)dt +
                  \left[\sigma_I\frac{S_p}{N_p}(N_p-S_p-L_p)
                  +\sigma_L\frac{S_p}{N_p}L^{**}_p\right]
                  dB_p(t)
                  \\
                  &amp;-
                  \sigma_v N^{\infty}_v
                  [(
                  S_v-S_v^{**})+(I_v-I_v^{**})
                  ] dB_v(t).
                  \end{align*}
                  Integrating both sides of above inequality yields
                  \begin{align*}
                  V_3(t) -
                  V_3(0)
                  &amp;\leq
                  \int_0^t F(s)ds
                  +\int_0^t
                  \left[\sigma_I\frac{S_p}{N_p}(N_p-S_p-L_p)
                  +\sigma_L\frac{S_p}{N_p}L^{**}_p\right]dB_p(s)
                  \\
                  &amp;-\int_0^t
                  \sigma_v N^{\infty}_v
                  [(S_v-S_v^{**})+(I_v-I_v^{**})]  dB_v(s).
                  \end{align*}
                </p>
                <p>
                  Let
                  \begin{align*}
                  M_1(t)&amp;:=
                  \int_0^t
                  \left[\sigma_I\frac{S_p}{N_p}(N_p-S_p-L_p)
                  +\sigma_L\frac{S_p}{N_p}L^{**}_p\right]
                  dB_p(s),
                  \\
                  M_2(t)&amp;:=
                  -\int_0^t
                  \sigma_v N^{\infty}_v
                  [(S_v-S_v^{**})+(I_v-I_v^{**})] dB_v(s),
                  \end{align*}
                  with quadratic variation
                  \begin{align*}
                  \langle M_1\rangle_t &amp;:=
                  \int_0^t
                  \left[\sigma_I\frac{S_p}{N_p}(N_p-S_p-L_p)
                  +\sigma_L\frac{S_p}{N_p}L^{**}_p\right]^2 ds,
                  \\
                  \langle M_2\rangle_t &amp;:=
                  \int_0^t
                  \sigma^2_v {N^{\infty}_v}^2
                  [(S_v-S_v^{**})+(I_v-I_v^{**})]^2 ds.
                  \end{align*}
                  Now, we bound $\langle M_1\rangle_t$.
                  \begin{align*}
                  \langle M_1\rangle_t
                  &amp;=\int_0^t
                  \left[\sigma_I\frac{S_p}{N_p}(N_p-S_p-L_p)
                  +\sigma_L\frac{S_p}{N_p}L^{**}_p\right]^2 ds
                  \\
                  &amp;\leq
                  \int_0^t
                  \left[\sigma_I\frac{S_p}{N_p}N_p
                  +\sigma_L S_p\right]^2 ds
                  \\
                  &amp;\leq
                  \int_0^t
                  \left(
                  \sigma_I+\sigma_L
                  \right)^2N^2_p ds,
                  \end{align*}
                  which $\langle M_1\rangle_t$ is finite in a finite interval. Similarly, for
                  $\langle M_2 \rangle_t$. Then, by Strong law of large numbers
                  <a class="citation" href="#Mao2008">[34, P. 12]</a>, we get a.s.
                  \begin{align*}
                  &amp;\lim
                  \limits_{t \to \infty}
                  \frac{M_1(t)}{t} = 0,
                  \qquad
                  \lim
                  \limits_{t\to \infty}
                  \frac{M_2(t)}{t} = 0.
                  \end{align*}
                  Then,
                  \begin{align*}
                  \limsup_{t\to\infty}\frac{1}{t}\int_{0}^{t} F(s) ds \geq 0,
                  \end{align*}
                  thus
                  \begin{align*}
                  \limsup_{t\to \infty}
                  \frac{1}{t}\int_{0}^{t} - F(s)ds \leq 0.
                  \end{align*}
                  Substituting $\alpha_i$ value, we have
                  \begin{align*}
                  \limsup \limits_{t \to 	\infty}
                  &amp;\frac{1}{t} \int_{0}^{t}
                  \biggl \{
                  \alpha_1 (S_p - S_p ^ {**}) ^ 2
                  \biggr.
                  +
                  \alpha_2
                  (L_p - L ^ {**}_p)^2
                  +
                  (\gamma +\gamma_f)
                  \alpha_3
                  (S_v - S_v ^{**})^2
                  \\
                  &amp;+
                  \biggl.
                  \alpha_4
                  (\gamma +\gamma_f)
                  (I_v - I_v ^ {**})^2
                  \biggr\} dr
                  \leq C
                  \quad
                  \text{a.s.}
                  \end{align*}
                  That is the required inequality \eqref{eqn:UpperBound}.
                </p>
                <span class="right">
                  $\square$
                </span>
              </div>
            </div>
          </div>
          <div class="sect1">
            <h2 id="sec:numerics">Numerical Experiments</h2>
            <div class="sectionbody">
            </div>
          </div>
          <div class="sect1">
            <h2 id="references">References</h2>
            <div class="sectionbody">
              <ul class="bibliography">
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Yan2021">[1]Z. Yan, A.-M. A. Wolters, J. Navas-Castillo, and Y. Bai, “The Global Dimension of Tomato Yellow Leaf Curl Disease: Current Status and Breeding Perspectives,” <i>Microorganisms</i>, vol. 9, no. 4, p. 740, Apr. 2021, doi: 10.3390/microorganisms9040740. [Online]. Available at: https://www.mdpi.com/2076-2607/9/4/740</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexYan2021()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.3390/microorganisms9040740">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.3390/microorganisms9040740">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractYan2021()">
                    Abstract
                  </button>
                  <a href="https://www.mdpi.com/2076-2607/9/4/740">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="https://www.mdpi.com/2076-2607/9/4/740">
                  </a>
                  <div id="aYan2021" style="display: none;">
                    <pre>@article{Yan2021,
  author = {Yan, Zhe and Wolters, Anne-Marie A. and Navas-Castillo, Jes{\'{u}}s and Bai, Yuling},
  journal = {Microorganisms},
  title = {{The Global Dimension of Tomato Yellow Leaf Curl Disease: Current Status and Breeding Perspectives}},
  year = {2021},
  issn = {2076-2607},
  month = apr,
  number = {4},
  pages = {740},
  volume = {9},
  doi = {10.3390/microorganisms9040740},
  file = {:C\:/Users/adria/Downloads/microorganisms-09-00740-v3.pdf:pdf},
  keywords = {Begomovirus, Disease resistance, Plant breeding, Solanum lycopersicum, TYLCD, TYLCV, Tomato},
  url = {https://www.mdpi.com/2076-2607/9/4/740}
}
</pre>
                  </div>
                  <div id="absYan2021" style="display: none;">
                    <p class="abstract">
                      Tomato yellow leaf curl disease (TYLCD) caused by tomato yellow leaf curl virus (TYLCV) and a group of related begomoviruses is an important disease which in recent years has caused serious economic problems in tomato (Solanum lycopersicum) production worldwide. Spreading of the vectors, whiteflies of the Bemisia tabaci complex, has been responsible for many TYLCD outbreaks. In this review, we summarize the current knowledge of TYLCV and TYLV-like begomoviruses and the driving forces of the increasing global significance through rapid evolution of begomovirus variants, mixed infection in the field, association with betasatellites and host range expansion. Breeding for host plant resistance is considered as one of the most promising and sustainable methods in controlling TYLCD. Resistance to TYLCD was found in several wild relatives of tomato from which six TYLCV resistance genes (Ty-1 to Ty-6) have been identified. Currently, Ty-1 and Ty-3 are the primary resistance genes widely used in tomato breeding programs. Ty-2 is also exploited commercially either alone or in combination with other Ty-genes (i.e., Ty-1, Ty-3 or ty-5). Additionally, screening of a large collection of wild tomato species has resulted in the identification of novel TYLCD resistance sources. In this review, we focus on genetic resources used to date in breeding for TYLCVD resistance. For future breeding strategies, we discuss several leads in order to make full use of the naturally occurring and engineered resistance to mount a broad-spectrum and sustainable begomovirus resistance.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexYan2021(parameter){
                        var x= document.getElementById('aYan2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractYan2021(parameter){
                        var x= document.getElementById('absYan2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="faiz2021">[2]H. Faiz and K. Abhinav, “Development of immunity against cotton leaf curl virus using antisense AC4 gene,” <i>Research Journal of Biotechnology Vol</i>, vol. 16, p. 9, 2021. </span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexfaiz2021()">
                    Bibtex
                  </button>
                  <div id="afaiz2021" style="display: none;">
                    <pre>@article{faiz2021,
  title = {Development of immunity against cotton leaf curl virus using antisense AC4 gene},
  author = {Faiz, Hashmi and Abhinav, Kumar},
  journal = {Research Journal of Biotechnology Vol},
  volume = {16},
  pages = {9},
  year = {2021}
}
</pre>
                  </div>
                  <div id="absfaiz2021" style="display: none;">
                    <p class="abstract">
                    </p>
                  </div>
                  <script>
                    function toggleBibtexfaiz2021(parameter){
                        var x= document.getElementById('afaiz2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractfaiz2021(parameter){
                        var x= document.getElementById('absfaiz2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Swathi2021">[3]M. Swathi, N. Gaur, and K. Singh, “Virus Vector Relationship of Yellow Mosaic Virus and Whitefly, Bemisia tabaci (Gennadius) in Soybean,” <i>Legume Research - an International Journal</i>, vol. I, no. Of, pp. 1–5, 2021, doi: 10.18805/lr-4479. </span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexSwathi2021()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.18805/lr-4479">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.18805/lr-4479">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractSwathi2021()">
                    Abstract
                  </button>
                  <div id="aSwathi2021" style="display: none;">
                    <pre>@article{Swathi2021,
  author = {Swathi, M. and Gaur, Neeta and Singh, Kamendra},
  journal = {Legume Research - an International Journal},
  title = {{Virus Vector Relationship of Yellow Mosaic Virus and Whitefly, Bemisia tabaci (Gennadius) in Soybean}},
  year = {2021},
  issn = {0250-5371},
  number = {Of},
  pages = {1--5},
  volume = {I},
  doi = {10.18805/lr-4479},
  file = {:C\:/Users/adria/Downloads/Attachment-at-accept-article-LR-4479.pdf:pdf},
  keywords = {acquisition access period, inoculation access period, vector, whitefly, yellow mosaic virus}
}
</pre>
                  </div>
                  <div id="absSwathi2021" style="display: none;">
                    <p class="abstract">
                      Background: Whitefly is one of the most destructive sucking pest in the tropical and subtropical regions of the world and causing significant crop losses directly by sucking sap from the plants and indirectly through the transmission of viral diseases specifically caused by the genus Begomovirus. The Begomovirus species viz., Mungbean yellow mosaic India virus (MYMIV) and Mungbean yellow mosaic virus (MYMV) are causing yellow mosaic virus disease in soybean, which is transmitted by whiteflies. The disease accounts to 30-70 per cent yield loss and increases up to 80 - 100 per cent during severe incidence. Hence, there is a need for development of integrated pest management strategies against disease and whiteflies, for this the knowledge on virus-vector relationship is required. But, the studies on biological relationship of yellow mosaic virus disease and whitefly in soybean are scarce. At this juncture, considering the importance of disease in soybean, the present investigation was carried out to know the virus -vector relationship of the YMV and whitefly in soybean.Methods: The experiment on virus-vector relationship of yellow mosaic virus and whitefly in soybean was conducted at Department of Entomology, College of Agriculture, G. B. Pant University of Agriculture and Technology, Pantnagar, Uttarakhand during 2016-17. The data on number of whiteflies per plant, acquisition and inoculation access feeding period and pre and post starvation period required for effective transmission of virus was recorded.Result: A single viruliferous whitefly was able to transmit virus and ten viruliferous whiteflies per plant were required for cent per cent transmission of virus. The minimum acquisition access and inoculation access feeding periods required for virus transmission was 0.25h (15 min) each; while the 100 per cent virus transmission was recorded with acquisition and inoculation period of 12h, each. The per cent transmission was increased with the increase of acquisition and inoculation periods. The rate of transmission was positively correlated with pre-acquisition starvation period and negatively correlated with post- acquisition starvation period.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexSwathi2021(parameter){
                        var x= document.getElementById('aSwathi2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractSwathi2021(parameter){
                        var x= document.getElementById('absSwathi2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Ali2021">[4]M. A. Ali, M. A. U. Khan, A. Q. Rao, A. Iqbal, S. ud Din, and A. A. Shahid, “Biochemical evidence of epicuticular wax compounds involved in cotton-whitefly interaction,” <i>PLoS ONE</i>, vol. 16, no. 5 May, pp. 1–14, 2021, doi: 10.1371/journal.pone.0250902. [Online]. Available at: http://dx.doi.org/10.1371/journal.pone.0250902</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexAli2021()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1371/journal.pone.0250902">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1371/journal.pone.0250902">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractAli2021()">
                    Abstract
                  </button>
                  <a href="http://dx.doi.org/10.1371/journal.pone.0250902">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="http://dx.doi.org/10.1371/journal.pone.0250902">
                  </a>
                  <div id="aAli2021" style="display: none;">
                    <pre>@article{Ali2021,
  author = {Ali, Muhammad Azam and Khan, Muhammad Azmat Ullah and Rao, Abdul Qayyum and Iqbal, Adnan and ud Din, Salah and Shahid, Ahmad Ali},
  doi = {10.1371/journal.pone.0250902},
  file = {:C\:/Users/adria/Downloads/journal.pone.0250902.pdf:pdf},
  isbn = {1111111111},
  issn = {19326203},
  journal = {PLoS ONE},
  number = {5 May},
  pages = {1--14},
  pmid = {33945542},
  title = {{Biochemical evidence of epicuticular wax compounds involved in cotton-whitefly interaction}},
  url = {http://dx.doi.org/10.1371/journal.pone.0250902},
  volume = {16},
  year = {2021}
}
</pre>
                  </div>
                  <div id="absAli2021" style="display: none;">
                    <p class="abstract">
                      Sucking insects require a surface of plants on which the legs and the eggs of insects will adhere and to which insect mouthparts will access. The primary plant protection against insects is their surface property, which hinders the attachment of the insect’s legs and eggs. The epicuticular waxes chemistry influences the fine structure of the cuticular surface. In current study, an attempt was made to investigate the variation of chemical compounds in epicuticular waxes of four cotton species that classify them resistant or susceptible i.e., Gossypium abroreum, G. hirsutum, G. arboreum wax deficient mutant (GaWM3) and G. harknessi which were evaluated for their interaction with whitefly and CLCuV transmission. Gossypium hirsutum an insect and CLCuV susceptible cotton variety, was found to have four compounds namely Trichloroacetic acid, hexadecylester, P-xylenolpthalein, 2-cyclo-pentene-1-ol, 1-phenyl-and Phenol, 2,5-bis [1,1- dimethyl] which could interact with chitin of whitefly while only two compounds in Gossypium arboreum an insect and CLCuV resistant cotton variety could interact with chitin of whitefly. Similarly, GaWM3 and Gossypium harkasnessi were found to have only a single compound. Number of whiteflies found on leaves of G. hirsutum was much higher as compared to other cotton species. Keeping this fact in mind a wax biosynthetic gene CER3, from Arabidopsis thaliana was transformed into G. hirsutum and the plants were evaluated for their resistance against whitefly and CLCuV transmission. In microscopic analysis transgenic plants clearly showed higher amounts of leaf waxes as compared to non-transgenics. The least whitefly population and CLCuV titer of &lt;10,000 units was found in transgenic plants compared to non-transgenic cotton where it was ≈4.5X106 units that confirmed the role of wax in insect interaction and ultimately to CLCuV transmission. This study provides novel insight on wax related compounds involved in cotton-whitefly interaction, which potentially can help in developing more efficient control strategies for this destructive pest.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexAli2021(parameter){
                        var x= document.getElementById('aAli2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractAli2021(parameter){
                        var x= document.getElementById('absAli2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Li2021">[5]J. Li, J. C. Wang, T. B. Ding, and D. Chu, “Synergistic Effects of a Tomato chlorosis virus and Tomato yellow leaf curl virus Mixed Infection on Host Tomato Plants and the Whitefly Vector,” <i>Frontiers in Plant Science</i>, vol. 12, no. May, pp. 1–11, 2021, doi: 10.3389/fpls.2021.672400. [Online]. Available at: https://doi.org/10.3389/fpls.2021.672400</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexLi2021()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.3389/fpls.2021.672400">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.3389/fpls.2021.672400">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractLi2021()">
                    Abstract
                  </button>
                  <a href="https://doi.org/10.3389/fpls.2021.672400">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="https://doi.org/10.3389/fpls.2021.672400">
                  </a>
                  <div id="aLi2021" style="display: none;">
                    <pre>@article{Li2021,
  author = {Li, Jie and Wang, Ji Cheng and Ding, Tian Bo and Chu, Dong},
  journal = {Frontiers in Plant Science},
  title = {{Synergistic Effects of a Tomato chlorosis virus and Tomato yellow leaf curl virus Mixed Infection on Host Tomato Plants and the Whitefly Vector}},
  year = {2021},
  issn = {1664462X},
  number = {May},
  pages = {1--11},
  volume = {12},
  doi = {10.3389/fpls.2021.672400},
  file = {:C\:/Users/adria/Downloads/fpls-12-672400.pdf:pdf},
  keywords = {Bemisia tabaci MED, Tomato chlorosis virus, Tomato yellow leaf curl virus, mixed infections, synergism},
  url = {https://doi.org/10.3389/fpls.2021.672400}
}
</pre>
                  </div>
                  <div id="absLi2021" style="display: none;">
                    <p class="abstract">
                      In China, Tomato chlorosis virus (ToCV) and Tomato yellow leaf curl virus (TYLCV) are widely present in tomato plants. The epidemiology of these viruses is intimately associated with their vector, the whitefly (Bemisia tabaci MED). However, how a ToCV+TYLCV mixed infection affects viral acquisition by their vector remains unknown. In this study, we examined the growth parameters of tomato seedlings, including disease symptoms and the heights and weights of non-infected, singly infected and mixed infected tomato plants. Additionally, the spatio-temporal dynamics of the viruses in tomato plants, and the viral acquisition and transmission by B. tabaci MED, were determined. The results demonstrated that: (i) ToCV+TYLCV mixed infections induced tomato disease synergism, resulting in a high disease severity index and decreased stem heights and weights; (ii) as the disease progressed, TYLCV accumulated more in upper leaves of TYLCV-infected tomato plants than in lower leaves, whereas ToCV accumulated less in upper leaves of ToCV-infected tomato plants than in lower leaves; (iii) viral accumulation in ToCV+TYLCV mixed infected plants was greater than in singly infected plants; and (iv) B. tabaci MED appeared to have a greater TYLCV, but a lower ToCV, acquisition rate from mixed infected plants compared with singly infected plants. However, mixed infections did not affect transmission by whiteflies. Thus, ToCV+TYLCV mixed infections may induce synergistic disease effects in tomato plants.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexLi2021(parameter){
                        var x= document.getElementById('aLi2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractLi2021(parameter){
                        var x= document.getElementById('absLi2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Lee2021">[6]J. H. Lee, D. J. Chung, J. M. Lee, and I. Yeam, “Development and Application of Gene-Specific Markers for Tomato Yellow Leaf Curl Virus Resistance in Both Field and Artificial Infections,” <i>Plants</i>, vol. 10, no. 1, p. 9, Dec. 2020, doi: 10.3390/plants10010009. [Online]. Available at: https://www.mdpi.com/2223-7747/10/1/9</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexLee2021()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.3390/plants10010009">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.3390/plants10010009">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractLee2021()">
                    Abstract
                  </button>
                  <a href="https://www.mdpi.com/2223-7747/10/1/9">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="https://www.mdpi.com/2223-7747/10/1/9">
                  </a>
                  <div id="aLee2021" style="display: none;">
                    <pre>@article{Lee2021,
  author = {Lee, Jang Hee and Chung, Dae Jun and Lee, Je Min and Yeam, Inhwa},
  journal = {Plants},
  title = {{Development and Application of Gene-Specific Markers for Tomato Yellow Leaf Curl Virus Resistance in Both Field and Artificial Infections}},
  year = {2020},
  issn = {2223-7747},
  month = dec,
  number = {1},
  pages = {9},
  volume = {10},
  doi = {10.3390/plants10010009},
  file = {:C\:/Users/adria/Downloads/plants-10-00009-v2.pdf:pdf},
  keywords = {Gene-pyramiding, Marker-assisted selection, Resistance gene, TYLCV, Tomato},
  url = {https://www.mdpi.com/2223-7747/10/1/9}
}
</pre>
                  </div>
                  <div id="absLee2021" style="display: none;">
                    <p class="abstract">
                      Tomato yellow leaf curl virus (TYLCV) is a disease that is damaging to tomato production worldwide. Resistance to TYLCV has been intensively investigated, and single resistance genes such as Ty-1 have been widely deployed in breeding programs. However, resistance-breaking incidences are frequently reported, and achieving durable resistance against TYLCV in the field is important. In this study, gene-specific markers for Ty-2 and ty-5, and closely-linked markers for Ty-4 were developed and applied to distinguish TYLCV resistance in various tomato genotypes. Quantitative infectivity assays using both natural infection in the field and artificial inoculation utilizing infectious TYLCV clones in a growth chamber were optimized and performed to investigate the individual and cumulative levels of resistance. We confirmed that Ty-2 could also be an effective source of resistance for TYLCV control, together with Ty-1. Improvement of resistance as a result of gene-pyramiding was speculated, and breeding lines including both Ty-1 and Ty-2 showed the strongest resistance in both field and artificial infections.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexLee2021(parameter){
                        var x= document.getElementById('aLee2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractLee2021(parameter){
                        var x= document.getElementById('absLee2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Cohen2007">[7]H. C. (eds.) Shlomo Cohen Moshe Lapidot (auth.), <i>Tomato Yellow Leaf Curl Virus Disease Management, Molecular Biology, Breeding for Resistance</i>, vol. 1. Springer, 100AD. </span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexCohen2007()">
                    Bibtex
                  </button>
                  <div id="aCohen2007" style="display: none;">
                    <pre>@book{Cohen2007,
  title = {Tomato Yellow Leaf Curl Virus Disease Management, Molecular Biology, Breeding for Resistance},
  title_sort = {Tomato Yellow Leaf Curl Virus Disease},
  author_sort = {S. Cohen, M. Lapidot.},
  author = {Shlomo Cohen, Moshe Lapidot (auth.), Henryk Czosnek (eds.)},
  cover = {C:\\Users\\adria\\Calibre Library\\Shlomo Cohen, Moshe Lapidot (auth.),\\Tomato Yellow Leaf Curl Virus Diseas (82)\\cover.jpg},
  formats = {pdf},
  calibreid = {82},
  library_name = {Calibre Library},
  year = {100},
  publisher = {Springer},
  month = {dic.},
  volume = {1},
  size = {10424290 octets},
  timestamp = {2021-08-01},
  uuid = {dc4386bb-6eff-41db-9d69-bebd45b1c273}
}
</pre>
                  </div>
                  <div id="absCohen2007" style="display: none;">
                    <p class="abstract">
                    </p>
                  </div>
                  <script>
                    function toggleBibtexCohen2007(parameter){
                        var x= document.getElementById('aCohen2007');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractCohen2007(parameter){
                        var x= document.getElementById('absCohen2007');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Holt1999">[8]J. Holt, J. Colvin, and V. Muniyappa, “Identifying control strategies for tomato leaf curl virus disease using an epidemiological model,” <i>Journal of Applied Ecology</i>, vol. 36, no. 5, pp. 625–633, Oct. 1999, doi: 10.1046/j.1365-2664.1999.00432.x. [Online]. Available at: http://doi.wiley.com/10.1046/j.1365-2664.1999.00432.x</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexHolt1999()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1046/j.1365-2664.1999.00432.x">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1046/j.1365-2664.1999.00432.x">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractHolt1999()">
                    Abstract
                  </button>
                  <a href="http://doi.wiley.com/10.1046/j.1365-2664.1999.00432.x">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="http://doi.wiley.com/10.1046/j.1365-2664.1999.00432.x">
                  </a>
                  <div id="aHolt1999" style="display: none;">
                    <pre>@article{Holt1999,
  author = {Holt, J. and Colvin, J. and Muniyappa, V.},
  doi = {10.1046/j.1365-2664.1999.00432.x},
  file = {:C$\backslash$:/Users/adria/AppData/Local/Mendeley Ltd./Mendeley Desktop/Downloaded/Holt, Colvin, Muniyappa - 1999 - Identifying control strategies for tomato leaf curl virus disease using an epidemiological model.pdf:pdf},
  issn = {0021-8901},
  journal = {Journal of Applied Ecology},
  keywords = {Bemisia tabaci,Geminivirus,India,Insect dispersal,Pest management},
  month = oct,
  number = {5},
  pages = {625--633},
  title = {{Identifying control strategies for tomato leaf curl virus disease using an epidemiological model}},
  url = {http://doi.wiley.com/10.1046/j.1365-2664.1999.00432.x},
  volume = {36},
  year = {1999}
}
</pre>
                  </div>
                  <div id="absHolt1999" style="display: none;">
                    <p class="abstract">
                      1. A number of insect vectors of plant-virus diseases make only transitory visits to the crop in which the economic effects of the disease are important. The incidence of disease in the crop depends primarily on the immigration of vectors from alternative hosts which act as a reservoir of both the virus and vector. 2. An epidemiological model was developed to represent this situation and parameters were estimated for the case of tomato leaf curl virus disease (TLCVD) (Geminiviridae, Subgroup III) in India. From an analysis of the model, the following possibilities for the management of TLCVD emerged. 3. It was clear that varietal resistance to infection could be an important component of disease management but whether, once infected, the tomato plants acted as a source of inoculum had little impact on disease incidence in the tomato crop. 4. A very low rate of simulated vector immigration into a tomato crop sufficed to cause almost total infection. Around Bangalore, vectors may migrate into tomato crops in numbers in excess of those required for disease ’saturation’, explaining why, using conventional insecticides, very efficient and intensive vector control is currently required to reduce disease incidence. 5. Disease incidence was sensitive to vector mortality only when vector numbers were low. In most cases, the immigration of viruliferous vectors made disease incidence insensitive to the mortality of vectors within the tomato crop. 6. A strategy for disease management which targets more than one of the parameters to which the model proved most sensitive is likely to be necessary. In particular, the use of protective netting combined with the growing of resistant varieties has the potential to reduce both B. tabaci immigration to the crop and to reduce virus inoculation by those insects which do reach the crop.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexHolt1999(parameter){
                        var x= document.getElementById('aHolt1999');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractHolt1999(parameter){
                        var x= document.getElementById('absHolt1999');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Chan1994">[9]M.-S. Chan and M. J. Jeger, “An Analytical Model of Plant Virus Disease Dynamics with Roguing and Replanting,” <i>The Journal of Applied Ecology</i>, vol. 31, no. 3, p. 413, 1994, doi: 10.2307/2404439. </span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexChan1994()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.2307/2404439">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.2307/2404439">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractChan1994()">
                    Abstract
                  </button>
                  <div id="aChan1994" style="display: none;">
                    <pre>@article{Chan1994,
  author = {Chan, Man-Suen and Jeger, Michael J.},
  doi = {10.2307/2404439},
  file = {:C$\backslash$:/Users/adria/AppData/Local/Mendeley Ltd./Mendeley Desktop/Downloaded/Chan, Jeger - 1994 - An Analytical Model of Plant Virus Disease Dynamics with Roguing and Replanting.pdf:pdf},
  issn = {00218901},
  journal = {The Journal of Applied Ecology},
  number = {3},
  pages = {413},
  title = {{An Analytical Model of Plant Virus Disease Dynamics with Roguing and Replanting}},
  volume = {31},
  year = {1994}
}
</pre>
                  </div>
                  <div id="absChan1994" style="display: none;">
                    <p class="abstract">
                      1. The dynamics of a virus disease in a perennial plant population are modelled. The population is divided into healthy, latently infected, infectious and post-infectious plants and linked differential equations describe the dynamics of each category. 2. Qualitative analysis of this model shows stable dynamics and threshold conditions for disease persistence. Stable equilibria are reached after several years. The dynamics of the model are highly sensitive to changes in contact rate and infectious period. 3. Disease management by roguing (removal) of infected plants and replanting with healthy ones is investigated. Roguing only in the post-infectious category confers no advantage. At low contact rates, roguing only when plants become infectious is sufficient to eradicate the disease. At high contact rates, roguing latently infected as well as infectious plants is advisable. 4. With disease present the equilibrium level of healthy plants does not depend on replanting rate, but at higher replanting rates the disease is more difficult to eradicate. There is a trade-off between roguing and replanting in designing optimal disease management strategies. 5. Using parameter values estimated from field studies on three plant virus diseases, the model indicates that eradication is achievable with realistic roguing intensities.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexChan1994(parameter){
                        var x= document.getElementById('aChan1994');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractChan1994(parameter){
                        var x= document.getElementById('absChan1994');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Bokil2019">[10]V. A. Bokil, L. J. S. Allen, M. J. Jeger, and S. Lenhart, “Optimal control of a vectored plant disease model for a crop with continuous replanting,” <i>Journal of Biological Dynamics</i>, vol. 13, no. sup1, pp. 325–353, Mar. 2019, doi: 10.1080/17513758.2019.1622808. [Online]. Available at: https://doi.org/10.1080/17513758.2019.1622808</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexBokil2019()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1080/17513758.2019.1622808">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1080/17513758.2019.1622808">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractBokil2019()">
                    Abstract
                  </button>
                  <a href="https://doi.org/10.1080/17513758.2019.1622808">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="https://doi.org/10.1080/17513758.2019.1622808">
                  </a>
                  <div id="aBokil2019" style="display: none;">
                    <pre>@article{Bokil2019,
  author = {Bokil, V. A. and Allen, L. J. S. and Jeger, M. J. and Lenhart, S.},
  doi = {10.1080/17513758.2019.1622808},
  file = {:C$\backslash$:/Users/adria/AppData/Local/Mendeley Ltd./Mendeley Desktop/Downloaded/Bokil et al. - 2019 - Optimal control of a vectored plant disease model for a crop with continuous replanting.pdf:pdf},
  issn = {1751-3758},
  journal = {Journal of Biological Dynamics},
  keywords = {Vectored plant disease,differential equations,optimal control,replanting strategies},
  month = mar,
  number = {sup1},
  pages = {325--353},
  title = {{Optimal control of a vectored plant disease model for a crop with continuous replanting}},
  url = {https://doi.org/10.1080/17513758.2019.1622808},
  volume = {13},
  year = {2019}
}
</pre>
                  </div>
                  <div id="absBokil2019" style="display: none;">
                    <p class="abstract">
                      Vector-transmitted diseases of plants have had devastating effects on agricultural production worldwide, resulting in drastic reductions in yield for crops such as cotton, soybean, tomato, and cassava. Plant-vector-virus models with continuous replanting are investigated in terms of the effects of selection of cuttings, roguing, and insecticide use on disease prevalence in plants. Previous models are extended to include two replanting strategies: frequencyreplanting and abundance-replanting. In frequency-replanting, replanting of infected cuttings depends on the selection frequency parameter ε, whereas in abundance-replanting, replanting depends on plant abundance via a selection rate parameter also denoted as ε. The two models are analysed and new thresholds for disease elimination are defined for each model. Parameter values for cassava, whiteflies, and African cassava mosaic virus serve as a case study. A numerical sensitivity analysis illustrates how the equilibrium densities of healthy and infected plants vary with parameter values. Optimal control theory is used to investigate the effects of roguing and insecticide use with a goal of maximizing the healthy plants that are harvested. Differences in the control strategies in the two models are seen for large values of ε. Also, the combined strategy of roguing and insecticide use performs better than a single control.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexBokil2019(parameter){
                        var x= document.getElementById('aBokil2019');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractBokil2019(parameter){
                        var x= document.getElementById('absBokil2019');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Luo2021">[11]Y. Luo, F. Zhang, Y. Liu, and S. Gao, “Analysis and optimal control of a Huanglongbing mathematical model with resistant vector,” <i>Infectious Disease Modelling</i>, vol. 6, pp. 782–804, 2021, doi: 10.1016/j.idm.2021.05.004. [Online]. Available at: https://doi.org/10.1016/j.idm.2021.05.004</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexLuo2021()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1016/j.idm.2021.05.004">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1016/j.idm.2021.05.004">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractLuo2021()">
                    Abstract
                  </button>
                  <a href="https://doi.org/10.1016/j.idm.2021.05.004">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="https://doi.org/10.1016/j.idm.2021.05.004">
                  </a>
                  <div id="aLuo2021" style="display: none;">
                    <pre>@article{Luo2021,
  year = {2021},
  title = {{Analysis and optimal control of a Huanglongbing mathematical model with resistant vector}},
  author = {Luo, Youquan and Zhang, Fumin and Liu, Yujiang and Gao, Shujing},
  journal = {Infectious Disease Modelling},
  issn = {2468-0427},
  doi = {10.1016/j.idm.2021.05.004},
  pmid = {34189362},
  pages = {782--804},
  volume = {6},
  keywords = {},
  url = {https://doi.org/10.1016/j.idm.2021.05.004}
}
</pre>
                  </div>
                  <div id="absLuo2021" style="display: none;">
                    <p class="abstract">
                      Huanglongbing (HLB) is an incurable disease that affects citrus trees. To better understand the transmission of HLB, the mathematical model is developed to investigate the transmission dynamics of the disease between Asian citrus psyllid (ACP) and citrus trees. Through rigorous mathematical derivations, we derive the expression of the basic reproduction number (R 0) of HLB. The findings show that the disease-free equilibrium is locally asymptotically stable if R 0 &lt; 1, and if R 0 &gt; 1 the system is uniformly persistent. By applying the global sensitivity analysis of R 0, we can obtain some parameters that have the greatest influence on the HLB transmission dynamics. Additionally, the optimal control theory is used to explore the corresponding optimal control problem of the HLB model. Numerical simulations are conducted to reinforce the analytical results. These theoretical and numerical results provide useful insights for understanding the transmission dynamics of HLB and may help policy makers to develop intervention strategies for the disease.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexLuo2021(parameter){
                        var x= document.getElementById('aLuo2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractLuo2021(parameter){
                        var x= document.getElementById('absLuo2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Hebert2016">[12]L. Allen and M. P. Hebert, “Disease outbreaks in plant-vector-virus models with vector aggregation and dispersal,” <i>Discrete and Continuous Dynamical Systems - Series B</i>, vol. 21, no. 7, pp. 2169–2191, Aug. 2016, doi: 10.3934/dcdsb.2016042. [Online]. Available at: http://www.aimsciences.org/journals/displayArticlesnew.jsp?paperID=12928</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexHebert2016()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.3934/dcdsb.2016042">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.3934/dcdsb.2016042">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractHebert2016()">
                    Abstract
                  </button>
                  <a href="http://www.aimsciences.org/journals/displayArticlesnew.jsp?paperID=12928">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="http://www.aimsciences.org/journals/displayArticlesnew.jsp?paperID=12928">
                  </a>
                  <div id="aHebert2016" style="display: none;">
                    <pre>@article{Hebert2016,
  author = {Allen, Linda and Hebert, Mary P.},
  doi = {10.3934/dcdsb.2016042},
  file = {:C\:/Users/adria/Downloads/Allen_Hebert___2016___Disease_outbreaks_in_plant_vector_virus_models_with_vector_aggregation_and_dispersal.pdf:pdf},
  issn = {1531-3492},
  journal = {Discrete and Continuous Dynamical Systems - Series B},
  keywords = {Cassava mosaic virus,Markov chain,Plant-vector-virus},
  month = aug,
  number = {7},
  pages = {2169--2191},
  title = {{Disease outbreaks in plant-vector-virus models with vector aggregation and dispersal}},
  url = {http://www.aimsciences.org/journals/displayArticlesnew.jsp?paperID=12928},
  volume = {21},
  year = {2016}
}
</pre>
                  </div>
                  <div id="absHebert2016" style="display: none;">
                    <p class="abstract">
                      While feeding on host plants, viruliferous insects serve as vectors for viruses. Successful viral transmission depends on vector behavior. Two behaviors that impact viral transmission are vector aggregation and dispersal. Vector aggregation may be due to chemical or visual cues or feeding preferences. Vector dispersal can result in widespread disease outbreaks among susceptible host plants. These two behaviors are investigated in plant-vector-virus models. Deterministic and stochastic models are formulated to account for stages of infection, vector aggregation and local dispersal between adjacent crops. First, models for a single crop are studied with aggregation included implicitly through the acquisition and inoculation rates. Second, models with aggregation and dispersal of vectors are studied when one field contains a disease-sensitive crop and another a disease-resistant crop. Analytical expressions are computed for the basic reproduction number in the deterministic models and for the probability of disease extinction in the stochastic models. These two expressions provide useful measures to assess effects of aggregation and dispersal on the rate of disease spread within and between crops and the potential for an outbreak. The modeling framework is based on cassava mosaic virus that causes significant damage in cassava crops in Africa.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexHebert2016(parameter){
                        var x= document.getElementById('aHebert2016');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractHebert2016(parameter){
                        var x= document.getElementById('absHebert2016');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Stella2019">[13]I. R. Stella and M. Ghosh, “Modeling plant disease with biological control of insect pests,” <i>Stochastic Analysis and Applications</i>, vol. 37, no. 6, pp. 1133–1154, Nov. 2019, doi: 10.1080/07362994.2019.1646139. [Online]. Available at: https://doi.org/10.1080/07362994.2019.1646139</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexStella2019()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1080/07362994.2019.1646139">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1080/07362994.2019.1646139">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractStella2019()">
                    Abstract
                  </button>
                  <a href="https://doi.org/10.1080/07362994.2019.1646139">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="https://doi.org/10.1080/07362994.2019.1646139">
                  </a>
                  <div id="aStella2019" style="display: none;">
                    <pre>@article{Stella2019,
  author = {Stella, I. R. and Ghosh, Mini},
  journal = {Stochastic Analysis and Applications},
  title = {{Modeling plant disease with biological control of insect pests}},
  year = {2019},
  issn = {0736-2994},
  month = nov,
  number = {6},
  pages = {1133--1154},
  volume = {37},
  doi = {10.1080/07362994.2019.1646139},
  file = {:C\:/Users/adria/AppData/Local/Mendeley Ltd./Mendeley Desktop/Downloaded/Stella, Ghosh - 2019 - Modeling plant disease with biological control of insect pests.pdf:pdf},
  keywords = {Plant disease model, insects, predators, stability, stochastic model},
  publisher = {Taylor &amp; Francis},
  url = {https://doi.org/10.1080/07362994.2019.1646139}
}
</pre>
                  </div>
                  <div id="absStella2019" style="display: none;">
                    <p class="abstract">
                      Introduction: This article discusses the problem of plant diseases that pose major threat to agriculture in several parts of the World. Herein, our focus is on viruses that are transmitted from one plant to another by insect vectors. We consider predators that prey on insect population leading to reduction in infection transmission of plant diseases. Methods: We formulate and analyze a deterministic model for plant disease by incorporating predators as biological control agents. Existence of equilibria and the stability of the model are discussed in-detail. Basic reproduction number R0 of the proposed model is also computed and this helps in determining the impact of different key parameters on the transmission dynamics of disease. Additionally, the proposed model is extended to stochastic model and simulation results of both deterministic and stochastic models are compared and analyzed. Results: Our results of stochastic model show the less number of infected plants and insects compared to corresponding results for deterministic model. Also, our results analyze the impact of different key parameters on the equilibrium levels of infected plants and identify the key parameters. Discussion: Presented results are used to conclude and demonstrate that the biological control is effective in reducing the infection transmission of plant disease and there is a need to use plant-insect-specific predators to get desirable results.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexStella2019(parameter){
                        var x= document.getElementById('aStella2019');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractStella2019(parameter){
                        var x= document.getElementById('absStella2019');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Agarwal2019">[14]R. P. Agarwal, Q. Badshah, G. ur Rahman, and S. Islam, “Optimal control &amp; dynamical aspects of a stochastic pine wilt disease model,” <i>Journal of the Franklin Institute</i>, vol. 356, no. 7, pp. 3991–4025, May 2019, doi: 10.1016/j.jfranklin.2019.03.007. [Online]. Available at: https://linkinghub.elsevier.com/retrieve/pii/S0016003219302029</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexAgarwal2019()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1016/j.jfranklin.2019.03.007">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1016/j.jfranklin.2019.03.007">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractAgarwal2019()">
                    Abstract
                  </button>
                  <a href="https://linkinghub.elsevier.com/retrieve/pii/S0016003219302029">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="https://linkinghub.elsevier.com/retrieve/pii/S0016003219302029">
                  </a>
                  <div id="aAgarwal2019" style="display: none;">
                    <pre>@article{Agarwal2019,
  author = {Agarwal, Ravi P. and Badshah, Qaisar and ur Rahman, Ghaus and Islam, Saeed},
  doi = {10.1016/j.jfranklin.2019.03.007},
  issn = {00160032},
  journal = {Journal of the Franklin Institute},
  month = may,
  number = {7},
  pages = {3991--4025},
  publisher = {Elsevier Ltd},
  title = {{Optimal control \&amp; dynamical aspects of a stochastic pine wilt disease model}},
  url = {https://linkinghub.elsevier.com/retrieve/pii/S0016003219302029},
  volume = {356},
  year = {2019}
}
</pre>
                  </div>
                  <div id="absAgarwal2019" style="display: none;">
                    <p class="abstract">
                      The present work is devoted to study the asymptotic nature of a stochastic pine wilt disease model. Investigating the basic reproductive number, we proved that it has sharp threshold properties. The underlying system of stochastic differential equations is analyzed, and proper Lyapunov functionals are formulated to show the stability analysis. Adequate conditions are provided to study the extinction and permanence of the disease. Reducing infected individuals involved in a substance abuse is a challenging task therefore, using some control variables the stochastic optimality system, is obtained. Furthermore, for the authenticity of the obtained results numerical simulations are carried out at the end of the manuscript.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexAgarwal2019(parameter){
                        var x= document.getElementById('aAgarwal2019');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractAgarwal2019(parameter){
                        var x= document.getElementById('absAgarwal2019');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Chen-Charpentier2021">[15]B. Chen-Charpentier, “Stochastic Modeling of Plant Virus Propagation with Biological Control,” <i>Mathematics</i>, vol. 9, no. 5, p. 456, 2021, doi: 10.3390/math9050456. [Online]. Available at: https://doi.org/10.3390/math9050456</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexChen-Charpentier2021()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.3390/math9050456">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.3390/math9050456">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractChen-Charpentier2021()">
                    Abstract
                  </button>
                  <a href="https://doi.org/10.3390/math9050456">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="https://doi.org/10.3390/math9050456">
                  </a>
                  <div id="aChen-Charpentier2021" style="display: none;">
                    <pre>@article{Chen-Charpentier2021,
  year = {2021},
  title = {{Stochastic Modeling of Plant Virus Propagation with Biological Control}},
  author = {Chen-Charpentier, Benito},
  journal = {Mathematics},
  doi = {10.3390/math9050456},
  pages = {456},
  number = {5},
  volume = {9},
  keywords = {},
  url = {https://doi.org/10.3390/math9050456}
}
</pre>
                  </div>
                  <div id="absChen-Charpentier2021" style="display: none;">
                    <p class="abstract">
                      Plants are vital for man and many species. They are sources of food, medicine, fiber for clothes and materials for shelter. They are a fundamental part of a healthy environment. However, plants are subject to virus diseases. In plants most of the virus propagation is done by a vector. The traditional way of controlling the insects is to use insecticides that have a negative effect on the environment. A more environmentally friendly way to control the insects is to use predators that will prey on the vector, such as birds or bats. In this paper we modify a plant-virus propagation model with delays. The model is written using delay differential equations. However, it can also be expressed in terms of biochemical reactions, which is more realistic for small populations. Since there are always variations in the populations, errors in the measured values and uncertainties, we use two methods to introduce randomness: stochastic differential equations and the Gillespie algorithm. We present numerical simulations. The Gillespie method produces good results for plant-virus population models.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexChen-Charpentier2021(parameter){
                        var x= document.getElementById('aChen-Charpentier2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractChen-Charpentier2021(parameter){
                        var x= document.getElementById('absChen-Charpentier2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Britton2019">[16]T. Britton, E. Pardoux, F. Ball, C. Laredo, D. Sirl, and Tran Viet Chı́, <i>Stochastic epidemic models with inference</i>, vol. 2255. Springer, 2019, p. 472 [Online]. Available at: https://doi.org/10.1007/978-3-030-30900-8</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexBritton2019()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1007/978-3-030-30900-8">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1007/978-3-030-30900-8">
                  </a>
                  <a href="https://doi.org/10.1007/978-3-030-30900-8">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="https://doi.org/10.1007/978-3-030-30900-8">
                  </a>
                  <div id="aBritton2019" style="display: none;">
                    <pre>@book{Britton2019,
  title = {Stochastic epidemic models with inference},
  author = {Britton, Tom and Pardoux, Etienne and Ball, Franck and Laredo, Catherine and Sirl, David and Tran, Viet Ch{\'\i}},
  series = {Lecture Notes in Mathematic},
  volume = {2255},
  note = {Mathematical Biosciences Subseries},
  publisher = {Springer},
  year = {2019},
  pages = {472},
  isbn = {978-3-030-30899-5; 978-3-030-30900-8},
  mrclass = {92-06 (60 92D30)},
  mrnumber = {4299418},
  doi = {10.1007/978-3-030-30900-8},
  url = {https://doi.org/10.1007/978-3-030-30900-8}
}
</pre>
                  </div>
                  <div id="absBritton2019" style="display: none;">
                    <p class="abstract">
                    </p>
                  </div>
                  <script>
                    function toggleBibtexBritton2019(parameter){
                        var x= document.getElementById('aBritton2019');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractBritton2019(parameter){
                        var x= document.getElementById('absBritton2019');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Gray2011">[17]A. Gray, D. Greenhalgh, L. Hu, X. Mao, and J. Pan, “A Stochastic Differential Equation SIS Epidemic Model,” <i>SIAM Journal on Applied Mathematics</i>, vol. 71, no. 3, pp. 876–902, Jan. 2011, doi: 10.1137/10081856X. [Online]. Available at: http://epubs.siam.org/doi/10.1137/10081856X</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexGray2011()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1137/10081856X">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1137/10081856X">
                  </a>
                  <a href="http://epubs.siam.org/doi/10.1137/10081856X">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="http://epubs.siam.org/doi/10.1137/10081856X">
                  </a>
                  <div id="aGray2011" style="display: none;">
                    <pre>@article{Gray2011,
  author = {Gray, A. and Greenhalgh, D. and Hu, L. and Mao, X. and Pan, J.},
  doi = {10.1137/10081856X},
  file = {:home/gabrielsalcedo/.local/share/data/Mendeley Ltd./Mendeley Desktop/Downloaded/Hu - 2008 - Copyright {\textcopyright} by SIAM . Unauthorized reproduction of this article is prohibited .pdf:pdf},
  issn = {0036-1399},
  journal = {SIAM Journal on Applied Mathematics},
  keywords = {backward stochastic,jump diffusions,linear quadratic control,partial information},
  month = jan,
  number = {3},
  pages = {876--902},
  title = {{A Stochastic Differential Equation SIS Epidemic Model}},
  url = {http://epubs.siam.org/doi/10.1137/10081856X},
  volume = {71},
  year = {2011}
}
</pre>
                  </div>
                  <div id="absGray2011" style="display: none;">
                    <p class="abstract">
                    </p>
                  </div>
                  <script>
                    function toggleBibtexGray2011(parameter){
                        var x= document.getElementById('aGray2011');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractGray2011(parameter){
                        var x= document.getElementById('absGray2011');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Cao2016a">[18]Y. Cao and D. Denu, “Analysis of stochastic vector-host epidemic model with direct transmission,” <i>Discrete and Continuous Dynamical Systems - Series B</i>, vol. 21, no. 7, pp. 2109–2127, Aug. 2016, doi: 10.3934/dcdsb.2016039. [Online]. Available at: http://www.aimsciences.org/journals/displayArticlesnew.jsp?paperID=12924</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexCao2016a()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.3934/dcdsb.2016039">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.3934/dcdsb.2016039">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractCao2016a()">
                    Abstract
                  </button>
                  <a href="http://www.aimsciences.org/journals/displayArticlesnew.jsp?paperID=12924">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="http://www.aimsciences.org/journals/displayArticlesnew.jsp?paperID=12924">
                  </a>
                  <div id="aCao2016a" style="display: none;">
                    <pre>@article{Cao2016a,
  author = {Cao, Yanzhao and Denu, Dawit},
  doi = {10.3934/dcdsb.2016039},
  file = {:C$\backslash$:/Users/adria/AppData/Local/Mendeley Ltd./Mendeley Desktop/Downloaded/Cao, Denu - 2016 - Analysis of stochastic vector-host epidemic model with direct transmission.pdf:pdf},
  issn = {1531-3492},
  journal = {Discrete and Continuous Dynamical Systems - Series B},
  keywords = {Stochastic biology modeling,Stochastic dynamical system,Vector-borne disease},
  month = aug,
  number = {7},
  pages = {2109--2127},
  publisher = {Southwest Missouri State University},
  title = {{Analysis of stochastic vector-host epidemic model with direct transmission}},
  url = {http://www.aimsciences.org/journals/displayArticlesnew.jsp?paperID=12924},
  volume = {21},
  year = {2016}
}
</pre>
                  </div>
                  <div id="absCao2016a" style="display: none;">
                    <p class="abstract">
                      In this paper, we consider the stochastic vector-host epidemic model with direct transmission. First, we study the existence of a positive global solution and stochastic boundedness of the system of stochastic differential equations which describes the model. Then we introduce the basic reproductive number R0s in the stochastic model, which reflects the deterministic counterpart, and investigate the dynamics of the stochastic epidemic model when R0s \textless 1 and R0s \textgreater 1. In particular, we show that random effects may lead to extinction in the stochastic case while the deterministic model predicts persistence. Additionally, we provide conditions for the extinction of the infection and stochastic stability of the solution. Finally, numerical simulations are presented to illustrate some of the theoretical results.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexCao2016a(parameter){
                        var x= document.getElementById('aCao2016a');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractCao2016a(parameter){
                        var x= document.getElementById('absCao2016a');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Acuna-Zegarra2018">[19]M. A. Acuña-Zegarra and S. Díaz-Infante, “Stochastic asymptotic analysis of a multi-host model with vector transmission,” <i>Physica A: Statistical Mechanics and its Applications</i>, vol. 510, pp. 243–260, Nov. 2018, doi: 10.1016/j.physa.2018.06.105. [Online]. Available at: https://linkinghub.elsevier.com/retrieve/pii/S037843711830829X</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexAcuna-Zegarra2018()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1016/j.physa.2018.06.105">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1016/j.physa.2018.06.105">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractAcuna-Zegarra2018()">
                    Abstract
                  </button>
                  <a href="https://linkinghub.elsevier.com/retrieve/pii/S037843711830829X">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="https://linkinghub.elsevier.com/retrieve/pii/S037843711830829X">
                  </a>
                  <div id="aAcuna-Zegarra2018" style="display: none;">
                    <pre>@article{Acuna-Zegarra2018,
  author = {Acu{\~{n}}a-Zegarra, Manuel Adrian and D{\'{i}}az-Infante, Sa{\'{u}}l},
  doi = {10.1016/j.physa.2018.06.105},
  file = {:C\:/Users/adria/AppData/Local/Mendeley Ltd./Mendeley Desktop/Downloaded/Acu{\~{n}}a-Zegarra, D{\'{i}}az-Infante - 2018 - Stochastic asymptotic analysis of a multi-host model with vector transmission.pdf:pdf},
  issn = {03784371},
  journal = {Physica A: Statistical Mechanics and its Applications},
  keywords = {Extinction,Multi-host,Persistence,Stochastic perturbation,Vector transmission},
  month = nov,
  pages = {243--260},
  publisher = {Elsevier B.V.},
  title = {{Stochastic asymptotic analysis of a multi-host model with vector transmission}},
  url = {https://linkinghub.elsevier.com/retrieve/pii/S037843711830829X},
  volume = {510},
  year = {2018}
}
</pre>
                  </div>
                  <div id="absAcuna-Zegarra2018" style="display: none;">
                    <p class="abstract">
                      We present a stochastic epidemic model with vectorial transmission and multi-host structure. To include environmental noise, we stochastically perturb biting rates with general state functional intensities. So, we derive a stochastic differential equation (SDE) which describes a vector disease with two types of hosts – humans and animals – and give conditions to assure disease extinction and persistence. Finally, via numerical experiments, we extend and illustrate our results using literature parameters for Chagas.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexAcuna-Zegarra2018(parameter){
                        var x= document.getElementById('aAcuna-Zegarra2018');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractAcuna-Zegarra2018(parameter){
                        var x= document.getElementById('absAcuna-Zegarra2018');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="ZhaoJiang2015">[20]Y. Zhao, D. Jiang, X. Mao, and A. Gray, “The threshold of a stochastic SIRS epidemic model in a population with varying size,” <i>Discrete and Continuous Dynamical Systems-Series B</i>, vol. 20, no. 4, pp. 1277–1295, 2015, doi: 10.3934/dcdsb.2015.20.1277. [Online]. Available at: https://doi.org/10.3934/dcdsb.2015.20.1277</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexZhaoJiang2015()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.3934/dcdsb.2015.20.1277">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.3934/dcdsb.2015.20.1277">
                  </a>
                  <a href="https://doi.org/10.3934/dcdsb.2015.20.1277">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value=" https://doi.org/10.3934/dcdsb.2015.20.1277">
                  </a>
                  <div id="aZhaoJiang2015" style="display: none;">
                    <pre>@article{ZhaoJiang2015,
  title = {The threshold of a stochastic SIRS epidemic model in a population with varying size},
  author = {Zhao, Yanan and Jiang, Daqing and Mao, Xuerong and Gray, Alison},
  journal = {Discrete and Continuous Dynamical Systems-Series B},
  volume = {20},
  number = {4},
  pages = {1277--1295},
  year = {2015},
  url = { https://doi.org/10.3934/dcdsb.2015.20.1277},
  doi = {10.3934/dcdsb.2015.20.1277}
}
</pre>
                  </div>
                  <div id="absZhaoJiang2015" style="display: none;">
                    <p class="abstract">
                    </p>
                  </div>
                  <script>
                    function toggleBibtexZhaoJiang2015(parameter){
                        var x= document.getElementById('aZhaoJiang2015');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractZhaoJiang2015(parameter){
                        var x= document.getElementById('absZhaoJiang2015');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Rifhat2021">[21]R. Rifhat, Z. Teng, and C. Wang, “Extinction and persistence of a stochastic SIRV epidemic model with nonlinear incidence rate,” <i>Advances in Difference Equations</i>, vol. 2021, no. 1, p. 200, Dec. 2021, doi: 10.1186/s13662-021-03347-3. [Online]. Available at: http://dx.doi.org/10.1186/s13662-021-03347-3</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexRifhat2021()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1186/s13662-021-03347-3">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1186/s13662-021-03347-3">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractRifhat2021()">
                    Abstract
                  </button>
                  <a href="http://dx.doi.org/10.1186/s13662-021-03347-3">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="http://dx.doi.org/10.1186/s13662-021-03347-3">
                  </a>
                  <div id="aRifhat2021" style="display: none;">
                    <pre>@article{Rifhat2021,
  author = {Rifhat, Ramziya and Teng, Zhidong and Wang, Chunxia},
  journal = {Advances in Difference Equations},
  title = {{Extinction and persistence of a stochastic SIRV epidemic model with nonlinear incidence rate}},
  year = {2021},
  issn = {1687-1847},
  month = dec,
  number = {1},
  pages = {200},
  volume = {2021},
  doi = {10.1186/s13662-021-03347-3},
  file = {:C\:/Users/adria/Downloads/Rifhat2021_Article_ExtinctionAndPersistenceOfASto.pdf:pdf},
  keywords = {Extinction, Permanence in the mean, Stochastic epidemic model, Threshold value},
  publisher = {The Author(s)},
  url = {http://dx.doi.org/10.1186/s13662-021-03347-3}
}
</pre>
                  </div>
                  <div id="absRifhat2021" style="display: none;">
                    <p class="abstract">
                      In this paper, a stochastic SIRV epidemic model with general nonlinear incidence and vaccination is investigated. The value of our study lies in two aspects. Mathematically, with the help of Lyapunov function method and stochastic analysis theory, we obtain a stochastic threshold of the model that completely determines the extinction and persistence of the epidemic. Epidemiologically, we find that random fluctuations can suppress disease outbreak, which can provide us some useful control strategies to regulate disease dynamics. In other words, neglecting random perturbations overestimates the ability of the disease to spread. The numerical simulations are given to illustrate the main theoretical results.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexRifhat2021(parameter){
                        var x= document.getElementById('aRifhat2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractRifhat2021(parameter){
                        var x= document.getElementById('absRifhat2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Ji2014">[22]C. Ji and D. Jiang, “Threshold behaviour of a stochastic SIR model,” <i>Applied Mathematical Modelling</i>, vol. 38, no. 21-22, pp. 5067–5079, Nov. 2014, doi: 10.1016/j.apm.2014.03.037. [Online]. Available at: http://dx.doi.org/10.1016/j.apm.2014.03.037</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexJi2014()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1016/j.apm.2014.03.037">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1016/j.apm.2014.03.037">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractJi2014()">
                    Abstract
                  </button>
                  <a href="http://dx.doi.org/10.1016/j.apm.2014.03.037">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="http://dx.doi.org/10.1016/j.apm.2014.03.037">
                  </a>
                  <div id="aJi2014" style="display: none;">
                    <pre>@article{Ji2014,
  author = {Ji, Chunyan and Jiang, Daqing},
  doi = {10.1016/j.apm.2014.03.037},
  file = {:C$\backslash$:/Users/adria/AppData/Local/Mendeley Ltd./Mendeley Desktop/Downloaded/Ji, Jiang - 2014 - Threshold behaviour of a stochastic SIR model.pdf:pdf},
  issn = {0307904X},
  journal = {Applied Mathematical Modelling},
  keywords = {Epidemic models,Extinction,Persistence,Stochastic,The basic reproduction number},
  month = nov,
  number = {21-22},
  pages = {5067--5079},
  publisher = {Elsevier Inc.},
  title = {{Threshold behaviour of a stochastic SIR model}},
  url = {http://dx.doi.org/10.1016/j.apm.2014.03.037},
  volume = {38},
  year = {2014}
}
</pre>
                  </div>
                  <div id="absJi2014" style="display: none;">
                    <p class="abstract">
                      In this paper, we investigate the threshold behaviour of a susceptible-infected-recovered (SIR) epidemic model with stochastic perturbation. When the noise is small, we show that the threshold determines the extinction and persistence of the epidemic. Compared with the corresponding deterministic system, this value is affected by white noise, which is less than the basic reproduction number of the deterministic system. On the other hand, we obtain that the large noise will also suppress the epidemic to prevail, which never happens in the deterministic system. These results are illustrated by computer simulations.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexJi2014(parameter){
                        var x= document.getElementById('aJi2014');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractJi2014(parameter){
                        var x= document.getElementById('absJi2014');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="rackauckas2017differentialequations">[23]C. Rackauckas and Q. Nie, “Differentialequations.jl–a performant and feature-rich ecosystem for solving differential equations in julia,” <i>Journal of Open Research Software</i>, vol. 5, no. 1, 2017. </span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexrackauckas2017differentialequations()">
                    Bibtex
                  </button>
                  <div id="arackauckas2017differentialequations" style="display: none;">
                    <pre>@article{rackauckas2017differentialequations,
  title = {Differentialequations.jl--a performant and feature-rich ecosystem for solving differential equations in julia},
  author = {Rackauckas, Christopher and Nie, Qing},
  journal = {Journal of Open Research Software},
  volume = {5},
  number = {1},
  year = {2017},
  publisher = {Ubiquity Press}
}
</pre>
                  </div>
                  <div id="absrackauckas2017differentialequations" style="display: none;">
                    <p class="abstract">
                    </p>
                  </div>
                  <script>
                    function toggleBibtexrackauckas2017differentialequations(parameter){
                        var x= document.getElementById('arackauckas2017differentialequations');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractrackauckas2017differentialequations(parameter){
                        var x= document.getElementById('absrackauckas2017differentialequations');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Abdulle2008">[24]A. Abdulle and T. Li, “S-ROCK methods for stiff Ito SDEs,” <i>Communications in Mathematical Sciences</i>, vol. 6, no. 4, pp. 845–868, 2008, doi: 10.4310/CMS.2008.v6.n4.a3. [Online]. Available at: https://dx.doi.org/10.4310/CMS.2008.v6.n4.a3</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexAbdulle2008()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.4310/CMS.2008.v6.n4.a3">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.4310/CMS.2008.v6.n4.a3">
                  </a>
                  <a href="https://dx.doi.org/10.4310/CMS.2008.v6.n4.a3">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="https://dx.doi.org/10.4310/CMS.2008.v6.n4.a3">
                  </a>
                  <div id="aAbdulle2008" style="display: none;">
                    <pre>@article{Abdulle2008,
  author = {Abdulle, A. and Li, T.},
  title = {{S-ROCK methods for stiff Ito SDEs}},
  volume = {6},
  journal = {Communications in Mathematical Sciences},
  number = {4},
  publisher = {International Press of Boston},
  pages = {845 -- 868},
  keywords = {explicit stochastic methods, multiscale systems, Runge-Kutta Chebyshev methods, stiff chemical Langevin equation, Stiff stochastic differential equations},
  year = {2008},
  doi = {10.4310/CMS.2008.v6.n4.a3},
  url = {https://dx.doi.org/10.4310/CMS.2008.v6.n4.a3}
}
</pre>
                  </div>
                  <div id="absAbdulle2008" style="display: none;">
                    <p class="abstract">
                    </p>
                  </div>
                  <script>
                    function toggleBibtexAbdulle2008(parameter){
                        var x= document.getElementById('aAbdulle2008');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractAbdulle2008(parameter){
                        var x= document.getElementById('absAbdulle2008');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Grinsztajn2021">[25]L. Grinsztajn, E. Semenova, C. C. Margossian, and J. Riou, “Bayesian workflow for disease transmission modeling in Stan,” <i>Stat. Med.</i>, vol. 40, no. 27, pp. 6209–6234, Nov. 2021, doi: 10.1002/sim.9164. [Online]. Available at: https://onlinelibrary.wiley.com/doi/10.1002/sim.9164</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexGrinsztajn2021()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1002/sim.9164">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1002/sim.9164">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractGrinsztajn2021()">
                    Abstract
                  </button>
                  <a href="https://onlinelibrary.wiley.com/doi/10.1002/sim.9164">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="https://onlinelibrary.wiley.com/doi/10.1002/sim.9164">
                  </a>
                  <div id="aGrinsztajn2021" style="display: none;">
                    <pre>@article{Grinsztajn2021,
  archiveprefix = {arXiv},
  arxivid = {2006.02985},
  author = {Grinsztajn, L{\'{e}}o and Semenova, Elizaveta and Margossian, Charles C. and Riou, Julien},
  doi = {10.1002/sim.9164},
  eprint = {2006.02985},
  file = {:home/gabrielsalcedo/Descargas/Bayesian workflow for disease transmission modeling in Stan | Enhanced Reader.pdf:pdf},
  issn = {0277-6715},
  journal = {Stat. Med.},
  keywords = {Bayesian workflow,compartmental models,epidemiology,infectious diseases},
  month = nov,
  number = {27},
  pages = {6209--6234},
  pmid = {34494686},
  title = {{Bayesian workflow for disease transmission modeling in Stan}},
  url = {https://onlinelibrary.wiley.com/doi/10.1002/sim.9164},
  volume = {40},
  year = {2021}
}
</pre>
                  </div>
                  <div id="absGrinsztajn2021" style="display: none;">
                    <p class="abstract">
                      This tutorial shows how to build, fit, and criticize disease transmission models in Stan, and should be useful to researchers interested in modeling the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) pandemic and other infectious diseases in a Bayesian framework. Bayesian modeling provides a principled way to quantify uncertainty and incorporate both data and prior knowledge into the model estimates. Stan is an expressive probabilistic programming language that abstracts the inference and allows users to focus on the modeling. As a result, Stan code is readable and easily extensible, which makes the modeler’s work more transparent. Furthermore, Stan’s main inference engine, Hamiltonian Monte Carlo sampling, is amiable to diagnostics, which means the user can verify whether the obtained inference is reliable. In this tutorial, we demonstrate how to formulate, fit, and diagnose a compartmental transmission model in Stan, first with a simple susceptible-infected-recovered model, then with a more elaborate transmission model used during the SARS-CoV-2 pandemic. We also cover advanced topics which can further help practitioners fit sophisticated models; notably, how to use simulations to probe the model and priors, and computational techniques to scale-up models based on ordinary differential equations.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexGrinsztajn2021(parameter){
                        var x= document.getElementById('aGrinsztajn2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractGrinsztajn2021(parameter){
                        var x= document.getElementById('absGrinsztajn2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Salcedo-varela2021">[26]G. Salcedo-varela, “Supplementary and extended material of Threshold be-havior of a stochastic vector plant model for Tomato Yel-low Curl Leaves disease : a study based on mathematicalanalysis and simu ... Supplementary and extended material of Threshold be- havior of a stoc,” November, 2021 [Online]. Available at: https://www.researchgate.net/publication/356360098_Supplementary_and_extended_material_of_Threshold_be-havior_of_a_stochastic_vector_plant_model_for_Tomato_Yel-low_Curl_Leaves_disease_a_study_based_on_mathematicalanalysis_and_simulation</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexSalcedo-varela2021()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.13140/RG.2.2.16644.45448">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.13140/RG.2.2.16644.45448">
                  </a>
                  <a href="https://www.researchgate.net/publication/356360098_Supplementary_and_extended_material_of_Threshold_be-havior_of_a_stochastic_vector_plant_model_for_Tomato_Yel-low_Curl_Leaves_disease_a_study_based_on_mathematicalanalysis_and_simulation">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="https://www.researchgate.net/publication/356360098_Supplementary_and_extended_material_of_Threshold_be-havior_of_a_stochastic_vector_plant_model_for_Tomato_Yel-low_Curl_Leaves_disease_a_study_based_on_mathematicalanalysis_and_simulation">
                  </a>
                  <div id="aSalcedo-varela2021" style="display: none;">
                    <pre>@techreport{Salcedo-varela2021,
  author = {Salcedo-varela, Gabriel},
  doi = {10.13140/RG.2.2.16644.45448},
  file = {:home/gabrielsalcedo/Descargas/Supplementary{\_}and{\_}extended{\_}material.pdf:pdf},
  isbn = {0000000195591},
  number = {November},
  pages = {9},
  title = {{Supplementary and extended material of Threshold be-havior of a stochastic vector plant model for Tomato Yel-low Curl Leaves disease : a study based on mathematicalanalysis and simu ... Supplementary and extended material of Threshold be- havior of a stoc}},
  url = {https://www.researchgate.net/publication/356360098\_Supplementary\_and\_extended\_material\_of\_Threshold\_be-havior\_of\_a\_stochastic\_vector\_plant\_model\_for\_Tomato\_Yel-low\_Curl\_Leaves\_disease\_a\_study\_based\_on\_mathematicalanalysis\_and\_simulation},
  year = {2021}
}
</pre>
                  </div>
                  <div id="absSalcedo-varela2021" style="display: none;">
                    <p class="abstract">
                    </p>
                  </div>
                  <script>
                    function toggleBibtexSalcedo-varela2021(parameter){
                        var x= document.getElementById('aSalcedo-varela2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractSalcedo-varela2021(parameter){
                        var x= document.getElementById('absSalcedo-varela2021');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Butter1978">[27]N. S. Butter and H. S. Rataul, “The virus-vector relationship of the tomato leafcurl virus (TLCV) and its vector,Bemisia tabaci gennadius (Hemiptera: aleyrodidae),” <i>Phytoparasitica</i>, vol. 5, no. 3, pp. 173–186, Oct. 1977, doi: 10.1007/BF02980351. [Online]. Available at: http://link.springer.com/10.1007/BF02980351</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexButter1978()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1007/BF02980351">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1007/BF02980351">
                  </a>
                  <a href="http://link.springer.com/10.1007/BF02980351">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="http://link.springer.com/10.1007/BF02980351">
                  </a>
                  <div id="aButter1978" style="display: none;">
                    <pre>@article{Butter1978,
  author = {Butter, N S and Rataul, H. S.},
  doi = {10.1007/BF02980351},
  file = {:C$\backslash$:/Users/adria/AppData/Local/Mendeley Ltd./Mendeley Desktop/Downloaded/Butter, Rataul - 1977 - The virus-vector relationship of the tomato leafcurl virus (TLCV) and its vector,Bemisia tabaci gennadius (Hemip.pdf:pdf},
  issn = {0334-2123},
  journal = {Phytoparasitica},
  month = oct,
  number = {3},
  pages = {173--186},
  title = {{The virus-vector relationship of the tomato leafcurl virus (TLCV) and its vector,Bemisia tabaci gennadius (Hemiptera: aleyrodidae)}},
  url = {http://link.springer.com/10.1007/BF02980351},
  volume = {5},
  year = {1977}
}
</pre>
                  </div>
                  <div id="absButter1978" style="display: none;">
                    <p class="abstract">
                    </p>
                  </div>
                  <script>
                    function toggleBibtexButter1978(parameter){
                        var x= document.getElementById('aButter1978');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractButter1978(parameter){
                        var x= document.getElementById('absButter1978');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="ramappa1994">[28]H. K. Ramappa, “Detection, host resistance and integrated management of tomato leaf curl geminivirus disease,” PhD thesis, UNIVERSITY OF AGRICULTURAL SCIENCES, BANGALORE, 1994 [Online]. Available at: https://krishikosh.egranth.ac.in/handle/1/5810126386</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexramappa1994()">
                    Bibtex
                  </button>
                  <a href="https://krishikosh.egranth.ac.in/handle/1/5810126386">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="https://krishikosh.egranth.ac.in/handle/1/5810126386">
                  </a>
                  <div id="aramappa1994" style="display: none;">
                    <pre>@phdthesis{ramappa1994,
  title = {Detection, host resistance and integrated management of tomato leaf curl geminivirus disease},
  author = {Ramappa, HK},
  year = {1994},
  school = {UNIVERSITY OF AGRICULTURAL SCIENCES, BANGALORE},
  url = {https://krishikosh.egranth.ac.in/handle/1/5810126386}
}
</pre>
                  </div>
                  <div id="absramappa1994" style="display: none;">
                    <p class="abstract">
                    </p>
                  </div>
                  <script>
                    function toggleBibtexramappa1994(parameter){
                        var x= document.getElementById('aramappa1994');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractramappa1994(parameter){
                        var x= document.getElementById('absramappa1994');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Nateshan1996">[29]H. M. Nateshan, S. H. Muniyappa, S. H. Jalikop, and H. K. Ramappa, <i>Resistance of lycopersicon species and hybrids to tomato  leaf curl geminivirus. Bemisia: 1995 taxonomy, biology, damage, control and management</i>. Andover: Intercept Limited, 1996, pp. p.369–377 [Online]. Available at: http://worldveg.tind.io/record/23543</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexNateshan1996()">
                    Bibtex
                  </button>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractNateshan1996()">
                    Abstract
                  </button>
                  <a href="http://worldveg.tind.io/record/23543">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="http://worldveg.tind.io/record/23543">
                  </a>
                  <div id="aNateshan1996" style="display: none;">
                    <pre>@book{Nateshan1996,
  author = {Nateshan, H.M. and Muniyappa, S.H. and Jalikop, S.H. and Ramappa, H.K.},
  url = {http://worldveg.tind.io/record/23543},
  title = {Resistance of lycopersicon species and hybrids to tomato  leaf curl geminivirus. Bemisia: 1995 taxonomy, biology, damage, control and management},
  publisher = {Intercept Limited},
  year = {1996},
  recid = {23543},
  pages = {p.369-377},
  address = {Andover}
}
</pre>
                  </div>
                  <div id="absNateshan1996" style="display: none;">
                    <p class="abstract">
                      Among the several Lycopersicon species tested, accessions EC 66003(L.glandulosum);
                      LA 386,LA 1777,PI 390658,PI  390659(L.hirsutum);PI 127830,PI 127831 and CMV  Sel.INRA
                      (L.peruianum) were resistant to TLCV. The other  Lycopersicon species expressed mild to
                      moderate reactions  against TLCV. The F1 hybrids viz.,F1 Tyking,F1 Fiona and F2  Fiona
                      were found to be resistant,all other tomato  hybirds/cultivars were highly susceptible.
                      The disease  spreads rapidly in the susciptible cultivars and affects  the plants at
                      an early stage,leading to economic yield  losses,whereas in resistant species and
                      hybrids the  symptoms appear at later stages and spread slowly. There  were very
                      few or no whiteflies recorded on resistant lines  PI 390658,PI 390659,LA 1777
                      and PI 127830,whereas on one  resistant line CMV Sel.INRA higher number of
                      whiteflies was  recorded and survived up to 11 days. Whiteflies survived up to
                      17-24 days on susceptible lines Arka Vikas and  ArkaSourabh. [AS]
                    </p>
                  </div>
                  <script>
                    function toggleBibtexNateshan1996(parameter){
                        var x= document.getElementById('aNateshan1996');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractNateshan1996(parameter){
                        var x= document.getElementById('absNateshan1996');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Ramappa1998">[30]H. K. Ramappa, V. Muniyappa, and J. Colvin, “The contribution of tomato and alternative host plants to tomato leaf curl virus inoculum pressure in different areas of South India,” <i>Annals of Applied Biology</i>, vol. 133, no. 2, pp. 187–198, 1998, doi: 10.1111/j.1744-7348.1998.tb05819.x. [Online]. Available at: https://doi.org/10.1111/j.1744-7348.1998.tb05819.x</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexRamappa1998()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1111/j.1744-7348.1998.tb05819.x">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1111/j.1744-7348.1998.tb05819.x">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractRamappa1998()">
                    Abstract
                  </button>
                  <a href="https://doi.org/10.1111/j.1744-7348.1998.tb05819.x">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="https://doi.org/10.1111/j.1744-7348.1998.tb05819.x">
                  </a>
                  <div id="aRamappa1998" style="display: none;">
                    <pre>@article{Ramappa1998,
  author = {Ramappa, H. K. and Muniyappa, V. and Colvin, J.},
  doi = {10.1111/j.1744-7348.1998.tb05819.x},
  file = {:C$\backslash$:/Users/adria/AppData/Local/Mendeley Ltd./Mendeley Desktop/Downloaded/Ramappa, Muniyappa, Colvin - 1998 - The contribution of tomato and alternative host plants to tomato leaf curl virus inoculum pressure i.pdf:pdf},
  issn = {00034746},
  journal = {Annals of Applied Biology},
  keywords = {Bemisia tabaci,Monoclonal antibody,Tomato,Tomato leaf curl virus},
  number = {2},
  pages = {187--198},
  title = {{The contribution of tomato and alternative host plants to tomato leaf curl virus inoculum pressure in different areas of South India}},
  volume = {133},
  year = {1998},
  url = {https://doi.org/10.1111/j.1744-7348.1998.tb05819.x}
}
</pre>
                  </div>
                  <div id="absRamappa1998" style="display: none;">
                    <p class="abstract">
                      Indian tomato leaf curl virus (ToLCV) (Geminiviridae: Sub-group III) was detected both in field-collected and laboratory-reared B. tabaci using a triple-antibody sandwich enzyme-linked immunosorbent assay (TAS-ELISA). ToLCV was detected in six of the 10 group samples of field collected B. tabaci. ToLCV was also identified in 13 weed species commonly found in Karnataka, both by symptom expression and TAS-ELISA. ToLCV from c. 61% of infected plants was transmitted successfully to tomato by B. tabaci. Tomato plots were planted at three locations on the University of Agricultural Sciences Campus, Bangalore. Indian tomato leaf curl virus disease (ToLCVD) incidence increased most rapidly when the tomato plot was situated adjacent to an older ToLCVD-infected tomato field. When the plots were positioned in a dryland or a wetland area, at least 500 m away from any infected tomato fields, the ToLCVD incidence increased less rapidly, although in all sites it was 100% by 11 wk after transplanting. The numbers of B. tabaci caught on yellow traps in all sites increased during weeks 1-3 after transplanting and thereafter remained at between 10-15 adults trap-1 24 h-1. Adult numbers recorded on tomato plants by direct counts remained approximately constant at 2-4 adults plant. Tomato fields were planted in three taluks (administrative areas) of Karnataka, that had different current and previous histories of tomato production. ToLCVD incidence increased most and least rapidly, respectively, in Kolar taluk where tomato is grown continuously and Doddaballapur tuluk where tomato was grown in the area for the first time. In Malur tuluk, where tomato was grown discontinuously (once a year), the incidence of ToLCVD increased at an intermediate rate. Weed host-plant species growing near the experimental sites had averages of between 1.5-10.0 B. tabaci nymphs per plant, whereas the tomato plants had only 0.3 nymphs per plant. The percentage parasitism of B. tabaci nymphs on tomato and weed species, respectively, was 0.7% and 2-6%. Nymphs and pupae were parasitised by an Encarsia sp. and Eretmocerus mundus Mercet. The relevance and implications of these findings for the epidemiology and management of ToLCVD in Karnataka State, South India is discussed.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexRamappa1998(parameter){
                        var x= document.getElementById('aRamappa1998');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractRamappa1998(parameter){
                        var x= document.getElementById('absRamappa1998');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Meiss2007">[31]J. D. Meiss, <i>Differential dynamical systems</i>, Revised., vol. 22. Society for Industrial and Applied Mathematics (SIAM),
                      Philadelphia, PA, 2017, p. xvii+392 [Online]. Available at: https://doi.org/10.1137/1.9781611974645</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexMeiss2007()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1137/1.9781611974645">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1137/1.9781611974645">
                  </a>
                  <a href="https://doi.org/10.1137/1.9781611974645">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="https://doi.org/10.1137/1.9781611974645">
                  </a>
                  <div id="aMeiss2007" style="display: none;">
                    <pre>@book{Meiss2007,
  author = {Meiss, James D.},
  title = {Differential dynamical systems},
  series = {Mathematical Modeling and Computation},
  volume = {22},
  edition = {Revised},
  publisher = {Society for Industrial and Applied Mathematics (SIAM),
  		Philadelphia, PA},
  year = {2017},
  pages = {xvii+392},
  isbn = {978-1-61197-464-5; 978-1-61197-463-8},
  mrclass = {37-01 (34-01 34C23 37C10 37J05 37Nxx)},
  mrnumber = {3614477},
  doi = {10.1137/1.9781611974645},
  url = {https://doi.org/10.1137/1.9781611974645}
}
</pre>
                  </div>
                  <div id="absMeiss2007" style="display: none;">
                    <p class="abstract">
                    </p>
                  </div>
                  <script>
                    function toggleBibtexMeiss2007(parameter){
                        var x= document.getElementById('aMeiss2007');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractMeiss2007(parameter){
                        var x= document.getElementById('absMeiss2007');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="VandenDriessche2017a">[32]P. van den Driessche, “Reproduction numbers of infectious disease models,” <i>Infectious Disease Modelling</i>, vol. 2, no. 3, pp. 288–303, Aug. 2017, doi: 10.1016/j.idm.2017.06.002. [Online]. Available at: http://dx.doi.org/10.1016/j.idm.2017.06.002</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexVandenDriessche2017a()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1016/j.idm.2017.06.002">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1016/j.idm.2017.06.002">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractVandenDriessche2017a()">
                    Abstract
                  </button>
                  <a href="http://dx.doi.org/10.1016/j.idm.2017.06.002">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="http://dx.doi.org/10.1016/j.idm.2017.06.002">
                  </a>
                  <div id="aVandenDriessche2017a" style="display: none;">
                    <pre>@article{VandenDriessche2017a,
  author = {van den Driessche, Pauline},
  title = {{Reproduction numbers of infectious disease models}},
  journal = {Infectious Disease Modelling},
  year = {2017},
  volume = {2},
  number = {3},
  pages = {288--303},
  month = aug,
  issn = {24680427},
  doi = {10.1016/j.idm.2017.06.002},
  file = {:C$\backslash$:/Users/adria/AppData/Local/Temp/1-s2.0-S2468042717300209-main.pdf:pdf},
  keywords = {Anthrax,Basic reproduction number,Cholera,Disease control,West Nile virus,Zika virus},
  publisher = {Elsevier Ltd},
  url = {http://dx.doi.org/10.1016/j.idm.2017.06.002}
}
</pre>
                  </div>
                  <div id="absVandenDriessche2017a" style="display: none;">
                    <p class="abstract">
                      This primer article focuses on the basic reproduction number, ℛ0, for infectious diseases, and other reproduction numbers related to ℛ0 that are useful in guiding control strategies. Beginning with a simple population model, the concept is developed for a threshold value of ℛ0 determining whether or not the disease dies out. The next generation matrix method of calculating ℛ0 in a compartmental model is described and illustrated. To address control strategies, type and target reproduction numbers are defined, as well as sensitivity and elasticity indices. These theoretical ideas are then applied to models that are formulated for West Nile virus in birds (a vector-borne disease), cholera in humans (a disease with two transmission pathways), anthrax in animals (a disease that can be spread by dead carcasses and spores), and Zika in humans (spread by mosquitoes and sexual contacts). Some parameter values from literature data are used to illustrate the results. Finally, references for other ways to calculate ℛ0 are given. These are useful for more complicated models that, for example, take account of variations in environmental fluctuation or stochasticity.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexVandenDriessche2017a(parameter){
                        var x= document.getElementById('aVandenDriessche2017a');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractVandenDriessche2017a(parameter){
                        var x= document.getElementById('absVandenDriessche2017a');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Schurz2015">[33]H. Schurz and K. Tosun, “Stochastic Asymptotic Stability of SIR Model with Variable Diffusion Rates,” <i>J. Dyn. Differ. Equations</i>, vol. 27, no. 1, pp. 69–82, Mar. 2015, doi: 10.1007/s10884-014-9415-9. [Online]. Available at: http://link.springer.com/10.1007/s10884-014-9415-9</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexSchurz2015()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1007/s10884-014-9415-9">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1007/s10884-014-9415-9">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractSchurz2015()">
                    Abstract
                  </button>
                  <a href="http://link.springer.com/10.1007/s10884-014-9415-9">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="http://link.springer.com/10.1007/s10884-014-9415-9">
                  </a>
                  <div id="aSchurz2015" style="display: none;">
                    <pre>@article{Schurz2015,
  author = {Schurz, Henri and Tosun, Kursad},
  doi = {10.1007/s10884-014-9415-9},
  file = {:home/gabrielsalcedo/.local/share/data/Mendeley Ltd./Mendeley Desktop/Downloaded/Schurz, Tosun - 2015 - Stochastic Asymptotic Stability of SIR Model with Variable Diffusion Rates(2).pdf:pdf},
  issn = {1040-7294},
  journal = {J. Dyn. Differ. Equations},
  keywords = {Mathematical epidemiology,Stochastic SIR,Stochastic asymptotic stability,Stochastic models},
  month = mar,
  number = {1},
  pages = {69--82},
  title = {{Stochastic Asymptotic Stability of SIR Model with Variable Diffusion Rates}},
  url = {http://link.springer.com/10.1007/s10884-014-9415-9},
  volume = {27},
  year = {2015}
}
</pre>
                  </div>
                  <div id="absSchurz2015" style="display: none;">
                    <p class="abstract">
                      We introduce random fluctuations on contact and recovery rates in deterministic SIR model with disease deaths in nonparametric manner and obtain stochastic counterparts with general diffusion coefficients (functional contact and recovery rates). (Formula presented.) (Formula presented.) (Formula presented.)The introduced stochastic model has functional diffusion coefficients which contains arbitrary local Lipschitz-continuous functions Fi’s defined on (Formula presented.). In this paper we prove the global existence of a unique strong solution and discuss stochastic asymptotic stability of disease free and endemic equilibria of the model and visualize our results with some simulations to confirm them.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexSchurz2015(parameter){
                        var x= document.getElementById('aSchurz2015');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractSchurz2015(parameter){
                        var x= document.getElementById('absSchurz2015');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Mao2008">[34]X. Mao, <i>Stochastic differential equations and applications</i>, Second. Horwood Publishing Limited, Chichester, 2008, p. xviii+422 [Online]. Available at: https://doi.org/10.1533/9780857099402</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexMao2008()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1533/9780857099402">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1533/9780857099402">
                  </a>
                  <a href="https://doi.org/10.1533/9780857099402">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="https://doi.org/10.1533/9780857099402">
                  </a>
                  <div id="aMao2008" style="display: none;">
                    <pre>@book{Mao2008,
  author = {Mao, Xuerong},
  publisher = {Horwood Publishing Limited, Chichester},
  title = {Stochastic differential equations and applications},
  year = {2008},
  edition = {Second},
  isbn = {978-1-904275-34-3},
  doi = {10.1533/9780857099402},
  mrclass = {60-02 (34F05 34K50 60H10 60H30 91B28)},
  mrnumber = {2380366},
  pages = {xviii+422},
  url = {https://doi.org/10.1533/9780857099402}
}
</pre>
                  </div>
                  <div id="absMao2008" style="display: none;">
                    <p class="abstract">
                    </p>
                  </div>
                  <script>
                    function toggleBibtexMao2008(parameter){
                        var x= document.getElementById('aMao2008');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractMao2008(parameter){
                        var x= document.getElementById('absMao2008');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Zhao2015a">[35]Y. Zhao, Q. Zhang, and D. Jiang, “The asymptotic behavior of a stochastic SIS epidemic model with vaccination,” <i>Advances in Difference Equations</i>, vol. 2015, no. 1, p. 328, Dec. 2015, doi: 10.1186/s13662-015-0592-6. [Online]. Available at: http://dx.doi.org/10.1186/s13662-015-0592-6</span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexZhao2015a()">
                    Bibtex
                  </button>
                  <a href="https://doi.org/10.1186/s13662-015-0592-6">
                    <input class="btn btn-outline-info btn-sm" type="button" value="DOI:10.1186/s13662-015-0592-6">
                  </a>
                  <button type="button" class="btn btn-secondary btn-sm" onclick="toggleAbstractZhao2015a()">
                    Abstract
                  </button>
                  <a href="http://dx.doi.org/10.1186/s13662-015-0592-6">
                    <input class="btn btn-success btn-raised btn-sm" type="button" value="URL:">
                    <input class="btn btn-outline-success btn-sm" type="button" value="http://dx.doi.org/10.1186/s13662-015-0592-6">
                  </a>
                  <div id="aZhao2015a" style="display: none;">
                    <pre>@article{Zhao2015a,
  author = {Zhao, Yanan and Zhang, Qiumei and Jiang, Daqing},
  doi = {10.1186/s13662-015-0592-6},
  file = {:home/gabrielsalcedo/Dropbox/Libros/Seminario/document.pdf:pdf},
  issn = {1687-1847},
  journal = {Advances in Difference Equations},
  keywords = {Lyapunov function,exponential stability,persistent in mean,stochastic SIS epidemic model,vaccination},
  month = dec,
  number = {1},
  pages = {328},
  publisher = {Zhao et al.},
  title = {{The asymptotic behavior of a stochastic SIS epidemic model with vaccination}},
  url = {http://dx.doi.org/10.1186/s13662-015-0592-6},
  volume = {2015},
  year = {2015}
}
</pre>
                  </div>
                  <div id="absZhao2015a" style="display: none;">
                    <p class="abstract">
                      In this paper, we discuss a stochastic SIS epidemic model with vaccination. We investigate the asymptotic behavior according to the perturbation and the reproduction number R0}R_{0}}. When the perturbation is large, the number of infected decays exponentially to zero and the solution converges to the disease-free equilibrium regardless of the magnitude of R0}R_{0}}. Moreover, we get the same exponential stability and the convergence if R0\textless1}R_{0}\textless1}. When the perturbation and the disease-related death rate are small, we derive that the disease will persist, which is measured through the difference between the solution and the endemic equilibrium of the deterministic model on average in time if R0\textgreater1}R_{0}\textgreater1}. Furthermore, we prove that the system is persistent in the mean. Finally, the results are illustrated by computer simulations.
                    </p>
                  </div>
                  <script>
                    function toggleBibtexZhao2015a(parameter){
                        var x= document.getElementById('aZhao2015a');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractZhao2015a(parameter){
                        var x= document.getElementById('absZhao2015a');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
                <li>
                  <style>
                    p.abstract{
                        background-color: #EFEFEF;
                        text-align: justify;
                        border-bottom: 2px #2E2E2E solid;
                        color: #2E2E2E;
                        width: 50em;
                        margin: auto auto;
                    }
                  </style>
                  <div>
                    <span id="Hardy-Littlewood-Polya1952">[36]G. H. Hardy, J. E. Littlewood, and G. Pólya, <i>Inequalities</i>. Cambridge, at the University Press, 1952, p. xii+324. </span>
                  </div>
                  <button type="button" class="btn btn-primary btn-raised btn-sm" onclick="toggleBibtexHardy-Littlewood-Polya1952()">
                    Bibtex
                  </button>
                  <div id="aHardy-Littlewood-Polya1952" style="display: none;">
                    <pre>@book{Hardy-Littlewood-Polya1952,
  author = {Hardy, G. H. and Littlewood, J. E. and P\'{o}lya, G.},
  title = {Inequalities},
  note = {2d ed},
  publisher = {Cambridge, at the University Press},
  year = {1952},
  pages = {xii+324},
  mrclass = {27.0X},
  mrnumber = {0046395}
}
</pre>
                  </div>
                  <div id="absHardy-Littlewood-Polya1952" style="display: none;">
                    <p class="abstract">
                    </p>
                  </div>
                  <script>
                    function toggleBibtexHardy-Littlewood-Polya1952(parameter){
                        var x= document.getElementById('aHardy-Littlewood-Polya1952');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                    function toggleAbstractHardy-Littlewood-Polya1952(parameter){
                        var x= document.getElementById('absHardy-Littlewood-Polya1952');
                        if (x.style.display === 'none') {
                            x.style.display = 'block';
                        } else {
                            x.style.display = 'none';
                        }
                    }
                  </script>
                </li>
              </ul>
            </div>
          </div>
          <!-- [INFO   ] [j1.layout.content_generator_page.html ] [ end place content ] -->
        </main>
        <!-- [INFO   ] [j1.layout.content_generator_page.html ] [ end main for full page ] -->
        <!-- [INFO   ] [j1.page.Threshold behaviour of a stochastic vector plant model for Tomato Yellow Curl Leaves disease                   ] [Save the current page to UserState cookie] -->
        <!-- script>
        var logger   = log4javascript.getLogger("j1.page.threshold behaviour of a stochastic vector plant model for tomato yellow curl leaves disease.init");
        var cookieUserState = 'j1.user.state'
        var log_text = 'Save the current browser page (threshold behaviour of a stochastic vector plant model for tomato yellow curl leaves disease) to cookie: ' +cookieUserState;
        var j1_user_state = {
          'authenticated' : 'false',
          'cookies_accepted' : 'false',
          'theme_css' : 'unknown',
          'theme_extension_css' : 'unknown',
          'theme_name' : 'unknown',
          'last_page' : 'unknown',
          'last_provider' : 'unknown',
          'last_signin' : 'unknown',
          'provider' : 'unknown',
          'user_name' : 'unknown',
          'user_id' : 'unknown'
        };
        // Set|Detect J1 App status
        j1_user_state.last_page = "/pages/public/preprints/sto_mle/main_sto_tycl/"
        j1.setUserState( j1_user_state );
        logger.info(log_text);
        </script -->
        <!-- [INFO   ] [j1.layout.content_writer.proc         ] [ load region 'body-main' for layout: page ] -->
        <section id="fab-container" class=""></section>
        <!-- [INFO   ] [j1.layout.content_writer.proc         ] [ placement for all lane items finished ] -->
        <!-- [DEBUG  ] [j1.layout.content_writer.proc         ] [ reset all lane items ] -->
        <!-- [INFO   ] [j1.layout.content_writer.proc         ] [ end processing ] -->
      </div>
      <!-- [INFO   ] [j1.layout.page.html                    ] [ end content_writer processing for region: 'body-main' ] -->
      <!-- [INFO   ] [j1.layout.default.html                ] [ end content inherited by layout: page ] -->
      <!-- [INFO   ] [j1.layout.default.html                ] [ call default_writer for region: 'body-footer' ] -->
      <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ start processing: load region body-footer, layout: page ] -->
      <footer id="exception_container" class="container-fluid no-padding"></footer>
      <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ end processing: load region head, layout: page ] -->
      <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ start processing: load region body-footer, layout: page ] -->
      <footer id="j1_footer" class=""></footer>
      <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ end processing: load region head, layout: page ] -->
      <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ start processing: load region body-footer, layout: page ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ start processing. collect data for region: body-footer, layout: page ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: core ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: core ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'template core is being initialized';
          logger.info(log_text);
                // [INFO   ] [j1.layout.module_writer.proc          ] [ pass init data for: core ]
                var settings = $.extend({}, {"layout": "page", "fab_menu_id": "page_ctrl_simple", "scrollbar": "false", "toc": "true", "comments": "", "scrollDynamicPagesTopOnChange": ""});
                  j1.init(settings);
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: ga-opt-in ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: ga-opt-in ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'gaOptIn is being initialized';
          logger.info(log_text);
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: jquery-ui ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: logger ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: logger ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'module logger is being initialized';
          logger.info(log_text);
                  j1.adapter.logger.init();
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: backstretch ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: backstretch ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'Backstretch is being initialized';
          logger.info(log_text);
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: attic ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: attic ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'module attic is being initialized';
          logger.info(log_text);
              // [INFO   ] [j1.layout.module_writer.proc          ] [ pass module specific frontmatter options for: attic ]
              var settings = $.extend({}, {"slides":[{"url":"/assets/images/modules/attics/annie-spratt-1920x1280.jpg", "alt":"Photo by Annie on Unsplash", "alignY":0.3, "badge":{"type":"unsplash", "author":"Sigmund", "href":"https://unsplash.com/@sigmund"}}]}, {"layout": "page", "scrollbar": "false", "toc": "true", "comments": ""});
                j1.adapter.attic.init(settings);
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: navigator ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: navigator ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'module navigator is being initialized';
          logger.info(log_text);
                  j1.adapter.navigator.init();
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: mmenu ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: mmenu ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'module mmenu is being initialized';
          logger.info(log_text);
                  j1.adapter.mmenu.init();
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: themer ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: themer ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'module themer is being initialized';
          logger.info(log_text);
                  j1.adapter.themer.init();
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: toccer ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: toccer ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'module toccer is being initialized';
          logger.info(log_text);
                // [INFO   ] [j1.layout.module_writer.proc          ] [ pass init data for: toccer ]
                var settings = $.extend({}, {"layout": "page", "fab_menu_id": "page_ctrl_simple", "scrollbar": "false", "toc": "true", "comments": "", "scrollDynamicPagesTopOnChange": ""});
                  j1.adapter.toccer.init(settings);
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: cookieConsent ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: cookieConsent ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'module cookieConsent is being initialized';
          logger.info(log_text);
                  j1.adapter.cookieConsent.init();
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: jq_mouse_wheel ] -->
      <!-- [WARN   ] [j1.layout.module_writer.proc          ] [ jq_mouse_wheel module detected as: false ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: theme_switcher ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: theme_switcher ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'module theme_switcher is being initialized';
          logger.info(log_text);
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: bs5 ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: bs5 ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'Bootstrap JS is being initialized';
          logger.info(log_text);
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: bmd ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: bmd ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'Bootstrap Material Design is being initialized';
          logger.info(log_text);
                  j1.adapter.bmd.init();
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: materialze ] -->
      <!-- [WARN   ] [j1.layout.module_writer.proc          ] [ materialze module detected as: false ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: rtable ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: rtable ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'module rtable is being initialized';
          logger.info(log_text);
                  j1.adapter.rtable.init();
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: asciidoctor ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: asciidoctor ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'HTML writer for Asciidoctor is being initialized';
          logger.info(log_text);
                  j1.adapter.asciidoctor.init();
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: css_flags_previewer ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: css_flags_previewer ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'module css_flags_previewer is being initialized';
          logger.info(log_text);
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: fabm ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: fabm ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'FAB Menus is being initialized';
          logger.info(log_text);
                // [INFO   ] [j1.layout.module_writer.proc          ] [ pass init data for: fabm ]
                var settings = $.extend({}, {"layout": "page", "fab_menu_id": "page_ctrl_simple", "scrollbar": "false", "toc": "true", "comments": "", "scrollDynamicPagesTopOnChange": ""});
                  j1.adapter.fab.init(settings);
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: lunr ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: lunr ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'J1 Lunr is being initialized';
          logger.info(log_text);
                  j1.adapter.lunr.init();
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: rtextresizer ] -->
      <!-- [WARN   ] [j1.layout.module_writer.proc          ] [ rtextresizer module detected as: false ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: msDropdown ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: msDropdown ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'module msDropdown is being initialized';
          logger.info(log_text);
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: translator ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: translator ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'module translator is being initialized';
          logger.info(log_text);
                  j1.adapter.translator.init();
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: analytics ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: analytics ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'module analytics is being initialized';
          logger.info(log_text);
                  j1.adapter.analytics.init();
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: CustomModule ] -->
      <!-- [WARN   ] [j1.layout.module_writer.proc          ] [ CustomModule module detected as: false ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: customFunctions ] -->
      <!-- [WARN   ] [j1.layout.module_writer.proc          ] [ customFunctions module detected as: false ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: rouge ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: rouge ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'theme rouge is being initialized';
          logger.info(log_text);
                  j1.adapter.rouge.init();
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: animate ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: animate ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'Animate is being initialized';
          logger.info(log_text);
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: justifiedGalleryCustomizer ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: framer ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: lightbox ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: lightbox ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'module lightbox is being initialized';
          logger.info(log_text);
                // [INFO   ] [j1.layout.module_writer.proc          ] [ pass init data for: lightbox ]
                var settings = $.extend({}, {"layout": "page", "fab_menu_id": "page_ctrl_simple", "scrollbar": "false", "toc": "true", "comments": "", "scrollDynamicPagesTopOnChange": ""});
                  j1.adapter.lightbox.init(settings);
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: twemoji ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: twemoji_picker ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: mdi_previewer ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: mdil_previewer ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: iconify ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: algolia ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: clipboard ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: clipboard ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'module clipboard is being initialized';
          logger.info(log_text);
                  j1.adapter.clipboard.init();
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: carousel ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: carousel ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'module carousel is being initialized';
          logger.info(log_text);
                // [INFO   ] [j1.layout.module_writer.proc          ] [ pass init data for: carousel ]
                var settings = $.extend({}, {"layout": "page", "fab_menu_id": "page_ctrl_simple", "scrollbar": "false", "toc": "true", "comments": "", "scrollDynamicPagesTopOnChange": ""});
                  j1.adapter.carousel.init(settings);
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: justifiedGallery ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: justifiedGallery ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'module justifiedGallery is being initialized';
          logger.info(log_text);
                // [INFO   ] [j1.layout.module_writer.proc          ] [ pass init data for: justifiedGallery ]
                var settings = $.extend({}, {"layout": "page", "fab_menu_id": "page_ctrl_simple", "scrollbar": "false", "toc": "true", "comments": "", "scrollDynamicPagesTopOnChange": ""});
                  j1.adapter.justifiedGallery.init(settings);
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: lightGallery ] -->
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ place J1 adapter for resource: lightGallery ] -->
      <script>
        $(document).ready(function() {
          var logger = log4javascript.getLogger("j1.module_loader");
          var log_text = 'module lightGallery is being initialized';
          logger.info(log_text);
         });
      </script>
      <!-- [INFO   ] [j1.layout.module_writer.proc          ] [ placement for all data_keys finished ] -->
      <!-- [DEBUG  ] [j1.layout.module_writer.proc          ] [ reset all data_keys ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: video_js ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: vimeo_froogaloop ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: vimeo_player ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: rangeslider ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: datatables ] -->
      <!-- [WARN   ] [j1.layout.module_writer.proc          ] [ datatables module detected as: false ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: scroller ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: deeplAPI ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: crypto ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: advertising ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: comments ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: dropdowns ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: nbinteract ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: plotly ] -->
      <!-- [WARN   ] [j1.layout.module_writer.proc          ] [ plotly module detected as: false ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: bokeh ] -->
      <!-- [WARN   ] [j1.layout.module_writer.proc          ] [ bokeh module detected as: false ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: vega ] -->
      <!-- [WARN   ] [j1.layout.module_writer.proc          ] [ vega module detected as: false ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ process resource: require ] -->
      <!-- [INFO   ] [j1.layout_module_generator.html       ] [ end processing ] -->
      <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ end processing: load region head, layout: page ] -->
      <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ placement for all lane items finished ] -->
      <!-- [DEBUG  ] [j1.layout.default_writer.proc         ] [ reset all lane items ] -->
      <!-- [INFO   ] [j1.layout.default_writer.proc         ] [ end processing ] -->
      <!-- [INFO   ] [j1.layout.default.html                ] [ end content_writer default_writer for region: 'body-footer' ] -->
    </div>
    <!-- [INFO   ] [j1.layout.default.html                ] [ end suppress flicker ] -->
    <!-- [INFO   ] [j1.layout.default.html                ] [ place noscript info|error page ] -->
    <noscript>
      <meta charset="utf-8">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <meta name="viewport" content="width=device-width, initial-scale=1">
      <link rel="icon" type="image/ico" href="/assets/images/modules/icons/j1/j1-32x32.ico">
      <title>Javascript disabled · 444</title>
      <style>
        .my { margin-top: 8.5rem !important;}
      </style>
      <style type="text/css">
        /*! normalize.css v5.0.0 | MIT License | github.com/necolas/normalize.css */html{font-family:sans-serif;line-height:1.15;-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%}body{margin:0}article,aside,footer,header,nav,section{display:block}h1{font-size:2em;margin:.67em 0}figcaption,figure,main{display:block}figure{margin:1em 40px}hr{box-sizing:content-box;height:0;overflow:visible}pre{font-family:monospace,monospace;font-size:1em}a{background-color:transparent;-webkit-text-decoration-skip:objects}a:active,a:hover{outline-width:0}abbr[title]{border-bottom:none;text-decoration:underline;text-decoration:underline dotted}b,strong{font-weight:inherit}b,strong{font-weight:bolder}code,kbd,samp{font-family:monospace,monospace;font-size:1em}dfn{font-style:italic}mark{background-color:#ff0;color:#000}small{font-size:80%}sub,sup{font-size:75%;line-height:0;position:relative;vertical-align:baseline}sub{bottom:-.25em}sup{top:-.5em}audio,video{display:inline-block}audio:not([controls]){display:none;height:0}img{border-style:none}svg:not(:root){overflow:hidden}button,input,optgroup,select,textarea{font-family:sans-serif;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=reset],[type=submit],button,html [type=button]{cursor: pointer}[type=button]::-moz-focus-inner,[type=reset]::-moz-focus-inner,[type=submit]::-moz-focus-inner,button::-moz-focus-inner{border-style:none;padding:0}[type=button]:-moz-focusring,[type=reset]:-moz-focusring,[type=submit]:-moz-focusring,button:-moz-focusring{outline:1px dotted ButtonText}fieldset{border:1px solid silver;margin:0 2px;padding:.35em .625em .75em}legend{box-sizing:border-box;color:inherit;display:table;max-width:100%;padding:0;white-space:normal}progress{display:inline-block;vertical-align:baseline}textarea{overflow:auto}[type=checkbox],[type=radio]{box-sizing:border-box;padding:0}[type=number]::-webkit-inner-spin-button,[type=number]::-webkit-outer-spin-button{height:auto}[type=search]{-webkit-appearance:textfield;outline-offset:-2px}[type=search]::-webkit-search-cancel-button,[type=search]::-webkit-search-decoration{-webkit-appearance:none}::-webkit-file-upload-button{cursor: pointer ;font:inherit}details,menu{display:block}summary{display:list-item}canvas{display:inline-block}template{display:none}[hidden]{display:none}body,html{width:100%;height:100%;background-color:#21232a}body{color:#fff;text-align:center;text-shadow:0 2px 4px rgba(0,0,0,.5);padding:0;min-height:100%;-webkit-box-shadow:inset 0 0 100px rgba(0,0,0,.8);box-shadow:inset 0 0 100px rgba(0,0,0,.8);display:table;font-family:"Open Sans",Arial,sans-serif}h1{font-family:inherit;font-weight:500;line-height:1.1;color:inherit;font-size:36px}h1 small{font-size:68%;font-weight:400;line-height:1;color:#777}a{text-decoration:none;color:#fff;font-size:inherit;border-bottom:dotted 1px #707070}.lead{color:silver;font-size:21px;line-height:1.4}.cover{display:table-cell;vertical-align:middle;padding:0 20px}footer{position:fixed;width:100%;height:40px;left:0;bottom:0;color:#a0a0a0;font-size:14px}
      </style>
      <div class="header my">
        <h1>Javascript disabled <small>444</small>
        </h1>
        <p class="lead">Your current browser settings reject the use of Javascript.</p>
        <div><a href="about:blank">Cannot continue</a></div>
      </div>
      <footer>
        <p>© 2022 J1 · All rights reserved · Technical Contact: <a href="mailto:saul.diazinfante@unison.mx?subject=Javascript%20disabled">saul.diazinfante@unison.mx</a></p>
      </footer>
    </noscript>
    <!-- [INFO   ] [j1.layout.default.html                ] [ place hidden color selector/themes: bg-primary ] -->
    <div id="bg-primary" class="bg-primary" style="display: none"></div>
    <!-- [INFO   ] [j1.layout.default.html                ] [ end default writer] -->
  </body>
  <!-- [INFO   ] [j1.layout.default.html                ] [ place container for the r-text resizer modal ] -->
  <div id="rtext_resizer_container"></div>
  <!-- [INFO   ] [j1.layout.default.html                ] [ initialize global JS scripts ] -->
  <script>
    $(function() {
      // initialize BS tooltips
      var tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'));
      var tooltipList = tooltipTriggerList.map(function (tooltipTriggerEl) {
        return new bootstrap.Tooltip(tooltipTriggerEl)
      });
      // initialize BS popovers
      var popoverTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="popover"]'));
      var popoverList = popoverTriggerList.map(function (popoverTriggerEl) {
        return new bootstrap.Popover(popoverTriggerEl)
      });
    });
  </script>
  <!-- [INFO   ] [j1.layout.default.html                ] [ end page processing ] -->
</html>

  
